---
title: >
  The `GeneTonic` Functionality, as it comes along
author:
- name: Federico Marini
  affiliation: 
  - &id1 Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz
  - Center for Thrombosis and Hemostasis (CTH), Mainz
  email: marinif@uni-mainz.de
- name: Simon Stahl
  affiliation: 
  - &id2 Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz
  email: simon.t.stahl@gmail.com
date: "`r BiocStyle::doc_date()`"
package: "`r BiocStyle::pkg_ver('GeneTonic')`"
output: 
  BiocStyle::html_document:
    toc_float: true
vignette: >
  %\VignetteIndexEntry{The GeneTonic Functionality}
  %\VignetteEncoding{UTF-8}  
  %\VignettePackage{GeneTonic}
  %\VignetteKeywords{GeneExpression, RNASeq, FunctionalAnnotation, Sequencing, Visualization, QualityControl, GUI}
  %\VignetteEngine{knitr::rmarkdown}
editor_options: 
  chunk_output_type: console
bibliography: GeneTonic.bib
---

**Compiled date**: `r Sys.Date()`

**Last edited**: 2019-06-24

**License**: `r packageDescription("GeneTonic")[["License"]]`

```{r setup, include = FALSE}
knitr::opts_chunk$set(
    collapse = TRUE,
    comment = "#>",
    error = FALSE,
    warning = FALSE,
    message = FALSE,
    eval = FALSE
)
```
  
---

```{r}
library(GeneTonic)
```

# Introduction

# Preliminaries

First of all we setup the required packages - this operation needs to be done once

```{r eval = FALSE}
install.packages("BiocManager")
BiocManager::install(c("DESeq2","macrophage","igraph","dplyr","tximeta","ideal", "pcaExplorer",
                       "visNetwork","ggraph", "clusterProfiler","plotly","knitr","rmarkdown"))
```

Other software/software packages include of course the latest version of R (and RStudio, for having a nice IDE for development).
Rtools might be required on Windows to guarantee all the package building process goes smooth, but this should matter at a later time point.

# Input required for `GeneTonic`

- some count matrix-like object, e.g. a `DESeqDataSet`
- some results-like object/table, e.g. a `DESeqResults`
- a table of results from functional enrichment annotation

## Obtaining the `DESeqDataSet` 

For the first steps we are going to use the `macrophage` dataset.
This dataset is described in the publication of [@Alasoo2018].

We process the quantifications in the chunks that follow

```{r, eval = TRUE}
library(macrophage)
library(magrittr)
dir <- system.file("extdata", package="macrophage")
coldata_macrophage <- read.csv(file.path(system.file("extdata", package="macrophage"), "coldata.csv"))
coldata_macrophage$files <- file.path(system.file("extdata", package="macrophage"), "quants", coldata_macrophage$names, "quant.sf.gz")
file.exists(coldata_macrophage$files)

coldata_macrophage$condition <- coldata_macrophage$condition_name
coldata_macrophage$line <- coldata_macrophage$line_id
coldata_macrophage$condition <- relevel(coldata_macrophage$condition, "naive")

head(coldata_macrophage)
```

Creating an object at the transcript level...

```{r, eval = TRUE}
library(SummarizedExperiment)
library(tximeta)
se_macrophage <- tximeta(coldata_macrophage, dropInfReps=TRUE)
se_macrophage
```

... and here summarized at the gene level.

```{r, eval = TRUE}
assayNames(se_macrophage)
gse_macrophage <- summarizeToGene(se_macrophage)
gse_macrophage
# adding gene names to facilitate readout later
library(org.Hs.eg.db)
gse_macrophage <- addIds(gse_macrophage,"SYMBOL")
```

It is easy with DESeq2 to run the complete differential expression analysis [@Love2014]. 
The design indicates that we want to control for the donor (`line`) and test for the effect on the `condition` (since it is specified as last in the formula of the `design` parameter). 

```{r}
library(DESeq2)
dds_macrophage <- DESeqDataSet(gse_macrophage, design = ~line + condition)

## using counts and average transcript lengths from tximeta
keep <- rowSums(counts(dds_macrophage) >= 10) >= 6
dds_macrophage <- dds_macrophage[keep,]
```

Before running the complete pipeline, we perform some basic exploratory analysis steps, looking at the PCA for the samples - this gives us an idea of how (dis)similar are the samples, e.g. whether they do cluster by condition, or cell line, or to spot out any obvious outlier.

```{r}
rld_macrophage <- rlog(dds_macrophage)
vst_macrophage <- vst(dds_macrophage)

pcaExplorer::pcaplot(vst_macrophage, intgroup = "condition", ellipse = FALSE)
pcaExplorer::pcaplot(vst_macrophage, intgroup = "line", ellipse = FALSE)
```

Seems all good to go.
We proceed then with running `DESeq` on `dds_macrophage`.

```{r}
dds_macrophage <- DESeq(dds_macrophage)
```

## Extracting the relevant `DESeqResults`

```{r}
res <- results(dds, contrast=c("condition","IFNg","naive"),
               lfcThreshold=1, alpha=0.01)
summary(res)

res$gene_name <- rowData(dds)$SYMBOL

DESeq2::plotMA(res, ylim=c(-10,10))

library("AnnotationDbi")
library("org.Hs.eg.db")

resOrdered <- as.data.frame(res[order(res$padj),])
de_df <- resOrdered[resOrdered$padj < .05 & !is.na(resOrdered$padj),]
de_symbols <- de_df$gene_name
bg_ids <- rowData(dds)$SYMBOL[rowSums(counts(dds)) > 0]
# bg_symbols <- mapIds(org.Hs.eg.db,
#                      keys=bg_ids,
#                      column="SYMBOL",
#                      keytype="ENSEMBL",
#                      multiVals="first")
library(topGO)
topgoDE_macro <- pcaExplorer::topGOtable(de_symbols, bg_ids,
                                           ontology = "BP",
                                           mapping = "org.Hs.eg.db",
                                           geneID = "symbol")


DT::datatable(topgoDE_macro)


## with goseq, from ideal

goseqDE_macro <- ideal::goseqTable(
  de.genes = substr(rownames(de_df),1,15),
  assayed.genes = substr(rownames(dds),1,15)
)
DT::datatable(goseqDE_macro)

```

Using other alternative procedures to similarly derive enriched functions/processes...



# Session Info {-}

```{r}
sessionInfo()
```

# References {-}
