---
title: >
  The `GeneTonic` Functionality, as it comes along
author:
- name: Federico Marini
  affiliation: 
  - &id1 Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz
  - Center for Thrombosis and Hemostasis (CTH), Mainz
  email: marinif@uni-mainz.de
- name: Simon Stahl
  affiliation: 
  - &id2 Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz
  email: simon.t.stahl@gmail.com
date: "`r BiocStyle::doc_date()`"
package: "`r BiocStyle::pkg_ver('GeneTonic')`"
output: 
  BiocStyle::html_document:
    toc_float: true
vignette: >
  %\VignetteIndexEntry{The GeneTonic Functionality}
  %\VignetteEncoding{UTF-8}  
  %\VignettePackage{GeneTonic}
  %\VignetteKeywords{GeneExpression, RNASeq, FunctionalAnnotation, Sequencing, Visualization, QualityControl, GUI}
  %\VignetteEngine{knitr::rmarkdown}
editor_options: 
  chunk_output_type: console
bibliography: GeneTonic.bib
---

**Compiled date**: `r Sys.Date()`

**Last edited**: 2019-06-24

**License**: `r packageDescription("GeneTonic")[["License"]]`

```{r setup, include = FALSE}
knitr::opts_chunk$set(
    collapse = TRUE,
    comment = "#>",
    error = FALSE,
    warning = FALSE,
    message = FALSE
)
```
  
---

```{r}
library(GeneTonic)
```

# Introduction

# Preliminaries

First of all we setup the required packages - this operation needs to be done once

```{r}
install.packages("BiocManager")
BiocManager::install(c("DESeq2","macrophage","igraph","dplyr","tximeta","ideal", "pcaExplorer",
                       "visNetwork","ggraph", "clusterProfiler","plotly","knitr","rmarkdown"))
```

Other software/software packages include of course the latest version of R (and RStudio, for having a nice IDE for development).
Rtools might be required on Windows to guarantee all the package building process goes smooth, but this should matter at a later time point.

# Input required for `GeneTonic`

- some count matrix-like object, e.g. a DESeqDataSet
- some results-like object/table, e.g. a DESeqResults
- a table of results from functional enrichment annotation

For the first steps we are going to use the `macrophage` dataset, that we process in the chunks that follow

```{r}
library(macrophage)
library(magrittr)
dir <- system.file("extdata", package="macrophage")
coldata <- file.path(dir, "coldata.csv") %>% 
  read.csv() %>% 
  dplyr::select(
    names, 
    id = sample_id, 
    line = line_id, 
    condition = condition_name
  ) %>%
  dplyr::mutate(
    files = file.path(dir, "quants", names, "quant.sf.gz"),
    condition = relevel(condition, "naive")
  )
head(coldata)

library(SummarizedExperiment)
library(tximeta)
se <- tximeta(coldata, dropInfReps=TRUE,)

assayNames(se)

gse <- summarizeToGene(se)



```

```{r}
library(DESeq2)
dds <- DESeqDataSet(gse, ~line + condition)
## using counts and average transcript lengths from tximeta
keep <- rowSums(counts(dds) >= 10) >= 6
dds <- dds[keep,]
dds <- DESeq(dds)
rld <- rlog(dds)
vst <- vst(dds)

dds <- addIds(dds,"SYMBOL")
```

```{r}
pcaExplorer::pcaplot(vst, intgroup = "condition")
pcaExplorer::pcaplot(vst, intgroup = "line")
```


# Session Info {-}

```{r}
sessionInfo()
```

