<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>The `GeneTonic` Functionality, as it comes along
 • GeneTonic</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="The `GeneTonic` Functionality, as it comes along
">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">GeneTonic</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.7.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/GeneTonic_manual.html">The `GeneTonic` User's Guide
</a>
    </li>
    <li>
      <a href="../articles/functionality_and_more.html">The `GeneTonic` Functionality, as it comes along
</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/federicomarini/GeneTonic">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>The <code>GeneTonic</code> Functionality, as it comes along</h1>
                        <h4 class="author">Federico Marini</h4>
            <address class="author_afil">
      Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz<br><a class="author_email" href="mailto:#"></a><a href="mailto:marinif@uni-mainz.de">marinif@uni-mainz.de</a>
      </address>
                  
            <h4 class="date">28 November 2019</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/federicomarini/GeneTonic/blob/master/vignettes/functionality_and_more.Rmd"><code>vignettes/functionality_and_more.Rmd</code></a></small>
      <div class="hidden name"><code>functionality_and_more.Rmd</code></div>

    </div>

    
    
<p><strong>Compiled date</strong>: 2019-11-28</p>
<p><strong>Last edited</strong>: 2019-06-24</p>
<p><strong>License</strong>: MIT + file LICENSE</p>
<hr>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(GeneTonic)</a></code></pre></div>
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
</div>
<div id="preliminaries" class="section level1">
<h1 class="hasAnchor">
<a href="#preliminaries" class="anchor"></a>Preliminaries</h1>
<p>First of all we setup the required packages - this operation needs to be done once</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span>(<span class="st">"BiocManager"</span>)</a>
<a class="sourceLine" id="cb2-2" data-line-number="2">BiocManager<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"DESeq2"</span>,<span class="st">"macrophage"</span>,<span class="st">"igraph"</span>,<span class="st">"dplyr"</span>,<span class="st">"tximeta"</span>,<span class="st">"ideal"</span>, <span class="st">"pcaExplorer"</span>, <span class="st">"org.Hs.eg.db"</span>,</a>
<a class="sourceLine" id="cb2-3" data-line-number="3">                       <span class="st">"visNetwork"</span>,<span class="st">"ggraph"</span>, <span class="st">"clusterProfiler"</span>,<span class="st">"plotly"</span>,<span class="st">"knitr"</span>,<span class="st">"rmarkdown"</span>))</a></code></pre></div>
<p>Other software/software packages include of course the latest version of R (and RStudio, for having a nice IDE for development). Rtools might be required on Windows to guarantee all the package building process goes smooth, but this should matter at a later time point.</p>
</div>
<div id="input-required-for-genetonic" class="section level1">
<h1 class="hasAnchor">
<a href="#input-required-for-genetonic" class="anchor"></a>Input required for <code>GeneTonic</code>
</h1>
<ul>
<li>some count matrix-like object, e.g. a <code>DESeqDataSet</code>
</li>
<li>some results-like object/table, e.g. a <code>DESeqResults</code>
</li>
<li>a table of results from functional enrichment annotation</li>
</ul>
<div id="obtaining-the-deseqdataset" class="section level2">
<h2 class="hasAnchor">
<a href="#obtaining-the-deseqdataset" class="anchor"></a>Obtaining the <code>DESeqDataSet</code>
</h2>
<p>For the first steps we are going to use the <code>macrophage</code> dataset. This dataset is described in the publication of <span class="citation">(Alasoo et al. 2018)</span>.</p>
<p>We process the quantifications in the chunks that follow</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(macrophage)</a>
<a class="sourceLine" id="cb3-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(magrittr)</a>
<a class="sourceLine" id="cb3-3" data-line-number="3">dir &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">"extdata"</span>, <span class="dt">package=</span><span class="st">"macrophage"</span>)</a>
<a class="sourceLine" id="cb3-4" data-line-number="4">coldata_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/utils/read.table.html">read.csv</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/file.path.html">file.path</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">"extdata"</span>, <span class="dt">package=</span><span class="st">"macrophage"</span>), <span class="st">"coldata.csv"</span>))</a>
<a class="sourceLine" id="cb3-5" data-line-number="5">coldata_macrophage<span class="op">$</span>files &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/file.path.html">file.path</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">"extdata"</span>, <span class="dt">package=</span><span class="st">"macrophage"</span>), <span class="st">"quants"</span>, coldata_macrophage<span class="op">$</span>names, <span class="st">"quant.sf.gz"</span>)</a>
<a class="sourceLine" id="cb3-6" data-line-number="6"><span class="kw"><a href="https://rdrr.io/r/base/files.html">file.exists</a></span>(coldata_macrophage<span class="op">$</span>files)</a>
<a class="sourceLine" id="cb3-7" data-line-number="7"><span class="co">#&gt;  [1] TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE</span></a>
<a class="sourceLine" id="cb3-8" data-line-number="8"><span class="co">#&gt; [16] TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE</span></a>
<a class="sourceLine" id="cb3-9" data-line-number="9"></a>
<a class="sourceLine" id="cb3-10" data-line-number="10">coldata_macrophage<span class="op">$</span>condition &lt;-<span class="st"> </span>coldata_macrophage<span class="op">$</span>condition_name</a>
<a class="sourceLine" id="cb3-11" data-line-number="11">coldata_macrophage<span class="op">$</span>line &lt;-<span class="st"> </span>coldata_macrophage<span class="op">$</span>line_id</a>
<a class="sourceLine" id="cb3-12" data-line-number="12">coldata_macrophage<span class="op">$</span>condition &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/stats/relevel.html">relevel</a></span>(coldata_macrophage<span class="op">$</span>condition, <span class="st">"naive"</span>)</a>
<a class="sourceLine" id="cb3-13" data-line-number="13"></a>
<a class="sourceLine" id="cb3-14" data-line-number="14"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(coldata_macrophage)</a>
<a class="sourceLine" id="cb3-15" data-line-number="15"><span class="co">#&gt;            names sample_id line_id replicate condition_name macrophage_harvest</span></a>
<a class="sourceLine" id="cb3-16" data-line-number="16"><span class="co">#&gt; 1 SAMEA103885102    diku_A  diku_1         1          naive          11/6/2015</span></a>
<a class="sourceLine" id="cb3-17" data-line-number="17"><span class="co">#&gt; 2 SAMEA103885347    diku_B  diku_1         1           IFNg          11/6/2015</span></a>
<a class="sourceLine" id="cb3-18" data-line-number="18"><span class="co">#&gt; 3 SAMEA103885043    diku_C  diku_1         1         SL1344          11/6/2015</span></a>
<a class="sourceLine" id="cb3-19" data-line-number="19"><span class="co">#&gt; 4 SAMEA103885392    diku_D  diku_1         1    IFNg_SL1344          11/6/2015</span></a>
<a class="sourceLine" id="cb3-20" data-line-number="20"><span class="co">#&gt; 5 SAMEA103885182    eiwy_A  eiwy_1         1          naive         11/25/2015</span></a>
<a class="sourceLine" id="cb3-21" data-line-number="21"><span class="co">#&gt; 6 SAMEA103885136    eiwy_B  eiwy_1         1           IFNg         11/25/2015</span></a>
<a class="sourceLine" id="cb3-22" data-line-number="22"><span class="co">#&gt;   salmonella_date ng_ul_mean rna_extraction rna_submit        library_pool</span></a>
<a class="sourceLine" id="cb3-23" data-line-number="23"><span class="co">#&gt; 1      11/13/2015   293.9625     11/30/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-24" data-line-number="24"><span class="co">#&gt; 2      11/13/2015   293.9625     11/30/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-25" data-line-number="25"><span class="co">#&gt; 3      11/13/2015   293.9625     11/30/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-26" data-line-number="26"><span class="co">#&gt; 4      11/13/2015   293.9625     11/30/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-27" data-line-number="27"><span class="co">#&gt; 5       12/2/2015   193.5450      12/3/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-28" data-line-number="28"><span class="co">#&gt; 6       12/2/2015   193.5450      12/3/2015  12/9/2015 3226_RNAauto-091215</span></a>
<a class="sourceLine" id="cb3-29" data-line-number="29"><span class="co">#&gt;   chemistry rna_auto</span></a>
<a class="sourceLine" id="cb3-30" data-line-number="30"><span class="co">#&gt; 1   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-31" data-line-number="31"><span class="co">#&gt; 2   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-32" data-line-number="32"><span class="co">#&gt; 3   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-33" data-line-number="33"><span class="co">#&gt; 4   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-34" data-line-number="34"><span class="co">#&gt; 5   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-35" data-line-number="35"><span class="co">#&gt; 6   V4_auto        1</span></a>
<a class="sourceLine" id="cb3-36" data-line-number="36"><span class="co">#&gt;                                                                         files</span></a>
<a class="sourceLine" id="cb3-37" data-line-number="37"><span class="co">#&gt; 1 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885102/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-38" data-line-number="38"><span class="co">#&gt; 2 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885347/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-39" data-line-number="39"><span class="co">#&gt; 3 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885043/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-40" data-line-number="40"><span class="co">#&gt; 4 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885392/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-41" data-line-number="41"><span class="co">#&gt; 5 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885182/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-42" data-line-number="42"><span class="co">#&gt; 6 /home/travis/R/Library/macrophage/extdata/quants/SAMEA103885136/quant.sf.gz</span></a>
<a class="sourceLine" id="cb3-43" data-line-number="43"><span class="co">#&gt;     condition   line</span></a>
<a class="sourceLine" id="cb3-44" data-line-number="44"><span class="co">#&gt; 1       naive diku_1</span></a>
<a class="sourceLine" id="cb3-45" data-line-number="45"><span class="co">#&gt; 2        IFNg diku_1</span></a>
<a class="sourceLine" id="cb3-46" data-line-number="46"><span class="co">#&gt; 3      SL1344 diku_1</span></a>
<a class="sourceLine" id="cb3-47" data-line-number="47"><span class="co">#&gt; 4 IFNg_SL1344 diku_1</span></a>
<a class="sourceLine" id="cb3-48" data-line-number="48"><span class="co">#&gt; 5       naive eiwy_1</span></a>
<a class="sourceLine" id="cb3-49" data-line-number="49"><span class="co">#&gt; 6        IFNg eiwy_1</span></a></code></pre></div>
<p>Creating an object at the transcript level…</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(SummarizedExperiment)</a>
<a class="sourceLine" id="cb4-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(tximeta)</a>
<a class="sourceLine" id="cb4-3" data-line-number="3">se_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/tximeta/man/tximeta.html">tximeta</a></span>(coldata_macrophage, <span class="dt">dropInfReps=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb4-4" data-line-number="4"><span class="co">#&gt; Error in function (type, msg, asError = TRUE)  : </span></a>
<a class="sourceLine" id="cb4-5" data-line-number="5"><span class="co">#&gt;   Failed to connect to ftp.ncbi.nlm.nih.gov port 21: Connection timed out</span></a>
<a class="sourceLine" id="cb4-6" data-line-number="6">se_macrophage</a>
<a class="sourceLine" id="cb4-7" data-line-number="7"><span class="co">#&gt; class: RangedSummarizedExperiment </span></a>
<a class="sourceLine" id="cb4-8" data-line-number="8"><span class="co">#&gt; dim: 205870 24 </span></a>
<a class="sourceLine" id="cb4-9" data-line-number="9"><span class="co">#&gt; metadata(6): tximetaInfo quantInfo ... txomeInfo txdbInfo</span></a>
<a class="sourceLine" id="cb4-10" data-line-number="10"><span class="co">#&gt; assays(3): counts abundance length</span></a>
<a class="sourceLine" id="cb4-11" data-line-number="11"><span class="co">#&gt; rownames(205870): ENST00000456328.2 ENST00000450305.2 ...</span></a>
<a class="sourceLine" id="cb4-12" data-line-number="12"><span class="co">#&gt;   ENST00000387460.2 ENST00000387461.2</span></a>
<a class="sourceLine" id="cb4-13" data-line-number="13"><span class="co">#&gt; rowData names(3): tx_id gene_id tx_name</span></a>
<a class="sourceLine" id="cb4-14" data-line-number="14"><span class="co">#&gt; colnames(24): SAMEA103885102 SAMEA103885347 ... SAMEA103885308</span></a>
<a class="sourceLine" id="cb4-15" data-line-number="15"><span class="co">#&gt;   SAMEA103884949</span></a>
<a class="sourceLine" id="cb4-16" data-line-number="16"><span class="co">#&gt; colData names(15): names sample_id ... condition line</span></a>
<a class="sourceLine" id="cb4-17" data-line-number="17"></a>
<a class="sourceLine" id="cb4-18" data-line-number="18"><span class="co"># saveRDS(se_macrophage,file="WIP/se_macrophage.rds")</span></a>
<a class="sourceLine" id="cb4-19" data-line-number="19"><span class="co"># dir.create("WIP")</span></a>
<a class="sourceLine" id="cb4-20" data-line-number="20"><span class="co"># se_macrophage &lt;- readRDS(file = "WIP/se_macrophage.rds")</span></a>
<a class="sourceLine" id="cb4-21" data-line-number="21">gse_macrophage &lt;-<span class="st"> </span><span class="kw">summarizeToGene</span>(se_macrophage)</a></code></pre></div>
<p>… and here summarized at the gene level.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">assayNames</a></span>(se_macrophage)</a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="co">#&gt; [1] "counts"    "abundance" "length"</span></a>
<a class="sourceLine" id="cb5-3" data-line-number="3">gse_macrophage &lt;-<span class="st"> </span><span class="kw">summarizeToGene</span>(se_macrophage)</a>
<a class="sourceLine" id="cb5-4" data-line-number="4">gse_macrophage</a>
<a class="sourceLine" id="cb5-5" data-line-number="5"><span class="co">#&gt; class: RangedSummarizedExperiment </span></a>
<a class="sourceLine" id="cb5-6" data-line-number="6"><span class="co">#&gt; dim: 58294 24 </span></a>
<a class="sourceLine" id="cb5-7" data-line-number="7"><span class="co">#&gt; metadata(6): tximetaInfo quantInfo ... txomeInfo txdbInfo</span></a>
<a class="sourceLine" id="cb5-8" data-line-number="8"><span class="co">#&gt; assays(3): counts abundance length</span></a>
<a class="sourceLine" id="cb5-9" data-line-number="9"><span class="co">#&gt; rownames(58294): ENSG00000000003.14 ENSG00000000005.5 ...</span></a>
<a class="sourceLine" id="cb5-10" data-line-number="10"><span class="co">#&gt;   ENSG00000285993.1 ENSG00000285994.1</span></a>
<a class="sourceLine" id="cb5-11" data-line-number="11"><span class="co">#&gt; rowData names(1): gene_id</span></a>
<a class="sourceLine" id="cb5-12" data-line-number="12"><span class="co">#&gt; colnames(24): SAMEA103885102 SAMEA103885347 ... SAMEA103885308</span></a>
<a class="sourceLine" id="cb5-13" data-line-number="13"><span class="co">#&gt;   SAMEA103884949</span></a>
<a class="sourceLine" id="cb5-14" data-line-number="14"><span class="co">#&gt; colData names(15): names sample_id ... condition line</span></a>
<a class="sourceLine" id="cb5-15" data-line-number="15"><span class="co"># adding gene names to facilitate readout later</span></a>
<a class="sourceLine" id="cb5-16" data-line-number="16"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(org.Hs.eg.db)</a>
<a class="sourceLine" id="cb5-17" data-line-number="17">gse_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/tximeta/man/addIds.html">addIds</a></span>(gse_macrophage,<span class="st">"SYMBOL"</span>)</a></code></pre></div>
<p>It is easy with DESeq2 to run the complete differential expression analysis <span class="citation">(Love, Huber, and Anders 2014)</span>. The design indicates that we want to control for the donor (<code>line</code>) and test for the effect on the <code>condition</code> (since it is specified as last in the formula of the <code>design</code> parameter).</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(DESeq2)</a>
<a class="sourceLine" id="cb6-2" data-line-number="2">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/DESeqDataSet.html">DESeqDataSet</a></span>(gse_macrophage, <span class="dt">design =</span> <span class="op">~</span>line <span class="op">+</span><span class="st"> </span>condition)</a>
<a class="sourceLine" id="cb6-3" data-line-number="3"><span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage) &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/substr.html">substr</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage),<span class="dv">1</span>,<span class="dv">15</span>)</a>
<a class="sourceLine" id="cb6-4" data-line-number="4"></a>
<a class="sourceLine" id="cb6-5" data-line-number="5">anno_df &lt;-<span class="st"> </span>pcaExplorer<span class="op">::</span><span class="kw">get_annotation_orgdb</span>(dds_macrophage,<span class="st">"org.Hs.eg.db"</span>,<span class="st">"ENSEMBL"</span>)</a>
<a class="sourceLine" id="cb6-6" data-line-number="6"></a>
<a class="sourceLine" id="cb6-7" data-line-number="7">## using counts and average transcript lengths from tximeta</a>
<a class="sourceLine" id="cb6-8" data-line-number="8">keep &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/colSums.html">rowSums</a></span>(<span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/counts.html">counts</a></span>(dds_macrophage) <span class="op">&gt;=</span><span class="st"> </span><span class="dv">10</span>) <span class="op">&gt;=</span><span class="st"> </span><span class="dv">6</span></a>
<a class="sourceLine" id="cb6-9" data-line-number="9">dds_macrophage &lt;-<span class="st"> </span>dds_macrophage[keep,]</a></code></pre></div>
<p>Before running the complete pipeline, we perform some basic exploratory analysis steps, looking at the PCA for the samples - this gives us an idea of how (dis)similar are the samples, e.g. whether they do cluster by condition, or cell line, or to spot out any obvious outlier.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">rld_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/rlog.html">rlog</a></span>(dds_macrophage)</a>
<a class="sourceLine" id="cb7-2" data-line-number="2">vst_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/vst.html">vst</a></span>(dds_macrophage)</a>
<a class="sourceLine" id="cb7-3" data-line-number="3"></a>
<a class="sourceLine" id="cb7-4" data-line-number="4">pcaExplorer<span class="op">::</span><span class="kw">pcaplot</span>(vst_macrophage, <span class="dt">intgroup =</span> <span class="st">"condition"</span>, <span class="dt">ellipse =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb7-5" data-line-number="5">pcaExplorer<span class="op">::</span><span class="kw">pcaplot</span>(vst_macrophage, <span class="dt">intgroup =</span> <span class="st">"line"</span>, <span class="dt">ellipse =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb7-6" data-line-number="6"></a>
<a class="sourceLine" id="cb7-7" data-line-number="7"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"org.Hs.eg.db"</span>)</a>
<a class="sourceLine" id="cb7-8" data-line-number="8">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/tximeta/man/addIds.html">addIds</a></span>(dds_macrophage,<span class="st">"SYMBOL"</span>)</a></code></pre></div>
<p>Seems all good to go. We proceed then with running <code>DESeq</code> on <code>dds_macrophage</code>.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/DESeq.html">DESeq</a></span>(dds_macrophage)</a></code></pre></div>
</div>
<div id="extracting-the-relevant-deseqresults" class="section level2">
<h2 class="hasAnchor">
<a href="#extracting-the-relevant-deseqresults" class="anchor"></a>Extracting the relevant <code>DESeqResults</code>
</h2>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1">res_macrophage_IFNg_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/results.html">results</a></span>(dds_macrophage, </a>
<a class="sourceLine" id="cb9-2" data-line-number="2">                                        <span class="dt">contrast=</span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"condition"</span>,<span class="st">"IFNg"</span>,<span class="st">"naive"</span>),</a>
<a class="sourceLine" id="cb9-3" data-line-number="3">                                        <span class="dt">lfcThreshold=</span><span class="dv">1</span>, <span class="dt">alpha=</span><span class="fl">0.01</span>)</a>
<a class="sourceLine" id="cb9-4" data-line-number="4"><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/summary.html">summary</a></span>(res_macrophage_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb9-5" data-line-number="5"></a>
<a class="sourceLine" id="cb9-6" data-line-number="6">res_macrophage_IFNg_vs_naive<span class="op">$</span>SYMBOL &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">rowData</a></span>(dds_macrophage)<span class="op">$</span>SYMBOL</a>
<a class="sourceLine" id="cb9-7" data-line-number="7"></a>
<a class="sourceLine" id="cb9-8" data-line-number="8">DESeq2<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/plotMA.html">plotMA</a></span>(res_macrophage_IFNg_vs_naive, <span class="dt">ylim=</span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="op">-</span><span class="dv">10</span>,<span class="dv">10</span>))</a>
<a class="sourceLine" id="cb9-9" data-line-number="9"></a>
<a class="sourceLine" id="cb9-10" data-line-number="10"></a>
<a class="sourceLine" id="cb9-11" data-line-number="11">res_macrophage_SL1344_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/results.html">results</a></span>(dds_macrophage, </a>
<a class="sourceLine" id="cb9-12" data-line-number="12">                                        <span class="dt">contrast=</span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"condition"</span>,<span class="st">"SL1344"</span>,<span class="st">"naive"</span>),</a>
<a class="sourceLine" id="cb9-13" data-line-number="13">                                        <span class="dt">lfcThreshold=</span><span class="dv">1</span>, <span class="dt">alpha=</span><span class="fl">0.01</span>)</a>
<a class="sourceLine" id="cb9-14" data-line-number="14">res_macrophage_SL1344_vs_naive<span class="op">$</span>SYMBOL &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">rowData</a></span>(dds_macrophage)<span class="op">$</span>SYMBOL</a>
<a class="sourceLine" id="cb9-15" data-line-number="15"></a>
<a class="sourceLine" id="cb9-16" data-line-number="16"><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/summary.html">summary</a></span>(res_macrophage_SL1344_vs_naive)</a>
<a class="sourceLine" id="cb9-17" data-line-number="17"></a>
<a class="sourceLine" id="cb9-18" data-line-number="18"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"AnnotationDbi"</span>)</a>
<a class="sourceLine" id="cb9-19" data-line-number="19"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"org.Hs.eg.db"</span>)</a>
<a class="sourceLine" id="cb9-20" data-line-number="20"></a>
<a class="sourceLine" id="cb9-21" data-line-number="21"><span class="co"># resOrdered &lt;- as.data.frame(res_macrophage_IFNg_vs_naive[order(res_macrophage_IFNg_vs_naive$padj),])</span></a>
<a class="sourceLine" id="cb9-22" data-line-number="22"><span class="co"># de_df &lt;- resOrdered[resOrdered$padj &lt; .05 &amp; !is.na(resOrdered$padj),]</span></a>
<a class="sourceLine" id="cb9-23" data-line-number="23">de_symbols_IFNg_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="../reference/deseqresult2df.html">deseqresult2df</a></span>(res_macrophage_IFNg_vs_naive, <span class="dt">FDR =</span> <span class="fl">0.05</span>)<span class="op">$</span>SYMBOL</a>
<a class="sourceLine" id="cb9-24" data-line-number="24">de_symbols_SL1344_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="../reference/deseqresult2df.html">deseqresult2df</a></span>(res_macrophage_SL1344_vs_naive, <span class="dt">FDR =</span> <span class="fl">0.05</span>)<span class="op">$</span>SYMBOL</a>
<a class="sourceLine" id="cb9-25" data-line-number="25"></a>
<a class="sourceLine" id="cb9-26" data-line-number="26">bg_ids &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">rowData</a></span>(dds_macrophage)<span class="op">$</span>SYMBOL[<span class="kw"><a href="https://rdrr.io/r/base/colSums.html">rowSums</a></span>(<span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/counts.html">counts</a></span>(dds_macrophage)) <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>]</a>
<a class="sourceLine" id="cb9-27" data-line-number="27"><span class="co"># bg_symbols &lt;- mapIds(org.Hs.eg.db,</span></a>
<a class="sourceLine" id="cb9-28" data-line-number="28"><span class="co">#                      keys=bg_ids,</span></a>
<a class="sourceLine" id="cb9-29" data-line-number="29"><span class="co">#                      column="SYMBOL",</span></a>
<a class="sourceLine" id="cb9-30" data-line-number="30"><span class="co">#                      keytype="ENSEMBL",</span></a>
<a class="sourceLine" id="cb9-31" data-line-number="31"><span class="co">#                      multiVals="first")</span></a></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(topGO)</a>
<a class="sourceLine" id="cb10-2" data-line-number="2">topgoDE_macrophage_IFNg_vs_naive &lt;-<span class="st"> </span>pcaExplorer<span class="op">::</span><span class="kw">topGOtable</span>(de_symbols_IFNg_vs_naive, </a>
<a class="sourceLine" id="cb10-3" data-line-number="3">                                                            bg_ids,</a>
<a class="sourceLine" id="cb10-4" data-line-number="4">                                                            <span class="dt">ontology =</span> <span class="st">"BP"</span>,</a>
<a class="sourceLine" id="cb10-5" data-line-number="5">                                                            <span class="dt">mapping =</span> <span class="st">"org.Hs.eg.db"</span>,</a>
<a class="sourceLine" id="cb10-6" data-line-number="6">                                                            <span class="dt">geneID =</span> <span class="st">"symbol"</span>)</a>
<a class="sourceLine" id="cb10-7" data-line-number="7">topgoDE_macrophage_SL1344_vs_naive &lt;-<span class="st"> </span>pcaExplorer<span class="op">::</span><span class="kw">topGOtable</span>(de_symbols_SL1344_vs_naive, </a>
<a class="sourceLine" id="cb10-8" data-line-number="8">                                                              bg_ids,</a>
<a class="sourceLine" id="cb10-9" data-line-number="9">                                                              <span class="dt">ontology =</span> <span class="st">"BP"</span>,</a>
<a class="sourceLine" id="cb10-10" data-line-number="10">                                                              <span class="dt">mapping =</span> <span class="st">"org.Hs.eg.db"</span>,</a>
<a class="sourceLine" id="cb10-11" data-line-number="11">                                                              <span class="dt">geneID =</span> <span class="st">"symbol"</span>)</a>
<a class="sourceLine" id="cb10-12" data-line-number="12"></a>
<a class="sourceLine" id="cb10-13" data-line-number="13"></a>
<a class="sourceLine" id="cb10-14" data-line-number="14">DT<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/DT/man/datatable.html">datatable</a></span>(topgoDE_macrophage_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb10-15" data-line-number="15">DT<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/DT/man/datatable.html">datatable</a></span>(topgoDE_macrophage_SL1344_vs_naive)</a>
<a class="sourceLine" id="cb10-16" data-line-number="16"></a>
<a class="sourceLine" id="cb10-17" data-line-number="17"></a>
<a class="sourceLine" id="cb10-18" data-line-number="18"><span class="co"># ## with goseq, from ideal</span></a>
<a class="sourceLine" id="cb10-19" data-line-number="19"><span class="co"># </span></a>
<a class="sourceLine" id="cb10-20" data-line-number="20"><span class="co"># goseqDE_macro &lt;- ideal::goseqTable(</span></a>
<a class="sourceLine" id="cb10-21" data-line-number="21"><span class="co">#   de.genes = substr(rownames(de_symbols_IFNg_vs_naive),1,15),</span></a>
<a class="sourceLine" id="cb10-22" data-line-number="22"><span class="co">#   assayed.genes = substr(rownames(dds_macrophage),1,15)</span></a>
<a class="sourceLine" id="cb10-23" data-line-number="23"><span class="co"># )</span></a>
<a class="sourceLine" id="cb10-24" data-line-number="24"><span class="co"># DT::datatable(goseqDE_macro)</span></a></code></pre></div>
<p>Using other alternative procedures to similarly derive enriched functions/processes…</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(clusterProfiler)</a>
<a class="sourceLine" id="cb11-2" data-line-number="2"></a>
<a class="sourceLine" id="cb11-3" data-line-number="3">ego_IFNg_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/enrichGO.html">enrichGO</a></span>(<span class="dt">gene =</span> de_symbols_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb11-4" data-line-number="4">                <span class="dt">universe      =</span> bg_ids,</a>
<a class="sourceLine" id="cb11-5" data-line-number="5">                <span class="dt">keyType =</span> <span class="st">"SYMBOL"</span>,</a>
<a class="sourceLine" id="cb11-6" data-line-number="6">                <span class="dt">OrgDb         =</span> org.Hs.eg.db,</a>
<a class="sourceLine" id="cb11-7" data-line-number="7">                <span class="dt">ont           =</span> <span class="st">"BP"</span>,</a>
<a class="sourceLine" id="cb11-8" data-line-number="8">                <span class="dt">pAdjustMethod =</span> <span class="st">"BH"</span>,</a>
<a class="sourceLine" id="cb11-9" data-line-number="9">                <span class="dt">pvalueCutoff  =</span> <span class="fl">0.01</span>,</a>
<a class="sourceLine" id="cb11-10" data-line-number="10">                <span class="dt">qvalueCutoff  =</span> <span class="fl">0.05</span>,</a>
<a class="sourceLine" id="cb11-11" data-line-number="11">                <span class="dt">readable      =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb11-12" data-line-number="12"></a>
<a class="sourceLine" id="cb11-13" data-line-number="13">ego_SL1344_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/enrichGO.html">enrichGO</a></span>(<span class="dt">gene =</span> de_symbols_SL1344_vs_naive,</a>
<a class="sourceLine" id="cb11-14" data-line-number="14">                <span class="dt">universe      =</span> bg_ids,</a>
<a class="sourceLine" id="cb11-15" data-line-number="15">                <span class="dt">keyType =</span> <span class="st">"SYMBOL"</span>,</a>
<a class="sourceLine" id="cb11-16" data-line-number="16">                <span class="dt">OrgDb         =</span> org.Hs.eg.db,</a>
<a class="sourceLine" id="cb11-17" data-line-number="17">                <span class="dt">ont           =</span> <span class="st">"BP"</span>,</a>
<a class="sourceLine" id="cb11-18" data-line-number="18">                <span class="dt">pAdjustMethod =</span> <span class="st">"BH"</span>,</a>
<a class="sourceLine" id="cb11-19" data-line-number="19">                <span class="dt">pvalueCutoff  =</span> <span class="fl">0.01</span>,</a>
<a class="sourceLine" id="cb11-20" data-line-number="20">                <span class="dt">qvalueCutoff  =</span> <span class="fl">0.05</span>,</a>
<a class="sourceLine" id="cb11-21" data-line-number="21">                <span class="dt">readable      =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb11-22" data-line-number="22"></a>
<a class="sourceLine" id="cb11-23" data-line-number="23"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(ego_SL1344_vs_naive)</a>
<a class="sourceLine" id="cb11-24" data-line-number="24"></a>
<a class="sourceLine" id="cb11-25" data-line-number="25"><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/reexports.html">emapplot</a></span>(ego_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb11-26" data-line-number="26"><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/reexports.html">emapplot</a></span>(ego_SL1344_vs_naive)</a>
<a class="sourceLine" id="cb11-27" data-line-number="27"></a>
<a class="sourceLine" id="cb11-28" data-line-number="28"><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/reexports.html">cnetplot</a></span>(ego_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb11-29" data-line-number="29"><span class="kw"><a href="https://rdrr.io/pkg/clusterProfiler/man/reexports.html">cnetplot</a></span>(ego_SL1344_vs_naive)</a></code></pre></div>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h1>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span>()</a></code></pre></div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-Alasoo2018">
<p>Alasoo, Kaur, Julia Rodrigues, Subhankar Mukhopadhyay, Andrew J. Knights, Alice L. Mann, Kousik Kundu, Christine Hale, Gordon Dougan, and Daniel J. Gaffney. 2018. “Shared genetic effects on chromatin and gene expression indicate a role for enhancer priming in immune response.” <em>Nature Genetics</em> 50 (3). Springer US: 424–31. <a href="https://doi.org/10.1038/s41588-018-0046-7" class="uri">https://doi.org/10.1038/s41588-018-0046-7</a>.</p>
</div>
<div id="ref-Love2014">
<p>Love, Michael I, Wolfgang Huber, and Simon Anders. 2014. “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.” <em>Genome Biology</em> 15 (12): 550. <a href="https://doi.org/10.1186/s13059-014-0550-8" class="uri">https://doi.org/10.1186/s13059-014-0550-8</a>.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">

        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#preliminaries">Preliminaries</a></li>
      <li>
<a href="#input-required-for-genetonic">Input required for <code>GeneTonic</code></a><ul class="nav nav-pills nav-stacked">
<li><a href="#obtaining-the-deseqdataset">Obtaining the <code>DESeqDataSet</code></a></li>
      <li><a href="#extracting-the-relevant-deseqresults">Extracting the relevant <code>DESeqResults</code></a></li>
      </ul>
</li>
      <li><a href="#session-info">Session Info</a></li>
      <li><a href="#references">References</a></li>
      </ul>
</div>
      </div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://federicomarini.github.io">Federico Marini</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
