<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>The `GeneTonic` User's Guide
 • GeneTonic</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="The `GeneTonic` User's Guide
">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">GeneTonic</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.6</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/GeneTonic_manual.html">The `GeneTonic` User's Guide
</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/federicomarini/GeneTonic">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><link href="GeneTonic_manual_files/font-awesome-5.3.1/css/all.min.css" rel="stylesheet">
<link href="GeneTonic_manual_files/font-awesome-5.3.1/css/v4-shims.min.css" rel="stylesheet">
<script src="GeneTonic_manual_files/htmlwidgets-1.5.1/htmlwidgets.js"></script><script src="GeneTonic_manual_files/plotly-binding-4.9.2/plotly.js"></script><script src="GeneTonic_manual_files/typedarray-0.1/typedarray.min.js"></script><script src="GeneTonic_manual_files/jquery-1.11.3/jquery.min.js"></script><link href="GeneTonic_manual_files/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet">
<script src="GeneTonic_manual_files/crosstalk-1.0.0/js/crosstalk.min.js"></script><link href="GeneTonic_manual_files/plotly-htmlwidgets-css-1.52.2/plotly-htmlwidgets.css" rel="stylesheet">
<script src="GeneTonic_manual_files/plotly-main-1.52.2/plotly-latest.min.js"></script><link href="GeneTonic_manual_files/vis-4.20.1/vis.css" rel="stylesheet">
<script src="GeneTonic_manual_files/vis-4.20.1/vis.min.js"></script><script src="GeneTonic_manual_files/visNetwork-binding-2.0.9/visNetwork.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>The <code>GeneTonic</code> User’s Guide</h1>
                        <h4 class="author">Federico Marini</h4>
            <address class="author_afil">
      Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz<br><a class="author_email" href="mailto:#"></a><a href="mailto:marinif@uni-mainz.de">marinif@uni-mainz.de</a>
      </address>
                  
            <h4 class="date">12 February 2020</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/federicomarini/GeneTonic/blob/master/vignettes/GeneTonic_manual.Rmd"><code>vignettes/GeneTonic_manual.Rmd</code></a></small>
      <div class="hidden name"><code>GeneTonic_manual.Rmd</code></div>

    </div>

    
    
<p><strong>Compiled date</strong>: 2020-02-12</p>
<p><strong>Last edited</strong>: 2020-01-23</p>
<p><strong>License</strong>: MIT + file LICENSE</p>
<hr>
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>This vignette describes how to use the <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> package for analyzing and integrating the results from Differential Expression analysis and functional enrichment analysis.</p>
<p>This package provides a Shiny application that aims to combine at different levels the existing pieces of the transcriptome data and results, in a way that makes it easier to generate insightful observations and hypothesis - combining the benefits of interactivity and reproducibility, e.g. by capturing the features and gene sets of interest highlighted during the live session, and creating an HTML report as an artifact where text, code, and output coexist.</p>
<p>In order to use <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> in your workflow, the following inputs are required:</p>
<ul>
<li>
<code>dds</code>, a <code>DESeqDataSet</code> containing the expression matrix;</li>
<li>
<code>res_de</code>, a <code>DESeqResults</code>, i.e. a <code>DataFrame</code> storing the results of the differentially expression analysis;</li>
<li>
<code>res_enrich</code>, a <code>data.frame</code> with the results of functional enrichment analysis;</li>
<li>
<code>annotation_obj</code>, a <code>data.frame</code> with the correspondence between identifiers for the features under inspection in the <code>dds</code> object.</li>
</ul>
<p>This workflow has mainly been tested with expression matrices with Ensembl identifiers, which are consistent and unambiguous across different releases, and results from functional enrichment analysis originating from ORA (OverRepresentation Analysis) methods, on the Gene Ontology signature database. It may need to be slightly adjusted to work with other formats and sources, but the core functionality will remain available.</p>
<p>In the remainder of this vignette, we will illustrate the main features of <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> on a publicly available dataset from Alasoo, et al. “Shared genetic effects on chromatin and gene expression indicate a role for enhancer priming in immune response”, published in Nature Genetics, January 2018 <span class="citation">(Alasoo et al. 2018)</span> <a href="https://doi.org/10.1038/s41588-018-0046-7">doi:10.1038/s41588-018-0046-7</a>.</p>
<p>The data is made available via the <em><a href="https://bioconductor.org/packages/3.11/macrophage">macrophage</a></em> Bioconductor package, which contains the files output from the Salmon quantification (version 0.12.0, with Gencode v29 reference), as well as the values summarized at the gene level, which we will use to exemplify.</p>
<p>In the <code>macrophage</code> experimental setting, the samples are available from 6 different donors, in 4 different conditions (naive, treated with Interferon gamma, with SL1344, or with a combination of Interferon gamma and SL1344). We will restrict our attention on the comparison between Interferon gamma treated samples vs naive samples.</p>
</div>
<div id="gettingstarted" class="section level1">
<h1 class="hasAnchor">
<a href="#gettingstarted" class="anchor"></a>Getting started</h1>
<p>To install this package, start R and enter:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="cf">if</span> (<span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/ns-load.html">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</a>
<a class="sourceLine" id="cb1-2" data-line-number="2">  <span class="kw"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span>(<span class="st">"BiocManager"</span>)</a>
<a class="sourceLine" id="cb1-3" data-line-number="3"></a>
<a class="sourceLine" id="cb1-4" data-line-number="4">BiocManager<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="st">"GeneTonic"</span>)</a></code></pre></div>
<p>Once installed, the package can be loaded and attached to your current workspace as follows:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"GeneTonic"</span>)</a></code></pre></div>
<p>If you have all four input elements ready, you can launch the <code><a href="../reference/GeneTonic.html">GeneTonic()</a></code> app by running:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1"><span class="kw"><a href="../reference/GeneTonic.html">GeneTonic</a></span>(<span class="dt">dds =</span> dds_object,</a>
<a class="sourceLine" id="cb3-2" data-line-number="2">          <span class="dt">res_de =</span> res_de_object,</a>
<a class="sourceLine" id="cb3-3" data-line-number="3">          <span class="dt">res_enrich =</span> res_enrich_object,</a>
<a class="sourceLine" id="cb3-4" data-line-number="4">          <span class="dt">annotation_obj =</span> annotation_object,</a>
<a class="sourceLine" id="cb3-5" data-line-number="5">          <span class="dt">project_id =</span> <span class="st">"myFirstGeneTonic"</span>)</a></code></pre></div>
<p>In this vignette, we showcase the functionality of <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> using the gene-level summarized version of the <code>macrophage</code> dataset. If you want to dive in and start playing with the app immediately, you can simply run:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/example.html">example</a></span>(<span class="st">"GeneTonic"</span>, <span class="dt">ask =</span> <span class="ot">FALSE</span>)</a></code></pre></div>
<p>Otherwise, you can follow the next chunks of code to generate the required input objects, step by step. In case some formatting requirements are expected, these will be specified in the text/comments, and they will be checked internally upon launching the <code>GeneTonic</code> app.</p>
<p>For running <code>GeneTonic</code>, you will need the four components mentioned above as parameters.</p>
<ol style="list-style-type: decimal">
<li>
<code>dds</code>: First you will need a <code>DESeqDataSet</code>, which is the main component in the <code>DESeq2</code> framework and extends the widely adopted <code>SummarizedExperiment</code> class. This object will store the information related to the expression matrix.</li>
</ol>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"macrophage"</span>)</a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"DESeq2"</span>)</a>
<a class="sourceLine" id="cb5-3" data-line-number="3"></a>
<a class="sourceLine" id="cb5-4" data-line-number="4"><span class="kw"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"gse"</span>, <span class="dt">package =</span> <span class="st">"macrophage"</span>)</a>
<a class="sourceLine" id="cb5-5" data-line-number="5"></a>
<a class="sourceLine" id="cb5-6" data-line-number="6">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/DESeqDataSet.html">DESeqDataSet</a></span>(gse, <span class="dt">design =</span> <span class="op">~</span>line <span class="op">+</span><span class="st"> </span>condition)</a>
<a class="sourceLine" id="cb5-7" data-line-number="7"><span class="co"># changing the ids to Ensembl instead of the Gencode used in the object</span></a>
<a class="sourceLine" id="cb5-8" data-line-number="8"><span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage) &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/substr.html">substr</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage), <span class="dv">1</span>, <span class="dv">15</span>)</a>
<a class="sourceLine" id="cb5-9" data-line-number="9">dds_macrophage</a>
<a class="sourceLine" id="cb5-10" data-line-number="10"><span class="co">#&gt; class: DESeqDataSet </span></a>
<a class="sourceLine" id="cb5-11" data-line-number="11"><span class="co">#&gt; dim: 58294 24 </span></a>
<a class="sourceLine" id="cb5-12" data-line-number="12"><span class="co">#&gt; metadata(7): tximetaInfo quantInfo ... txdbInfo version</span></a>
<a class="sourceLine" id="cb5-13" data-line-number="13"><span class="co">#&gt; assays(3): counts abundance avgTxLength</span></a>
<a class="sourceLine" id="cb5-14" data-line-number="14"><span class="co">#&gt; rownames(58294): ENSG00000000003 ENSG00000000005 ... ENSG00000285993 ENSG00000285994</span></a>
<a class="sourceLine" id="cb5-15" data-line-number="15"><span class="co">#&gt; rowData names(2): gene_id SYMBOL</span></a>
<a class="sourceLine" id="cb5-16" data-line-number="16"><span class="co">#&gt; colnames(24): SAMEA103885102 SAMEA103885347 ... SAMEA103885308 SAMEA103884949</span></a>
<a class="sourceLine" id="cb5-17" data-line-number="17"><span class="co">#&gt; colData names(15): names sample_id ... condition line</span></a></code></pre></div>
<ol start="2" style="list-style-type: decimal">
<li>
<p><code>res_de</code>: Next you are going to need the results of Differential Expression analysis, computed on the <code>dds_macrophage</code> object you just obtained.</p>
<p>The expected format is a <code>DESeqResults</code> object. Here we are going to contrast two different conditions, <code>IFNg</code> and <code>naive</code>, while controlling for the cell line of origin (which has 6 levels, namely naive, IFNg, IFNg_SL1344, and SL1344).</p>
<p>We start by filtering lowly expressed features (at least 10 counts in at least 6 samples - 6 being the size of the smallest group). Then, we test against a null hypothesis of a log2FoldChange of 1 (instead of the default value of 0), in order to specify that we want to call DE genes with a consistent <em>and</em> robust expression change. Finally, we add the gene symbol to the output <code>DataFrame</code>.</p>
</li>
</ol>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">keep &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/colSums.html">rowSums</a></span>(<span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/counts.html">counts</a></span>(dds_macrophage) <span class="op">&gt;=</span><span class="st"> </span><span class="dv">10</span>) <span class="op">&gt;=</span><span class="st"> </span><span class="dv">6</span></a>
<a class="sourceLine" id="cb6-2" data-line-number="2">dds_macrophage &lt;-<span class="st"> </span>dds_macrophage[keep, ]</a>
<a class="sourceLine" id="cb6-3" data-line-number="3">dds_macrophage</a>
<a class="sourceLine" id="cb6-4" data-line-number="4"><span class="co">#&gt; class: DESeqDataSet </span></a>
<a class="sourceLine" id="cb6-5" data-line-number="5"><span class="co">#&gt; dim: 17806 24 </span></a>
<a class="sourceLine" id="cb6-6" data-line-number="6"><span class="co">#&gt; metadata(7): tximetaInfo quantInfo ... txdbInfo version</span></a>
<a class="sourceLine" id="cb6-7" data-line-number="7"><span class="co">#&gt; assays(3): counts abundance avgTxLength</span></a>
<a class="sourceLine" id="cb6-8" data-line-number="8"><span class="co">#&gt; rownames(17806): ENSG00000000003 ENSG00000000419 ... ENSG00000285982 ENSG00000285994</span></a>
<a class="sourceLine" id="cb6-9" data-line-number="9"><span class="co">#&gt; rowData names(2): gene_id SYMBOL</span></a>
<a class="sourceLine" id="cb6-10" data-line-number="10"><span class="co">#&gt; colnames(24): SAMEA103885102 SAMEA103885347 ... SAMEA103885308 SAMEA103884949</span></a>
<a class="sourceLine" id="cb6-11" data-line-number="11"><span class="co">#&gt; colData names(15): names sample_id ... condition line</span></a></code></pre></div>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/DESeq.html">DESeq</a></span>(dds_macrophage)</a>
<a class="sourceLine" id="cb7-2" data-line-number="2"><span class="co"># vst_macrophage &lt;- vst(dds_macrophage)</span></a>
<a class="sourceLine" id="cb7-3" data-line-number="3">res_macrophage_IFNg_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/results.html">results</a></span>(dds_macrophage,</a>
<a class="sourceLine" id="cb7-4" data-line-number="4">                                        <span class="dt">contrast =</span> <span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"condition"</span>, <span class="st">"IFNg"</span>, <span class="st">"naive"</span>),</a>
<a class="sourceLine" id="cb7-5" data-line-number="5">                                        <span class="dt">lfcThreshold =</span> <span class="dv">1</span>, <span class="dt">alpha =</span> <span class="fl">0.05</span>)</a>
<a class="sourceLine" id="cb7-6" data-line-number="6">res_macrophage_IFNg_vs_naive<span class="op">$</span>SYMBOL &lt;-<span class="st"> </span><span class="kw">rowData</span>(dds_macrophage)<span class="op">$</span>SYMBOL</a></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1">## To speed up the operations in the vignette, we can also load this object directly</a>
<a class="sourceLine" id="cb8-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"res_de_macrophage"</span>)</a>
<a class="sourceLine" id="cb8-3" data-line-number="3"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(res_macrophage_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb8-4" data-line-number="4"><span class="co">#&gt; log2 fold change (MLE): condition IFNg vs naive </span></a>
<a class="sourceLine" id="cb8-5" data-line-number="5"><span class="co">#&gt; Wald test p-value: condition IFNg vs naive </span></a>
<a class="sourceLine" id="cb8-6" data-line-number="6"><span class="co">#&gt; DataFrame with 6 rows and 7 columns</span></a>
<a class="sourceLine" id="cb8-7" data-line-number="7"><span class="co">#&gt;                  baseMean log2FoldChange     lfcSE      stat      pvalue        padj      SYMBOL</span></a>
<a class="sourceLine" id="cb8-8" data-line-number="8"><span class="co">#&gt;                 &lt;numeric&gt;      &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;   &lt;numeric&gt;   &lt;numeric&gt; &lt;character&gt;</span></a>
<a class="sourceLine" id="cb8-9" data-line-number="9"><span class="co">#&gt; ENSG00000000003   171.571     -0.2822450 0.3005710   0.00000 1.00000e+00 1.00000e+00      TSPAN6</span></a>
<a class="sourceLine" id="cb8-10" data-line-number="10"><span class="co">#&gt; ENSG00000000419   967.751      0.0391223 0.0859708   0.00000 1.00000e+00 1.00000e+00        DPM1</span></a>
<a class="sourceLine" id="cb8-11" data-line-number="11"><span class="co">#&gt; ENSG00000000457   682.433      1.2846179 0.1969067   1.44545 1.48333e-01 1.00000e+00       SCYL3</span></a>
<a class="sourceLine" id="cb8-12" data-line-number="12"><span class="co">#&gt; ENSG00000000460   262.963     -1.4718762 0.2186916  -2.15772 3.09493e-02 4.09728e-01    C1orf112</span></a>
<a class="sourceLine" id="cb8-13" data-line-number="13"><span class="co">#&gt; ENSG00000000938  2660.102      0.6754781 0.2360530   0.00000 1.00000e+00 1.00000e+00         FGR</span></a>
<a class="sourceLine" id="cb8-14" data-line-number="14"><span class="co">#&gt; ENSG00000000971  3052.123      4.9871107 0.4964284   8.03159 9.62155e-16 1.37057e-13         CFH</span></a></code></pre></div>
<ol start="3" style="list-style-type: decimal">
<li>
<p><code>res_enrich</code>: With the DE results of the previous step, we are going to extract the vector of DE genes (via <code>deseqresult2df</code>), as well as the list of genes to be used as background, and feed these two objects into a function that computes the functional enrichment of the DE genes.</p>
<p>We are going to use the ORA method implemented in <em><a href="https://bioconductor.org/packages/3.11/topGO">topGO</a></em>, wrapped in the function <code>topGOtable</code> available in the <em><a href="https://bioconductor.org/packages/3.11/pcaExplorer">pcaExplorer</a></em> package, which by default uses the <code>BP</code> ontology and the <code>elim</code> method to decorrelate the GO graph structure and deliver less redundant functional categories. You can also use as an alternative the <code>enrichGO</code> function from <em><a href="https://bioconductor.org/packages/3.11/clusterProfiler">clusterProfiler</a></em>.</p>
</li>
</ol>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"AnnotationDbi"</span>)</a>
<a class="sourceLine" id="cb9-2" data-line-number="2">de_symbols_IFNg_vs_naive &lt;-<span class="st"> </span><span class="kw"><a href="../reference/deseqresult2df.html">deseqresult2df</a></span>(res_macrophage_IFNg_vs_naive, <span class="dt">FDR =</span> <span class="fl">0.05</span>)<span class="op">$</span>SYMBOL</a>
<a class="sourceLine" id="cb9-3" data-line-number="3">bg_ids &lt;-<span class="st"> </span><span class="kw">rowData</span>(dds_macrophage)<span class="op">$</span>SYMBOL[<span class="kw"><a href="https://rdrr.io/r/base/colSums.html">rowSums</a></span>(<span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/counts.html">counts</a></span>(dds_macrophage)) <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>]</a></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"topGO"</span>)</a>
<a class="sourceLine" id="cb10-2" data-line-number="2">topgoDE_macrophage_IFNg_vs_naive &lt;-</a>
<a class="sourceLine" id="cb10-3" data-line-number="3"><span class="st">  </span>pcaExplorer<span class="op">::</span><span class="kw">topGOtable</span>(de_symbols_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb10-4" data-line-number="4">                          bg_ids,</a>
<a class="sourceLine" id="cb10-5" data-line-number="5">                          <span class="dt">ontology =</span> <span class="st">"BP"</span>,</a>
<a class="sourceLine" id="cb10-6" data-line-number="6">                          <span class="dt">mapping =</span> <span class="st">"org.Hs.eg.db"</span>,</a>
<a class="sourceLine" id="cb10-7" data-line-number="7">                          <span class="dt">geneID =</span> <span class="st">"symbol"</span>,</a>
<a class="sourceLine" id="cb10-8" data-line-number="8">                          <span class="dt">topTablerows =</span> <span class="dv">500</span>)</a></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" data-line-number="1">## To speed up the operations in the vignette, we also load this object directly</a>
<a class="sourceLine" id="cb11-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="st">"res_enrich_macrophage"</span>)</a>
<a class="sourceLine" id="cb11-3" data-line-number="3"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(topgoDE_macrophage_IFNg_vs_naive, <span class="dv">2</span>)</a>
<a class="sourceLine" id="cb11-4" data-line-number="4"><span class="co">#&gt;        GO.ID                                        Term Annotated Significant Expected</span></a>
<a class="sourceLine" id="cb11-5" data-line-number="5"><span class="co">#&gt; 1 GO:0002250                    adaptive immune response       323          89    17.93</span></a>
<a class="sourceLine" id="cb11-6" data-line-number="6"><span class="co">#&gt; 2 GO:0060333 interferon-gamma-mediated signaling pathway        82          34     4.55</span></a>
<a class="sourceLine" id="cb11-7" data-line-number="7"><span class="co">#&gt;   Rank in p.value_classic p.value_elim p.value_classic</span></a>
<a class="sourceLine" id="cb11-8" data-line-number="8"><span class="co">#&gt; 1                       5      9.2e-23              NA</span></a>
<a class="sourceLine" id="cb11-9" data-line-number="9"><span class="co">#&gt; 2                      18      1.2e-20         9.9e-22</span></a>
<a class="sourceLine" id="cb11-10" data-line-number="10"><span class="co">#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             genes</span></a>
<a class="sourceLine" id="cb11-11" data-line-number="11"><span class="co">#&gt; 1 B2M,BCL6,BTN3A1,BTN3A2,BTN3A3,C1QB,C1R,C1RL,C1S,C2,C3,C4A,C4B,CD1A,CD1C,CD274,CD28,CD40,CD7,CD74,CD80,CEACAM1,CLEC10A,CLEC6A,CTLA4,CTSS,ERAP2,EXO1,FGL1,FGL2,GCNT3,GPR183,HLA-A,HLA-B,HLA-C,HLA-DMA,HLA-DMB,HLA-DOA,HLA-DOB,HLA-DPA1,HLA-DPB1,HLA-DQA1,HLA-DQB1,HLA-DQB2,HLA-DRA,HLA-DRB1,HLA-DRB5,HLA-E,HLA-F,HLA-G,ICAM1,IL12A,IL12RB1,IL18BP,IL27,IRF1,IRF7,ITK,JAK2,JAK3,KLRK1,LAG3,LAMP3,LILRA1,LILRB3,LYN,MCOLN2,P2RX7,PDCD1,PDCD1LG2,RELB,RIPK2,RNF19B,RSAD2,SERPING1,SIT1,SLAMF1,SLAMF6,SLAMF7,SLC11A1,TAP1,TAP2,TBX21,TLR8,TNFRSF11A,TNFRSF21,TNFSF13B,TNFSF18,ZC3H12A</span></a>
<a class="sourceLine" id="cb11-12" data-line-number="12"><span class="co">#&gt; 2                                                                                                                                                                                                                                                                                                                                                       B2M,CAMK2D,CIITA,GBP1,GBP2,HLA-A,HLA-B,HLA-C,HLA-DPA1,HLA-DPB1,HLA-DQA1,HLA-DQB1,HLA-DQB2,HLA-DRA,HLA-DRB1,HLA-DRB5,HLA-E,HLA-F,HLA-G,ICAM1,IRF1,IRF7,JAK2,MT2A,NLRC5,NMI,OAS2,PARP14,PML,SOCS1,STAT1,TRIM22,TRIM31,VCAM1</span></a></code></pre></div>
<p>Since in this step the object can be created from different methods, but the <code>GeneTonic</code> main functions require some specific columns to be present, we have created some functions to convert the output of the upstream tools into a format compatible with <code>GeneTonic</code>. The functions <code><a href="../reference/shake_enrichResult.html">shake_enrichResult()</a></code> and <code><a href="../reference/shake_topGOtableResult.html">shake_topGOtableResult()</a></code> serve this purpose - if your favorite tool might be different, you can write your own conversion function and contribute it to <code>GeneTonic</code>.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1">res_enrich_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="../reference/shake_topGOtableResult.html">shake_topGOtableResult</a></span>(topgoDE_macrophage_IFNg_vs_naive)</a>
<a class="sourceLine" id="cb12-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-toTable.html">colnames</a></span>(res_enrich_macrophage)</a>
<a class="sourceLine" id="cb12-3" data-line-number="3"><span class="co">#&gt; [1] "gs_id"          "gs_description" "gs_pvalue"      "gs_genes"       "gs_de_count"   </span></a>
<a class="sourceLine" id="cb12-4" data-line-number="4"><span class="co">#&gt; [6] "gs_bg_count"    "Expected"</span></a></code></pre></div>
<ol start="4" style="list-style-type: decimal">
<li>
<code>annotation_obj</code>: Last object required is the annotation data frame, i.e. a simple data frame composed at least of two columns, <code>gene_id</code>, with a set of unambiguous identifiers (e.g. ENSEMBL ids) corresponding to the row names of the <code>dds</code> object, and <code>gene_name</code>, containing e.g. HGNC-based gene symbols. This object can be constructed via the <code>org.eg.XX.db</code> packages, e.g. with convenience functions such as <code>pcaExplorer::get_annotation_orgdb()</code>. Here we display the general way of obtaining this object.</li>
</ol>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"org.Hs.eg.db"</span>)</a>
<a class="sourceLine" id="cb13-2" data-line-number="2">anno_df &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/data.frame.html">data.frame</a></span>(</a>
<a class="sourceLine" id="cb13-3" data-line-number="3">  <span class="dt">gene_id =</span> <span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage),</a>
<a class="sourceLine" id="cb13-4" data-line-number="4">  <span class="dt">gene_name =</span> <span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/AnnotationDb-class.html">mapIds</a></span>(org.Hs.eg.db, <span class="dt">keys =</span> <span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage), <span class="dt">column =</span> <span class="st">"SYMBOL"</span>, <span class="dt">keytype =</span> <span class="st">"ENSEMBL"</span>),</a>
<a class="sourceLine" id="cb13-5" data-line-number="5">  <span class="dt">stringsAsFactors =</span> <span class="ot">FALSE</span>,</a>
<a class="sourceLine" id="cb13-6" data-line-number="6">  <span class="dt">row.names =</span> <span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(dds_macrophage)</a>
<a class="sourceLine" id="cb13-7" data-line-number="7">)</a>
<a class="sourceLine" id="cb13-8" data-line-number="8">## alternatively:</a>
<a class="sourceLine" id="cb13-9" data-line-number="9"><span class="co"># anno_df &lt;- pcaExplorer::get_annotation_orgdb(dds_macrophage, "org.Hs.eg.db", "ENSEMBL")</span></a></code></pre></div>
<p>For some operations in the main app and in the single functions, sometimes it is required to have aggregated scores for the <code>res_enrich</code> data frame.</p>
<p>We can do so by calling <code><a href="../reference/get_aggrscores.html">get_aggrscores()</a></code> on the combination of objects we just generated. This adds two columns to the provided <code>res_enrich</code> object, <code>z_score</code> and <code>aggr_score</code>, which summarize at the gene set level the effect (log2FoldChange) of the differentially expressed genes which are its members. In particular, the z score attempts to determine the “direction” of change, computed as <span class="math inline">\(z = \frac{(upgenes - downgenes)}{\sqrt{(upgenes + downgenes)}}\)</span>, regardless of the effect size of the single members of the gene set.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" data-line-number="1">res_enrich_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="../reference/get_aggrscores.html">get_aggrscores</a></span>(<span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb14-2" data-line-number="2">                                        <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb14-3" data-line-number="3">                                        <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb14-4" data-line-number="4">                                        <span class="dt">aggrfun =</span> mean)</a></code></pre></div>
<div id="all-set" class="section level2">
<h2 class="hasAnchor">
<a href="#all-set" class="anchor"></a>All set!</h2>
<p>Once you have the four objects specified above, you can set the <code>project_id</code> to be a character string at wish, and you can just launch the <code><a href="../reference/GeneTonic.html">GeneTonic()</a></code> app:</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1"><span class="kw"><a href="../reference/GeneTonic.html">GeneTonic</a></span>(<span class="dt">dds =</span> dds_macrophage,</a>
<a class="sourceLine" id="cb15-2" data-line-number="2">          <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb15-3" data-line-number="3">          <span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb15-4" data-line-number="4">          <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb15-5" data-line-number="5">          <span class="dt">project_id =</span> <span class="st">"GT1"</span>)</a></code></pre></div>
</div>
</div>
<div id="userinterface" class="section level1">
<h1 class="hasAnchor">
<a href="#userinterface" class="anchor"></a>Description of the <code>GeneTonic</code> user interface</h1>
<p>The <code>GeneTonic</code> app is built with <em><a href="https://CRAN.R-project.org/package=shiny">shiny</a></em>, and its layout is built around the components of <em><a href="https://CRAN.R-project.org/package=bs4Dash">bs4Dash</a></em>, a modern looking dashboard based on Bootstrap 4.</p>
<div id="header-navbar" class="section level2">
<h2 class="hasAnchor">
<a href="#header-navbar" class="anchor"></a>Header (navbar)</h2>
<p>The dashboard navbar (as it is called in <code>bs4Dash</code>) contains two dropdown menus, which can be expanded by clicking on the respective icons (<i class="fa fa-book"></i> and <i class="fa fa-info"></i>). These provide additional buttons to do one the following:</p>
<ul>
<li>
<i class="fa fa-book"></i> inspect the <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> vignette (either the one installed in bundle with the package, or the online version)</li>
<li>
<i class="fa fa-question-circle"></i> read some first help information on <code>GeneTonic</code> (What the package does in brief, what is required to run properly)</li>
<li>
<i class="fa fa-info-circle"></i> display information about the current session (via <code><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo()</a></code>)</li>
<li>
<i class="fa fa-heart"></i> show some general information on <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em>, like where to find its development version (and contribute to that), or how to cite this tool.</li>
</ul>
<p>Additionally, one fundamental button is present in the navbar: <button class="btn btn-default action-button ml-5" id="bookmarker" style="color: #ffffff; background-color: #ac0000; border-color: #ffffff" type="button">
<i class="fa fa-heart"></i>
Bookmark
</button>. This serves the purpose of bookmarking genes and gene sets of interest while exploring your data in an interactive session. This functionality will be exploited in the <i class="fa fa-bookmark"></i> section of the app, where a complete HTML report can be generated as a reproducible artifact.</p>
</div>
<div id="sidebar" class="section level2">
<h2 class="hasAnchor">
<a href="#sidebar" class="anchor"></a>Sidebar</h2>
<p>On the left side of the app, by clicking on the <i class="fa fa-bars"></i> icon (or if the app is viewed in full screen, by simply moving the mouse over to the left side), the sidebar menu is triggered. This constitutes the main way to access the different tabs of <code><a href="../reference/GeneTonic.html">GeneTonic()</a></code>, which will be explained in more detail in the next section (@ref(functionality)).</p>
</div>
<div id="controlbar" class="section level2">
<h2 class="hasAnchor">
<a href="#controlbar" class="anchor"></a>Controlbar</h2>
<p>The controlbar is triggered by clicking on the cogs <i class="fa fa-cogs"></i> icon on the upper right part of the navbar. There you can find widgets that control the appearance of output (plots, tables, graphs) in multiple tabs of the main application, such as for example the number of gene sets to focus your attention on.</p>
</div>
<div id="body" class="section level2">
<h2 class="hasAnchor">
<a href="#body" class="anchor"></a>Body</h2>
<p>The main body of the <code>GeneTonic</code> application is structured in tabs that are activated by clicking on the respective icons or text in the sidebar.</p>
<p>While the Welcome tab might be self-explanatory, the functionality of each tab can go in depth, and new users can benefit of the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button to trigger an interactive tour of <code>GeneTonic</code>, which allows users to learn the basic usage mechanisms by doing. During a tour, the highlighted elements respond to the user’s actions, while the rest of the UI is kept shaded. Tours can be interrupted anytime by clicking outside of the focused window, and arrows (left, right) can be used as well to navigate between steps. The tour functionality is provided by the beautiful <em><a href="https://CRAN.R-project.org/package=rintrojs">rintrojs</a></em> package.</p>
</div>
</div>
<div id="functionality" class="section level1">
<h1 class="hasAnchor">
<a href="#functionality" class="anchor"></a>The <code>GeneTonic</code> functionality</h1>
<p>The main <code>GeneTonic</code> app is structured in different panels, and each of these is presented in detail in the subsequent sections.</p>
<div id="the-welcome-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-welcome-panel" class="anchor"></a>The Welcome panel</h2>
<p>This panel in intended to give an overview on the different ingredients you just provided for launching <code>GeneTonic</code>.</p>
<p>Collapsible elements, colored each by the typology of data it refers to, contain the underlying tabular information, and a value box provides a quick summary on the dimensions of the input parameters.</p>
<p>The tables are rendered with <code><a href="https://rdrr.io/pkg/DT/man/datatable.html">DT::datatable</a></code>, which provides a practical interface for exploring, sorting, and filtering. This framework is also extendable, for example as we do for the <code>log2FoldChange</code> column in the <code>res_de</code> object, where the values are also visually encoded as diverging bars of different color based on the expression change direction.</p>
</div>
<div id="the-gene-geneset-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-gene-geneset-panel" class="anchor"></a>The Gene-Geneset panel</h2>
<p>This panel has as its main component an interactive bipartite graph illustrating the relationship between genesets and genes that belong to them. One of the relevant parameters for this output is the number of gene sets to be displayed, adjustable from the widget in the control panel on the right.</p>
<p>Such a representation is particularly useful to see overlap between terms, and grasp the way how different genes may play a concerting role in different processes.</p>
<p>The color and shape of the nodes depend on the entity they are related to:</p>
<ul>
<li>genes are shown as ellipses, colored with a palette diverging from a neutral color to red (log2FC &gt; 0) or to blue (log2FC &lt; 0)</li>
<li>gene sets are enclosed in gold boxes</li>
</ul>
<p>The interactivity in this graph is essential for zooming, panning, and selecting its nodes. The node selection, in particular, triggers a drill-down exploration, which populates a box on the right side, either with a signature heatmap (for a gene set), or with an expression plot, split by the experimental covariates of interest (for a gene) - this behavior can be controlled by the “Group/color by” selectize widget in the control sidebar. In both cases, some HTML content is also generated, with buttons linking to external databases where to retrieve additional information (e.g. ENSEMBL, NCBI, AmiGO, …).</p>
<p>A demonstration of how to use this panel effectively is provided by the tour, triggered by clicking on the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button in the upper right corner of the tab.</p>
<p><strong>Bookmarking from the Gene-Geneset panel:</strong> When clicking on the Bookmark button while in this tab, it is possible to add to the shortlisted items of interest both genes and gene sets. To do so, while one of the nodes is selected, click on the button with the mouse - or alternatively, click on the Left Ctrl button of your keyboard. A notification will be triggered, informing the status of the bookmarked elements.</p>
</div>
<div id="the-enrichment-map-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-enrichment-map-panel" class="anchor"></a>The Enrichment Map panel</h2>
<p>This panel, similar to the Gene-Geneset tab, has again an interactive graph of a subset of the gene sets (you can control again the number of included sets from the control sidebar). This is also called an enrichment map - it is also possible to generate them in tools such as Cytoscape. In this case, the genesets are directly connected by an edge, where the thickness encodes the degree of similarity (e.g. in terms of overlap) between them; sets below a similarity threshold are not displayed for the sake of better clarity.</p>
<p>The size of the node encodes the information of the number of genes detected as Differentially Expressed (from the column <code>gs_de_count</code>), while the color is representative of the computed Z score for each set. This value is computed as <span class="math inline">\(z = \frac{(upgenes - downgenes)}{\sqrt{(upgenes + downgenes)}}\)</span> and is indicative of the expression changes of the genes assigned to a particular set. Please, keep in mind that methods based on overrepresentation analysis only do not account for the topology that might be behind the set of interacting features (i.e. an upregulated gene involved in inhibition of a pathway would still be counted as a positive element in the equation above).</p>
<p>The <code>z_score</code> and <code>aggr_score</code> (in the simplest form, a mean of the log2FoldChange values for all the genes of a set) can be easily computed with the <code><a href="../reference/get_aggrscores.html">get_aggrscores()</a></code>, which appends these scores for each gene set to the original <code>res_enrich</code> data frame.</p>
<p>The interactive flavor in this tab permits a quick exploration of the signature heatmaps for the genesets of interest by simply clicking on a node; this also triggers the display of some HTML content, if related to a Gene Ontology term.</p>
<p>A demonstration of how to use this panel effectively is provided by the tour, triggered by clicking on the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button in the upper right corner of the tab.</p>
<p><strong>Bookmarking from the Enrichment Map panel:</strong> If the user clicks on the Bookmark button while in this tab, it will add the selected node to the shortlist of genesets, visible later in the Bookmarks panel. The Left Ctrl button of your keyboard can again be used for this purpose, and this will trigger a notification in the lower right part of the screen.</p>
</div>
<div id="the-overview-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-overview-panel" class="anchor"></a>The Overview panel</h2>
<p>This panel presents to the user a series of summary plots, based on the outputs of <code><a href="../reference/enhance_table.html">enhance_table()</a></code> and <code><a href="../reference/gs_volcano.html">gs_volcano()</a></code>, aiming to provide a mean to summarize the most relevant genesets in the <code>res_enrich</code> object. The number of displayed (or labeled, according to each function) genesets can be as usual controlled by the widget in the right sidebar.</p>
<p>We avoid (re)rendering all plots at once by placing them as content of different tabs in a <code>bs4TabCard</code> container - this allows the user to navigate in a more agile way to the desired output elements.</p>
<p>Interactive versions of the plots are returned via calls to <code>ggplotly</code>.</p>
<p>A demonstration of how to use this panel effectively is provided by the tour, triggered by clicking on the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button in the upper right corner of the tab.</p>
</div>
<div id="the-gsviz-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-gsviz-panel" class="anchor"></a>The GSViz panel</h2>
<p>This panel, similarly to the Overview panel, presents a variety of plots visualizing the gene sets - and their mutual relationships, or their behavior across samples. This includes, for example:</p>
<ul>
<li>a genesets by sample heatmap (via <code><a href="../reference/gs_scores.html">gs_scores()</a></code> and <code><a href="../reference/gs_scoresheat.html">gs_scoresheat()</a></code>), with a score for each matrix element corresponding to the sum of the standardized expression values of its components</li>
<li>a genesets-gene Sankey diagram (via <code><a href="../reference/gs_alluvial.html">gs_alluvial()</a></code>, aliased to <code><a href="../reference/gs_alluvial.html">gs_sankey()</a></code>), to represent the N:N membership relationships</li>
<li>a summary heatmap (via <code><a href="../reference/gs_summary_heat.html">gs_summary_heat()</a></code>) to display the memberships and at the same time the expression change of each gene set member</li>
<li>a multi-dimensional scaling (MDS) plot, based on <code><a href="../reference/gs_mds.html">gs_mds()</a></code>, to provide a 2d visualization of the distance among genesets (based on a similarity measure e.g. their overlap or other criteria, such as their semantic similarity)</li>
<li>a compact summary of <code>res_enrich</code>, via <code><a href="../reference/gs_summary_overview.html">gs_summary_overview()</a></code>, where aggregated scores and significance can be displayed on the same plot - for this, there are functions to compare different <code>res_enrich</code> objects (<code><a href="../reference/gs_summary_overview.html">gs_summary_overview()</a></code>, <code><a href="../reference/gs_summary_overview_pair.html">gs_summary_overview_pair()</a></code>)</li>
<li>alternative visual representation of the <code>res_enrich</code> objects, based on <code><a href="../reference/gs_radar.html">gs_radar()</a></code> (equivalent to <code><a href="../reference/gs_radar.html">gs_spider()</a></code>) and <code><a href="../reference/gs_dendro.html">gs_dendro()</a></code>
</li>
</ul>
<p>As in many other panels, the control over the number of genesets displayed is tweaked with the widget in the right sidebar, “Number of genesets”.</p>
<p>A demonstration of how to use this panel effectively is provided by the tour, triggered by clicking on the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button in the upper right corner of the tab.</p>
</div>
<div id="the-bookmarks-panel" class="section level2">
<h2 class="hasAnchor">
<a href="#the-bookmarks-panel" class="anchor"></a>The Bookmarks panel</h2>
<p>In this panel, the user can obtain a complete overview on the bookmarked features of interest, both genes (left side) and genesets (right side).</p>
<p>During a live exploration session, it might be tricky to recall the exact aspect of each point of interest.</p>
<p>Below the interactive <code><a href="https://rdrr.io/pkg/DT/man/datatable.html">DT::datatable</a></code> objects, the content of each table can also be separately downloaded by clicking on the respective button. Moreover, there is a button (with the cocktail icon <i class="fa fa-cocktail"></i>) that triggers the generation of a content-rich report which will focus on all the shortlisted elements. This leverages a template report delivered as a part of <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em>, which accesses the input elements and the reactive values for the bookmarks, based on the <code><a href="../reference/happy_hour.html">happy_hour()</a></code> function.</p>
<p>If a user already knows a set of features and geneset of interest, it is also possible to practically call <code><a href="../reference/happy_hour.html">happy_hour()</a></code> directly from the command line (see the example in the chunk below). Please keep in mind that the identifiers to be used should match the values used as row names of the <code>dds</code> object and as <code>gs_id</code> column of the original <code>res_enrich</code> input.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" data-line-number="1"><span class="kw"><a href="../reference/happy_hour.html">happy_hour</a></span>(<span class="dt">dds =</span> dds_macrophage,</a>
<a class="sourceLine" id="cb16-2" data-line-number="2">           <span class="dt">res_de =</span> res_de,</a>
<a class="sourceLine" id="cb16-3" data-line-number="3">           <span class="dt">res_enrich =</span> res_enrich,</a>
<a class="sourceLine" id="cb16-4" data-line-number="4">           <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb16-5" data-line-number="5">           <span class="dt">project_id =</span> <span class="st">"examplerun"</span>,</a>
<a class="sourceLine" id="cb16-6" data-line-number="6">           <span class="dt">mygenesets =</span> res_enrich<span class="op">$</span>gs_id[<span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="dv">1</span><span class="op">:</span><span class="dv">5</span>,<span class="dv">11</span>,<span class="dv">31</span>)],</a>
<a class="sourceLine" id="cb16-7" data-line-number="7">           <span class="dt">mygenes =</span> <span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"ENSG00000125347"</span>,</a>
<a class="sourceLine" id="cb16-8" data-line-number="8">                       <span class="st">"ENSG00000172399"</span>,</a>
<a class="sourceLine" id="cb16-9" data-line-number="9">                       <span class="st">"ENSG00000137496"</span>)</a>
<a class="sourceLine" id="cb16-10" data-line-number="10">)</a></code></pre></div>
<p>This report will serve as a useful means to generate a permanent and reproducible analysis output, that can be further stored or shared.</p>
<p>Ideally, each live session with <code>GeneTonic</code> could terminate with a call to <code><a href="../reference/happy_hour.html">happy_hour()</a></code> - admittedly, an aperitivo should start with that, and not end - please accept our apologies :) .</p>
<p>A demonstration of how to use this panel effectively is provided by the tour, triggered by clicking on the <button id="tour" type="button" class="btn btn-default action-button" style="color: #0092AC; background-color: #FFFFFF; border-color: #ffffff">
<i class="fa fa-question-circle"></i>

</button> button in the upper right corner of the tab.</p>
</div>
</div>
<div id="functions" class="section level1">
<h1 class="hasAnchor">
<a href="#functions" class="anchor"></a>Using the <code>GeneTonic</code> functions</h1>
<p>The <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> package provides, apart from the main app where to perform the analysis in an interactive way, a number of functions which can be used also as standalone components for a workflow.</p>
<p>This sections contains some showcase examples on how to use them.</p>
<div id="overview-functions-genes-and-gene-sets" class="section level2">
<h2 class="hasAnchor">
<a href="#overview-functions-genes-and-gene-sets" class="anchor"></a>Overview functions: genes and gene sets</h2>
<p>Summarize the top genesets by displaying the logFC of each set’s components - and transform the <code>ggplot</code> output into a <code>plotly</code> visualization, where tooltips can deliver additional information:</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" data-line-number="1">p &lt;-<span class="st"> </span><span class="kw"><a href="../reference/enhance_table.html">enhance_table</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb17-2" data-line-number="2">                   res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb17-3" data-line-number="3">                   <span class="dt">n_gs =</span> <span class="dv">30</span>,</a>
<a class="sourceLine" id="cb17-4" data-line-number="4">                   <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb17-5" data-line-number="5">                   <span class="dt">chars_limit =</span> <span class="dv">60</span>)</a>
<a class="sourceLine" id="cb17-6" data-line-number="6">p</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/enhancetable-1.png" width="960"></p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"plotly"</span>)</a>
<a class="sourceLine" id="cb18-2" data-line-number="2"><span class="kw"><a href="https://rdrr.io/pkg/plotly/man/ggplotly.html">ggplotly</a></span>(p)</a></code></pre></div>
<div id="htmlwidget-62faa80c4cbe4909ba42" style="width:960px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-62faa80c4cbe4909ba42">{"x":{"data":[{"x":[-2.83700609152004,-3.0493139219731,-2.16129031495653,-3.15120929777201,-1.93815068217646,-2.7234323790764,-2.84643631297818,-3.109454731142,1.77290390339302],"y":[2,2,2,2,2,2,2,2,2],"text":["gs_id: GO:0000083<br />TYMS<br />log2FoldChange: -2.837006<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />RRM2<br />log2FoldChange: -3.049314<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />ORC1<br />log2FoldChange: -2.161290<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />E2F1<br />log2FoldChange: -3.151209<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />DHFR<br />log2FoldChange: -1.938151<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />CDT1<br />log2FoldChange: -2.723432<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />CDC6<br />log2FoldChange: -2.846436<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />CDC45<br />log2FoldChange: -3.109455<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle","gs_id: GO:0000083<br />BACH1<br />log2FoldChange:  1.772904<br />gs_desc: regulation of transcription involved in G1/S transition of mitotic cell cycle"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0000083","legendgroup":"GO:0000083","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.32929758693524,-2.37899557656504,-1.94554827104877,-2.22530593550273,-2.2122409523077,-2.90642115780955,-3.109454731142],"y":[17,17,17,17,17,17,17],"text":["gs_id: GO:0000727<br />MCM7<br />log2FoldChange: -2.329298<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />MCM6<br />log2FoldChange: -2.378996<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />MCM5<br />log2FoldChange: -1.945548<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />MCM4<br />log2FoldChange: -2.225306<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />MCM2<br />log2FoldChange: -2.212241<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />GINS2<br />log2FoldChange: -2.906421<br />gs_desc: double-strand break repair via break-induced replication","gs_id: GO:0000727<br />CDC45<br />log2FoldChange: -3.109455<br />gs_desc: double-strand break repair via break-induced replication"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(239,127,73,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0000727","legendgroup":"GO:0000727","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.65617673504703,3.22568381521746,3.16227124908215,3.1034580940225,4.78738236447549,2.30653734457889,2.39194269108928,3.0659319792535,2.34031157754719,-1.84166274676337,-2.12745075431637,1.93191419065976],"y":[25,25,25,25,25,25,25,25,25,25,25,25],"text":["gs_id: GO:0001916<br />P2RX7<br />log2FoldChange:  4.656177<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />CD1C<br />log2FoldChange: -1.841663<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />CD1A<br />log2FoldChange: -2.127451<br />gs_desc: positive regulation of T cell mediated cytotoxicity","gs_id: GO:0001916<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: positive regulation of T cell mediated cytotoxicity"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(229,135,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0001916","legendgroup":"GO:0001916","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,4.41195174127458,3.70967219352629,-2.71277184208635,-3.92611328347182,3.62890094919992,8.65904343346314,3.45826565890319,5.35676707581348,-2.25916703587365,6.05862950756622,2.38379236554552,4.85247349802418,-3.01970889949216,5.1510350648227,3.55795223589057,2.5845093382978,2.17008785297495,2.01811148151959,3.92453990454958,3.61049159070631,4.65617673504703,3.96386661619838,1.62110192793976,2.63778704213988,2.96548781254021,5.21875763645883,4.61284697613189,-4.91175754423084,2.81319390418594,3.04653194988371,4.63675766139974,2.16563558238417,5.56012373180949,4.79046436725014,4.05866209030408,3.22568381521746,3.16227124908215,2.88118162177267,3.1034580940225,4.78738236447549,2.30653734457889,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,3.56059272011321,5.78265733072056,3.81958342347924,4.0715746819976,2.39194269108928,3.0659319792535,2.34031157754719,-3.6587836931735,3.0222425769349,3.35195380511026,-2.31936015050347,-2.59091041087223,3.65448866732852,1.80132288850068,3.84145264917065,5.78213163776364,1.92755872429423,4.96182825464539,2.77768451745292,4.15277182970607,4.53751032668774,3.47849682236114,-3.72976035601486,6.58533754666227,-1.84166274676337,-2.12745075431637,6.31116704251998,5.02791597410857,1.89004273071348,2.16004237748983,2.54200886737226,2.08096803817866,2.80208585419783,3.51953533038161,3.38992601682897,2.55582300669305,2.91426300051913,1.9228185674644,1.93191419065976],"y":[30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30,30],"text":["gs_id: GO:0002250<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TNFSF18<br />log2FoldChange:  4.411952<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TNFSF13B<br />log2FoldChange:  3.709672<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TNFRSF21<br />log2FoldChange: -2.712772<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TNFRSF11A<br />log2FoldChange: -3.926113<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TLR8<br />log2FoldChange:  3.628901<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TBX21<br />log2FoldChange:  8.659043<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TAP2<br />log2FoldChange:  3.458266<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />TAP1<br />log2FoldChange:  5.356767<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SLAMF7<br />log2FoldChange:  6.058630<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SLAMF6<br />log2FoldChange:  2.383792<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SLAMF1<br />log2FoldChange:  4.852473<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SIT1<br />log2FoldChange: -3.019709<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />SERPING1<br />log2FoldChange:  5.151035<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />RSAD2<br />log2FoldChange:  3.557952<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />RNF19B<br />log2FoldChange:  2.584509<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />RELB<br />log2FoldChange:  2.018111<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />PDCD1LG2<br />log2FoldChange:  3.924540<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />PDCD1<br />log2FoldChange:  3.610492<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />P2RX7<br />log2FoldChange:  4.656177<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />MCOLN2<br />log2FoldChange:  3.963867<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />LYN<br />log2FoldChange:  1.621102<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />LILRB3<br />log2FoldChange:  2.637787<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />LILRA1<br />log2FoldChange:  2.965488<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />LAMP3<br />log2FoldChange:  5.218758<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />LAG3<br />log2FoldChange:  4.612847<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />KLRK1<br />log2FoldChange: -4.911758<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />JAK3<br />log2FoldChange:  2.813194<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />ITK<br />log2FoldChange:  4.636758<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IL27<br />log2FoldChange:  4.790464<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IL18BP<br />log2FoldChange:  4.058662<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DOB<br />log2FoldChange:  3.560593<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DOA<br />log2FoldChange:  5.782657<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DMB<br />log2FoldChange:  3.819583<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-DMA<br />log2FoldChange:  4.071575<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />GPR183<br />log2FoldChange: -3.658784<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />GCNT3<br />log2FoldChange:  3.022243<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />FGL2<br />log2FoldChange:  3.351954<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />FGL1<br />log2FoldChange: -2.319360<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />EXO1<br />log2FoldChange: -2.590910<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />ERAP2<br />log2FoldChange:  3.654489<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CTSS<br />log2FoldChange:  1.801323<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CTLA4<br />log2FoldChange:  3.841453<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CLEC6A<br />log2FoldChange:  5.782132<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CLEC10A<br />log2FoldChange:  1.927559<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CEACAM1<br />log2FoldChange:  4.961828<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD80<br />log2FoldChange:  2.777685<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD74<br />log2FoldChange:  4.152772<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD7<br />log2FoldChange:  4.537510<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD40<br />log2FoldChange:  3.478497<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD28<br />log2FoldChange: -3.729760<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD274<br />log2FoldChange:  6.585338<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD1C<br />log2FoldChange: -1.841663<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />CD1A<br />log2FoldChange: -2.127451<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C4B<br />log2FoldChange:  6.311167<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C4A<br />log2FoldChange:  5.027916<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C2<br />log2FoldChange:  2.160042<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C1S<br />log2FoldChange:  2.542009<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C1RL<br />log2FoldChange:  2.080968<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C1R<br />log2FoldChange:  2.802086<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />C1QB<br />log2FoldChange:  3.519535<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />BTN3A3<br />log2FoldChange:  3.389926<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />BTN3A2<br />log2FoldChange:  2.555823<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />BTN3A1<br />log2FoldChange:  2.914263<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />BCL6<br />log2FoldChange:  1.922819<br />gs_desc: adaptive immune response","gs_id: GO:0002250<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: adaptive immune response"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(216,144,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0002250","legendgroup":"GO:0002250","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.45826565890319,3.1034580940225,4.78738236447549,2.30653734457889,2.39194269108928,3.0659319792535,2.34031157754719,-1.84166274676337,-2.12745075431637,1.93191419065976],"y":[24,24,24,24,24,24,24,24,24,24],"text":["gs_id: GO:0002475<br />TAP2<br />log2FoldChange:  3.458266<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />CD1C<br />log2FoldChange: -1.841663<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />CD1A<br />log2FoldChange: -2.127451<br />gs_desc: antigen processing and presentation via MHC class Ib","gs_id: GO:0002475<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: antigen processing and presentation via MHC class Ib"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(201,152,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0002475","legendgroup":"GO:0002475","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.86500691828739,3.45826565890319,5.35676707581348,3.35897149231688,2.06580501739756,5.36101085730258,2.54482374369772,2.13221835413023,2.09076637177846,3.1034580940225,4.78738236447549,2.30653734457889,2.39194269108928,3.0659319792535,2.34031157754719,1.93191419065976],"y":[12,12,12,12,12,12,12,12,12,12,12,12,12,12,12,12],"text":["gs_id: GO:0002479<br />TAPBP<br />log2FoldChange:  1.865007<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />TAP2<br />log2FoldChange:  3.458266<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />TAP1<br />log2FoldChange:  5.356767<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />PSME2<br />log2FoldChange:  3.358971<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />PSME1<br />log2FoldChange:  2.065805<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />PSMB9<br />log2FoldChange:  5.361011<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />PSMB8<br />log2FoldChange:  2.544824<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />PSMB10<br />log2FoldChange:  2.132218<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />NCF1<br />log2FoldChange:  2.090766<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","gs_id: GO:0002479<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(183,159,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0002479","legendgroup":"GO:0002479","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-2.83433379073174,-2.34308591123898,-2.16129031495653,-2.32929758693524,-2.37899557656504,-1.94554827104877,-2.22530593550273,-2.2122409523077,-3.33161008910374,-2.7234323790764,-2.84643631297818,-3.109454731142,-3.85847837276111],"y":[23,23,23,23,23,23,23,23,23,23,23,23,23],"text":["gs_id: GO:0006270<br />TICRR<br />log2FoldChange: -2.834334<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />POLE2<br />log2FoldChange: -2.343086<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />ORC1<br />log2FoldChange: -2.161290<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM7<br />log2FoldChange: -2.329298<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM6<br />log2FoldChange: -2.378996<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM5<br />log2FoldChange: -1.945548<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM4<br />log2FoldChange: -2.225306<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM2<br />log2FoldChange: -2.212241<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />MCM10<br />log2FoldChange: -3.331610<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />CDT1<br />log2FoldChange: -2.723432<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />CDC6<br />log2FoldChange: -2.846436<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />CDC45<br />log2FoldChange: -3.109455<br />gs_desc: DNA replication initiation","gs_id: GO:0006270<br />CCNE2<br />log2FoldChange: -3.858478<br />gs_desc: DNA replication initiation"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(163,165,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0006270","legendgroup":"GO:0006270","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,5.67252472783529,-3.92611328347182,3.87587685949976,3.62890094919992,2.02910367999242,-2.17883314323416,2.7597135772053,1.98231370454255,2.71128550656837,3.47963558248526,3.41669562174808,-2.25916703587365,1.88670752672949,5.1510350648227,1.87055840762627,2.17008785297495,2.01811148151959,2.23018082649667,1.82791085595686,4.65617673504703,-2.04399005733986,3.4342201708568,2.45859772219047,-2.40082769533595,-2.35643419920815,2.08650809952435,3.07802632679234,1.62110192793976,1.95469818106238,4.60581863970274,2.72369423503615,1.45960495399795,3.04653194988371,4.79046436725014,-2.93463798302811,2.81740349265852,4.45361893975568,9.54515259204768,2.88118162177267,2.30653734457889,4.0633126752432,1.99985393753493,-3.52869456303385,2.86848406194017,10.3764778869772,-2.11763222636487,1.75309967700762,-3.20092457252238,10.3554913627931,-2.5740744669526,8.73344469430067,8.76155418580896,-2.65432112372819,4.3268936504892,6.11224883805674,4.98711071929753,4.33269590317081,-2.82479481552114,3.47849682236114,-3.72976035601486,2.5641240049707,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,1.9360556139888,6.31116704251998,5.02791597410857,1.89004273071348,2.16004237748983,2.54200886737226,2.80208585419783,3.51953533038161,1.9228185674644,-1.62031063892701,7.97721557262308,3.80912471227255,-2.83132683176397,7.8042530826668,3.77188750386325,-2.72213577200739,3.04596795898775,-2.67226467283332,7.87008771892115],"y":[21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21],"text":["gs_id: GO:0006954<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TNFRSF11A<br />log2FoldChange: -3.926113<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TNFAIP6<br />log2FoldChange:  3.875877<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TLR8<br />log2FoldChange:  3.628901<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TLR7<br />log2FoldChange:  2.029104<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TLR5<br />log2FoldChange: -2.178833<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TLR10<br />log2FoldChange:  2.759714<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TICAM2<br />log2FoldChange:  1.982314<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />TGM2<br />log2FoldChange:  2.711286<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />SUCNR1<br />log2FoldChange:  3.479636<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />STAP1<br />log2FoldChange:  3.416696<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />SLAMF8<br />log2FoldChange:  1.886708<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />SERPING1<br />log2FoldChange:  5.151035<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />SBNO2<br />log2FoldChange:  1.870558<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />RELB<br />log2FoldChange:  2.018111<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />PTGER4<br />log2FoldChange:  2.230181<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />POLB<br />log2FoldChange:  1.827911<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />P2RX7<br />log2FoldChange:  4.656177<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />OSM<br />log2FoldChange: -2.043990<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />NOD2<br />log2FoldChange:  3.434220<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />NMI<br />log2FoldChange:  2.458598<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />NLRP12<br />log2FoldChange: -2.400828<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />MMP9<br />log2FoldChange: -2.356434<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />MMP25<br />log2FoldChange:  2.086508<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />MEFV<br />log2FoldChange:  3.078026<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />LYN<br />log2FoldChange:  1.621102<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />LY75<br />log2FoldChange:  1.954698<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />LRRK2<br />log2FoldChange:  4.605819<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />KLF4<br />log2FoldChange:  2.723694<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />KARS1<br />log2FoldChange:  1.459605<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />IL27<br />log2FoldChange:  4.790464<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />IL22RA2<br />log2FoldChange: -2.934638<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />IL1RN<br />log2FoldChange:  2.817403<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />IL15<br />log2FoldChange:  4.453619<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />IDO1<br />log2FoldChange:  9.545153<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />HGF<br />log2FoldChange:  4.063313<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />GSDMD<br />log2FoldChange:  1.999854<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />GHRL<br />log2FoldChange: -3.528695<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />GGT5<br />log2FoldChange:  2.868484<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />GBP5<br />log2FoldChange: 10.376478<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CYSLTR1<br />log2FoldChange: -2.117632<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CYLD<br />log2FoldChange:  1.753100<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CXCR3<br />log2FoldChange: -3.200925<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CXCL5<br />log2FoldChange: -2.574074<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CXCL11<br />log2FoldChange:  8.733445<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CRHBP<br />log2FoldChange: -2.654321<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CR1L<br />log2FoldChange:  4.326894<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CIITA<br />log2FoldChange:  6.112249<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CFH<br />log2FoldChange:  4.987111<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CFB<br />log2FoldChange:  4.332696<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CDH5<br />log2FoldChange: -2.824795<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CD40<br />log2FoldChange:  3.478497<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CD28<br />log2FoldChange: -3.729760<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCRL2<br />log2FoldChange:  2.564124<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />CASP4<br />log2FoldChange:  1.936056<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C4B<br />log2FoldChange:  6.311167<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C4A<br />log2FoldChange:  5.027916<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C2<br />log2FoldChange:  2.160042<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C1S<br />log2FoldChange:  2.542009<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C1R<br />log2FoldChange:  2.802086<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />C1QB<br />log2FoldChange:  3.519535<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />BCL6<br />log2FoldChange:  1.922819<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />APPL2<br />log2FoldChange: -1.620311<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />APOL3<br />log2FoldChange:  7.977216<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />APOL2<br />log2FoldChange:  3.809125<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />AOC3<br />log2FoldChange: -2.831327<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />AIM2<br />log2FoldChange:  7.804253<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />AGT<br />log2FoldChange:  3.771888<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />ADORA3<br />log2FoldChange: -2.722136<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />ADGRE5<br />log2FoldChange:  3.045968<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />ADAM8<br />log2FoldChange: -2.672265<br />gs_desc: inflammatory response","gs_id: GO:0006954<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: inflammatory response"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(138,171,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0006954","legendgroup":"GO:0006954","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.1510350648227,6.31116704251998,5.02791597410857,1.89004273071348,2.16004237748983,2.54200886737226,2.08096803817866,2.80208585419783,3.51953533038161],"y":[18,18,18,18,18,18,18,18,18],"text":["gs_id: GO:0006958<br />SERPING1<br />log2FoldChange:  5.151035<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C4B<br />log2FoldChange:  6.311167<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C4A<br />log2FoldChange:  5.027916<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C2<br />log2FoldChange:  2.160042<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C1S<br />log2FoldChange:  2.542009<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C1RL<br />log2FoldChange:  2.080968<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C1R<br />log2FoldChange:  2.802086<br />gs_desc: complement activation, classical pathway","gs_id: GO:0006958<br />C1QB<br />log2FoldChange:  3.519535<br />gs_desc: complement activation, classical pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(107,177,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0006958","legendgroup":"GO:0006958","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.71128550656837,3.47963558248526,-1.90048134172168,3.75685619125814,-5.36585747842242,5.2929165976604,-2.82234913425517,2.08417394861648,2.23018082649667,3.35949673221329,3.38889368923079,-2.20620113543193,9.86390377567812,5.52786099356419,3.50206148895693,-4.63805794625894,-2.07185903746445,4.60581863970274,3.04653194988371,-1.53833931752796,1.67228074038225,-2.86104929342773,6.08500963649982,6.56049072560547,3.64380939706531,4.39496481128588,-3.6587836931735,-2.24480461627157,-3.52869456303385,-2.57892966875333,-2.55498834059302,-2.03707618388711,3.48458869155741,6.40684498426235,1.69762584230684,1.62259999437351,-2.26563476080381,4.7411680101808,-2.11763222636487,-3.20092457252238,10.3554913627931,-2.5740744669526,8.73344469430067,8.76155418580896,-4.47628545862656,2.1820181147792,-1.93942740929992,2.5641240049707,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,1.89004273071348,3.77188750386325,-2.72213577200739,2.32596943639399,3.04596795898775],"y":[5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5],"text":["gs_id: GO:0007186<br />TGM2<br />log2FoldChange:  2.711286<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />SUCNR1<br />log2FoldChange:  3.479636<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />STMN1<br />log2FoldChange: -1.900481<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />SSTR2<br />log2FoldChange:  3.756856<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />S1PR1<br />log2FoldChange: -5.365857<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />RXFP1<br />log2FoldChange:  5.292917<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />RGS18<br />log2FoldChange: -2.822349<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />RAPGEF2<br />log2FoldChange:  2.084174<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />PTGER4<br />log2FoldChange:  2.230181<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />PLEK<br />log2FoldChange:  3.359497<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />PDE4B<br />log2FoldChange:  3.388894<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />PDE3B<br />log2FoldChange: -2.206201<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />P2RY14<br />log2FoldChange:  9.863904<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />NXPH2<br />log2FoldChange:  5.527861<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />NPR1<br />log2FoldChange:  3.502061<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />NECAB2<br />log2FoldChange: -4.638058<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />MCF2L<br />log2FoldChange: -2.071859<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />LRRK2<br />log2FoldChange:  4.605819<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />ITSN1<br />log2FoldChange: -1.538339<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />IGF2R<br />log2FoldChange:  1.672281<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />HRH2<br />log2FoldChange: -2.861049<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />HCAR3<br />log2FoldChange:  6.085010<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />HCAR2<br />log2FoldChange:  6.560491<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GRM8<br />log2FoldChange:  3.643809<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GPR31<br />log2FoldChange:  4.394965<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GPR183<br />log2FoldChange: -3.658784<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GNG2<br />log2FoldChange: -2.244805<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GHRL<br />log2FoldChange: -3.528695<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />GABRA3<br />log2FoldChange: -2.578930<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />FCN1<br />log2FoldChange: -2.554988<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />F2RL3<br />log2FoldChange: -2.037076<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />ECEL1<br />log2FoldChange:  6.406845<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />DYNLT1<br />log2FoldChange:  1.697626<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />DTNBP1<br />log2FoldChange:  1.622600<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />DNM1<br />log2FoldChange: -2.265635<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CYSLTR2<br />log2FoldChange:  4.741168<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CYSLTR1<br />log2FoldChange: -2.117632<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CXCR3<br />log2FoldChange: -3.200925<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CXCL5<br />log2FoldChange: -2.574074<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CXCL11<br />log2FoldChange:  8.733445<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CX3CR1<br />log2FoldChange: -4.476285<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CISH<br />log2FoldChange:  2.182018<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CELSR1<br />log2FoldChange: -1.939427<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCRL2<br />log2FoldChange:  2.564124<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />AGT<br />log2FoldChange:  3.771888<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />ADORA3<br />log2FoldChange: -2.722136<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />ADGRG5<br />log2FoldChange:  2.325969<br />gs_desc: G protein-coupled receptor signaling pathway","gs_id: GO:0007186<br />ADGRE5<br />log2FoldChange:  3.045968<br />gs_desc: G protein-coupled receptor signaling pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(57,182,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0007186","legendgroup":"GO:0007186","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.86500691828739,3.45826565890319,5.35676707581348,3.1034580940225,4.78738236447549,2.30653734457889,2.39194269108928,3.0659319792535,2.34031157754719,3.65448866732852,1.93191419065976],"y":[27,27,27,27,27,27,27,27,27,27,27],"text":["gs_id: GO:0019885<br />TAPBP<br />log2FoldChange:  1.865007<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />TAP2<br />log2FoldChange:  3.458266<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />TAP1<br />log2FoldChange:  5.356767<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />ERAP2<br />log2FoldChange:  3.654489<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I","gs_id: GO:0019885<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: antigen processing and presentation of endogenous peptide antigen via MHC class I"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,186,56,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0019885","legendgroup":"GO:0019885","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.61284697613189,-2.28548769136045,-2.67336107888763,-2.18455933180657,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,3.56059272011321,5.78265733072056,3.81958342347924,4.0715746819976,1.80132288850068,4.15277182970607],"y":[11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11],"text":["gs_id: GO:0019886<br />LAG3<br />log2FoldChange:  4.612847<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />KIF26A<br />log2FoldChange: -2.285488<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />KIF15<br />log2FoldChange: -2.673361<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />KIF11<br />log2FoldChange: -2.184559<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DOB<br />log2FoldChange:  3.560593<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DOA<br />log2FoldChange:  5.782657<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DMB<br />log2FoldChange:  3.819583<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />HLA-DMA<br />log2FoldChange:  4.071575<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />CTSS<br />log2FoldChange:  1.801323<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II","gs_id: GO:0019886<br />CD74<br />log2FoldChange:  4.152772<br />gs_desc: antigen processing and presentation of exogenous peptide antigen via MHC class II"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,189,95,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0019886","legendgroup":"GO:0019886","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.1510350648227,4.3268936504892,4.98711071929753,4.33269590317081,6.31116704251998,5.02791597410857,1.89004273071348,2.16004237748983,2.54200886737226,2.80208585419783,3.51953533038161],"y":[14,14,14,14,14,14,14,14,14,14,14],"text":["gs_id: GO:0030449<br />SERPING1<br />log2FoldChange:  5.151035<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />CR1L<br />log2FoldChange:  4.326894<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />CFH<br />log2FoldChange:  4.987111<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />CFB<br />log2FoldChange:  4.332696<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C4B<br />log2FoldChange:  6.311167<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C4A<br />log2FoldChange:  5.027916<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C2<br />log2FoldChange:  2.160042<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C1S<br />log2FoldChange:  2.542009<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C1R<br />log2FoldChange:  2.802086<br />gs_desc: regulation of complement activation","gs_id: GO:0030449<br />C1QB<br />log2FoldChange:  3.519535<br />gs_desc: regulation of complement activation"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,191,125,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0030449","legendgroup":"GO:0030449","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.38889368923079,2.81740349265852,3.48458869155741,2.34349992917502,10.3554913627931,-2.5740744669526,8.73344469430067,8.76155418580896,-2.38980468204878,4.15277182970607,-4.49789067255898,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491],"y":[10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10],"text":["gs_id: GO:0030593<br />PDE4B<br />log2FoldChange:  3.388894<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />IL1RN<br />log2FoldChange:  2.817403<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />DYSF<br />log2FoldChange:  2.343500<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CXCL5<br />log2FoldChange: -2.574074<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CXCL11<br />log2FoldChange:  8.733445<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CXADR<br />log2FoldChange: -2.389805<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CD74<br />log2FoldChange:  4.152772<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CD300H<br />log2FoldChange: -4.497891<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: neutrophil chemotaxis","gs_id: GO:0030593<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: neutrophil chemotaxis"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,192,151,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0030593","legendgroup":"GO:0030593","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.62890094919992,2.02910367999242,-2.25916703587365,2.38379236554552,2.17008785297495,3.38889368923079,-4.91175754423084,4.79046436725014,3.08201160215221,3.22568381521746,3.16227124908215,4.29792470711347,5.593758623049],"y":[6,6,6,6,6,6,6,6,6,6,6,6,6],"text":["gs_id: GO:0032729<br />TLR8<br />log2FoldChange:  3.628901<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />TLR7<br />log2FoldChange:  2.029104<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />SLAMF6<br />log2FoldChange:  2.383792<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />PDE4B<br />log2FoldChange:  3.388894<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />KLRK1<br />log2FoldChange: -4.911758<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />IL27<br />log2FoldChange:  4.790464<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />IL12RB2<br />log2FoldChange:  3.082012<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: positive regulation of interferon-gamma production","gs_id: GO:0032729<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: positive regulation of interferon-gamma production"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,192,175,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0032729","legendgroup":"GO:0032729","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.67252472783529,10.2693828470306,3.04590598607216,2.39661366930567,3.15217022771912,7.6897534215834,-2.25916703587365,2.19406813625637,2.11243275228888,1.99922981854563,2.50768783402577,2.4201447661022,2.45859772219047,3.27556759488586,2.72699778946657,3.07802632679234,3.04653194988371,2.16563558238417,5.56012373180949,3.22568381521746,2.10210271975426,2.44590700081427,3.32564098307476,2.88118162177267,3.1034580940225,4.78738236447549,2.30653734457889,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,2.39194269108928,3.0659319792535,2.34031157754719,4.74271607830313,7.81623716208599,3.39223239474714,10.3764778869772,9.75276915703323,3.67245506894423,6.66580377223998,7.68616604774906,3.48458869155741,6.11224883805674,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,2.50801757300173,2.82812524913625,1.93191419065976,7.87008771892115],"y":[22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22,22],"text":["gs_id: GO:0034341<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />UBD<br />log2FoldChange: 10.269383<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />TRIM31<br />log2FoldChange:  3.045906<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />TRIM22<br />log2FoldChange:  2.396614<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />SOCS1<br />log2FoldChange:  7.689753<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />RAB20<br />log2FoldChange:  2.194068<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />PML<br />log2FoldChange:  2.112433<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />PARP14<br />log2FoldChange:  1.999230<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />NUB1<br />log2FoldChange:  2.420145<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />NMI<br />log2FoldChange:  2.458598<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />MT2A<br />log2FoldChange:  2.726998<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />MEFV<br />log2FoldChange:  3.078026<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IFITM3<br />log2FoldChange:  2.102103<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IFITM2<br />log2FoldChange:  2.445907<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />IFITM1<br />log2FoldChange:  3.325641<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GCH1<br />log2FoldChange:  4.742716<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP7<br />log2FoldChange:  7.816237<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP6<br />log2FoldChange:  3.392232<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP5<br />log2FoldChange: 10.376478<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP4<br />log2FoldChange:  9.752769<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP3<br />log2FoldChange:  3.672455<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CIITA<br />log2FoldChange:  6.112249<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CAMK2D<br />log2FoldChange:  2.508018<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />CALCOCO2<br />log2FoldChange:  2.828125<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: response to interferon-gamma","gs_id: GO:0034341<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: response to interferon-gamma"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0034341","legendgroup":"GO:0034341","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.67252472783529,3.70967219352629,2.17008785297495,3.92453990454958,2.81319390418594,4.45361893975568,3.22568381521746,3.16227124908215,2.30653734457889,4.29792470711347,5.593758623049,3.81958342347924,2.77768451745292,-3.72976035601486,6.58533754666227,2.29474257213188],"y":[8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8],"text":["gs_id: GO:0042102<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />TNFSF13B<br />log2FoldChange:  3.709672<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />PDCD1LG2<br />log2FoldChange:  3.924540<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />JAK3<br />log2FoldChange:  2.813194<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />IL15<br />log2FoldChange:  4.453619<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />HLA-DMB<br />log2FoldChange:  3.819583<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />CD80<br />log2FoldChange:  2.777685<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />CD28<br />log2FoldChange: -3.729760<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />CD274<br />log2FoldChange:  6.585338<br />gs_desc: positive regulation of T cell proliferation","gs_id: GO:0042102<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: positive regulation of T cell proliferation"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,188,216,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0042102","legendgroup":"GO:0042102","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.84372011500737,-2.25916703587365,3.16227124908215,7.81623716208599,3.39223239474714,9.75276915703323,6.66580377223998,3.47849682236114,4.68936838743429],"y":[16,16,16,16,16,16,16,16,16],"text":["gs_id: GO:0042832<br />TSPAN32<br />log2FoldChange:  3.843720<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />GBP7<br />log2FoldChange:  7.816237<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />GBP6<br />log2FoldChange:  3.392232<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />GBP4<br />log2FoldChange:  9.752769<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />CD40<br />log2FoldChange:  3.478497<br />gs_desc: defense response to protozoan","gs_id: GO:0042832<br />BATF2<br />log2FoldChange:  4.689368<br />gs_desc: defense response to protozoan"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,183,233,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0042832","legendgroup":"GO:0042832","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,3.55795223589057,2.20289546675553,5.04348744419464,2.10210271975426,2.44590700081427,3.32564098307476,1.92376793164837,5.04548843809308,2.29474257213188,2.57211029701464,3.52553506636734,4.78812527168352],"y":[7,7,7,7,7,7,7,7,7,7,7,7,7],"text":["gs_id: GO:0045071<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />RSAD2<br />log2FoldChange:  3.557952<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />PARP10<br />log2FoldChange:  2.202895<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />ISG20<br />log2FoldChange:  5.043487<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />IFITM3<br />log2FoldChange:  2.102103<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />IFITM2<br />log2FoldChange:  2.445907<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />IFITM1<br />log2FoldChange:  3.325641<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />IFIT5<br />log2FoldChange:  1.923768<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />APOBEC3G<br />log2FoldChange:  2.572110<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />APOBEC3D<br />log2FoldChange:  3.525535<br />gs_desc: negative regulation of viral genome replication","gs_id: GO:0045071<br />APOBEC3A<br />log2FoldChange:  4.788125<br />gs_desc: negative regulation of viral genome replication"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,176,246,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0045071","legendgroup":"GO:0045071","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.50182698034051,2.98487322842038,5.67252472783529,10.2693828470306,3.04590598607216,2.39661366930567,2.72336953419559,3.62890094919992,2.02910367999242,-2.17883314323416,2.7597135772053,5.19344776114725,1.98231370454255,2.27897800399411,3.15217022771912,7.6897534215834,-2.25916703587365,1.88670752672949,6.05862950756622,2.38379236554552,4.85247349802418,5.1510350648227,-1.66153540047202,3.55795223589057,2.17008785297495,2.01811148151959,2.19406813625637,2.32021971263508,3.35897149231688,2.06580501739756,5.36101085730258,2.54482374369772,2.13221835413023,2.11243275228888,1.99922981854563,3.81865242232792,2.50768783402577,2.4201447661022,3.4342201708568,2.45859772219047,3.27556759488586,2.09076637177846,3.91895578419912,2.72699778946657,2.66765749871062,3.07802632679234,3.96386661619838,1.62110192793976,2.34619510075457,4.61284697613189,-4.91175754423084,2.81319390418594,3.04653194988371,5.04348744419464,2.16563558238417,5.56012373180949,4.79046436725014,3.22568381521746,3.16227124908215,2.10210271975426,2.44590700081427,3.32564098307476,1.92376793164837,3.52431460055161,3.66316736696903,2.52518457338175,2.6300310875818,4.52807122533192,1.9452862624663,2.88118162177267,3.1034580940225,4.78738236447549,2.30653734457889,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,2.39194269108928,3.0659319792535,2.34031157754719,2.35983818139827,1.99985393753493,-2.1526014249945,4.74271607830313,7.81623716208599,3.39223239474714,10.3764778869772,9.75276915703323,3.67245506894423,6.66580377223998,7.68616604774906,-2.55498834059302,3.48458869155741,1.90531137609405,2.35887039049424,1.75309967700762,-4.47628545862656,1.80132288850068,-3.0736829491665,5.78213163776364,1.92755872429423,6.11224883805674,4.98711071929753,4.33269590317081,4.96182825464539,2.59579881150508,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,1.9360556139888,2.50801757300173,2.82812524913625,6.31116704251998,5.02791597410857,1.89004273071348,2.16004237748983,2.54200886737226,2.08096803817866,2.80208585419783,3.51953533038161,1.93191419065976,-1.62031063892701,7.13529667019009,2.57211029701464,3.52553506636734,4.78812527168352,7.8042530826668,-2.67226467283332,7.87008771892115],"y":[19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19],"text":["gs_id: GO:0045087<br />ZBP1<br />log2FoldChange:  3.501827<br />gs_desc: innate immune response","gs_id: GO:0045087<br />XAF1<br />log2FoldChange:  2.984873<br />gs_desc: innate immune response","gs_id: GO:0045087<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: innate immune response","gs_id: GO:0045087<br />UBD<br />log2FoldChange: 10.269383<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TRIM31<br />log2FoldChange:  3.045906<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TRIM22<br />log2FoldChange:  2.396614<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TRAFD1<br />log2FoldChange:  2.723370<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TLR8<br />log2FoldChange:  3.628901<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TLR7<br />log2FoldChange:  2.029104<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TLR5<br />log2FoldChange: -2.178833<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TLR10<br />log2FoldChange:  2.759714<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TIFA<br />log2FoldChange:  5.193448<br />gs_desc: innate immune response","gs_id: GO:0045087<br />TICAM2<br />log2FoldChange:  1.982314<br />gs_desc: innate immune response","gs_id: GO:0045087<br />STAT2<br />log2FoldChange:  2.278978<br />gs_desc: innate immune response","gs_id: GO:0045087<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SOCS1<br />log2FoldChange:  7.689753<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SLC11A1<br />log2FoldChange: -2.259167<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SLAMF8<br />log2FoldChange:  1.886708<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SLAMF7<br />log2FoldChange:  6.058630<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SLAMF6<br />log2FoldChange:  2.383792<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SLAMF1<br />log2FoldChange:  4.852473<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SERPING1<br />log2FoldChange:  5.151035<br />gs_desc: innate immune response","gs_id: GO:0045087<br />SERINC5<br />log2FoldChange: -1.661535<br />gs_desc: innate immune response","gs_id: GO:0045087<br />RSAD2<br />log2FoldChange:  3.557952<br />gs_desc: innate immune response","gs_id: GO:0045087<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: innate immune response","gs_id: GO:0045087<br />RELB<br />log2FoldChange:  2.018111<br />gs_desc: innate immune response","gs_id: GO:0045087<br />RAB20<br />log2FoldChange:  2.194068<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PYHIN1<br />log2FoldChange:  2.320220<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PSME2<br />log2FoldChange:  3.358971<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PSME1<br />log2FoldChange:  2.065805<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PSMB9<br />log2FoldChange:  5.361011<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PSMB8<br />log2FoldChange:  2.544824<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PSMB10<br />log2FoldChange:  2.132218<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PML<br />log2FoldChange:  2.112433<br />gs_desc: innate immune response","gs_id: GO:0045087<br />PARP14<br />log2FoldChange:  1.999230<br />gs_desc: innate immune response","gs_id: GO:0045087<br />OPTN<br />log2FoldChange:  3.818652<br />gs_desc: innate immune response","gs_id: GO:0045087<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: innate immune response","gs_id: GO:0045087<br />NUB1<br />log2FoldChange:  2.420145<br />gs_desc: innate immune response","gs_id: GO:0045087<br />NOD2<br />log2FoldChange:  3.434220<br />gs_desc: innate immune response","gs_id: GO:0045087<br />NMI<br />log2FoldChange:  2.458598<br />gs_desc: innate immune response","gs_id: GO:0045087<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: innate immune response","gs_id: GO:0045087<br />NCF1<br />log2FoldChange:  2.090766<br />gs_desc: innate immune response","gs_id: GO:0045087<br />MUC1<br />log2FoldChange:  3.918956<br />gs_desc: innate immune response","gs_id: GO:0045087<br />MT2A<br />log2FoldChange:  2.726998<br />gs_desc: innate immune response","gs_id: GO:0045087<br />MSRB1<br />log2FoldChange:  2.667657<br />gs_desc: innate immune response","gs_id: GO:0045087<br />MEFV<br />log2FoldChange:  3.078026<br />gs_desc: innate immune response","gs_id: GO:0045087<br />MCOLN2<br />log2FoldChange:  3.963867<br />gs_desc: innate immune response","gs_id: GO:0045087<br />LYN<br />log2FoldChange:  1.621102<br />gs_desc: innate immune response","gs_id: GO:0045087<br />LILRA2<br />log2FoldChange:  2.346195<br />gs_desc: innate immune response","gs_id: GO:0045087<br />LAG3<br />log2FoldChange:  4.612847<br />gs_desc: innate immune response","gs_id: GO:0045087<br />KLRK1<br />log2FoldChange: -4.911758<br />gs_desc: innate immune response","gs_id: GO:0045087<br />JAK3<br />log2FoldChange:  2.813194<br />gs_desc: innate immune response","gs_id: GO:0045087<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: innate immune response","gs_id: GO:0045087<br />ISG20<br />log2FoldChange:  5.043487<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IL27<br />log2FoldChange:  4.790464<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFITM3<br />log2FoldChange:  2.102103<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFITM2<br />log2FoldChange:  2.445907<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFITM1<br />log2FoldChange:  3.325641<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFIT5<br />log2FoldChange:  1.923768<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFIT3<br />log2FoldChange:  3.524315<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFIT2<br />log2FoldChange:  3.663167<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFIH1<br />log2FoldChange:  2.525185<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFI35<br />log2FoldChange:  2.630031<br />gs_desc: innate immune response","gs_id: GO:0045087<br />IFI27<br />log2FoldChange:  4.528071<br />gs_desc: innate immune response","gs_id: GO:0045087<br />ICAM2<br />log2FoldChange:  1.945286<br />gs_desc: innate immune response","gs_id: GO:0045087<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: innate immune response","gs_id: GO:0045087<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: innate immune response","gs_id: GO:0045087<br />H2BC21<br />log2FoldChange:  2.359838<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GSDMD<br />log2FoldChange:  1.999854<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GRAMD4<br />log2FoldChange: -2.152601<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GCH1<br />log2FoldChange:  4.742716<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP7<br />log2FoldChange:  7.816237<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP6<br />log2FoldChange:  3.392232<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP5<br />log2FoldChange: 10.376478<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP4<br />log2FoldChange:  9.752769<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP3<br />log2FoldChange:  3.672455<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: innate immune response","gs_id: GO:0045087<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: innate immune response","gs_id: GO:0045087<br />FCN1<br />log2FoldChange: -2.554988<br />gs_desc: innate immune response","gs_id: GO:0045087<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: innate immune response","gs_id: GO:0045087<br />DTX3L<br />log2FoldChange:  1.905311<br />gs_desc: innate immune response","gs_id: GO:0045087<br />DDX60<br />log2FoldChange:  2.358870<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CYLD<br />log2FoldChange:  1.753100<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CX3CR1<br />log2FoldChange: -4.476285<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CTSS<br />log2FoldChange:  1.801323<br />gs_desc: innate immune response","gs_id: GO:0045087<br />COLEC12<br />log2FoldChange: -3.073683<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CLEC6A<br />log2FoldChange:  5.782132<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CLEC10A<br />log2FoldChange:  1.927559<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CIITA<br />log2FoldChange:  6.112249<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CFH<br />log2FoldChange:  4.987111<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CFB<br />log2FoldChange:  4.332696<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CEACAM1<br />log2FoldChange:  4.961828<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CD300LF<br />log2FoldChange:  2.595799<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CASP4<br />log2FoldChange:  1.936056<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CAMK2D<br />log2FoldChange:  2.508018<br />gs_desc: innate immune response","gs_id: GO:0045087<br />CALCOCO2<br />log2FoldChange:  2.828125<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C4B<br />log2FoldChange:  6.311167<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C4A<br />log2FoldChange:  5.027916<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C2<br />log2FoldChange:  2.160042<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C1S<br />log2FoldChange:  2.542009<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C1RL<br />log2FoldChange:  2.080968<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C1R<br />log2FoldChange:  2.802086<br />gs_desc: innate immune response","gs_id: GO:0045087<br />C1QB<br />log2FoldChange:  3.519535<br />gs_desc: innate immune response","gs_id: GO:0045087<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: innate immune response","gs_id: GO:0045087<br />APPL2<br />log2FoldChange: -1.620311<br />gs_desc: innate immune response","gs_id: GO:0045087<br />APOL1<br />log2FoldChange:  7.135297<br />gs_desc: innate immune response","gs_id: GO:0045087<br />APOBEC3G<br />log2FoldChange:  2.572110<br />gs_desc: innate immune response","gs_id: GO:0045087<br />APOBEC3D<br />log2FoldChange:  3.525535<br />gs_desc: innate immune response","gs_id: GO:0045087<br />APOBEC3A<br />log2FoldChange:  4.788125<br />gs_desc: innate immune response","gs_id: GO:0045087<br />AIM2<br />log2FoldChange:  7.804253<br />gs_desc: innate immune response","gs_id: GO:0045087<br />ADAM8<br />log2FoldChange: -2.672265<br />gs_desc: innate immune response","gs_id: GO:0045087<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: innate immune response"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,167,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0045087","legendgroup":"GO:0045087","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.92611328347182,2.02910367999242,2.7597135772053,2.71128550656837,3.47963558248526,3.41669562174808,2.23018082649667,-2.04399005733986,3.4342201708568,-2.40082769533595,4.60581863970274,1.45960495399795,3.04653194988371,4.45361893975568,9.54515259204768,2.30653734457889,10.3764778869772,-3.72976035601486,3.76451516743393,1.89004273071348,3.77188750386325,-2.67226467283332],"y":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"text":["gs_id: GO:0050729<br />TNFRSF11A<br />log2FoldChange: -3.926113<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />TLR7<br />log2FoldChange:  2.029104<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />TLR10<br />log2FoldChange:  2.759714<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />TGM2<br />log2FoldChange:  2.711286<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />SUCNR1<br />log2FoldChange:  3.479636<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />STAP1<br />log2FoldChange:  3.416696<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />PTGER4<br />log2FoldChange:  2.230181<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />OSM<br />log2FoldChange: -2.043990<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />NOD2<br />log2FoldChange:  3.434220<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />NLRP12<br />log2FoldChange: -2.400828<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />LRRK2<br />log2FoldChange:  4.605819<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />KARS1<br />log2FoldChange:  1.459605<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />IL15<br />log2FoldChange:  4.453619<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />IDO1<br />log2FoldChange:  9.545153<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />GBP5<br />log2FoldChange: 10.376478<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />CD28<br />log2FoldChange: -3.729760<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />C3<br />log2FoldChange:  1.890043<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />AGT<br />log2FoldChange:  3.771888<br />gs_desc: positive regulation of inflammatory response","gs_id: GO:0050729<br />ADAM8<br />log2FoldChange: -2.672265<br />gs_desc: positive regulation of inflammatory response"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(97,156,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0050729","legendgroup":"GO:0050729","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-3.03526122166895,2.17008785297495,4.65617673504703,-4.91175754423084,3.16227124908215,2.30653734457889,2.35983818139827,1.99985393753493,7.81623716208599,3.39223239474714,9.75276915703323,6.66580377223998],"y":[3,3,3,3,3,3,3,3,3,3,3,3],"text":["gs_id: GO:0050830<br />TNFSF8<br />log2FoldChange: -3.035261<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />P2RX7<br />log2FoldChange:  4.656177<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />KLRK1<br />log2FoldChange: -4.911758<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />H2BC21<br />log2FoldChange:  2.359838<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />GSDMD<br />log2FoldChange:  1.999854<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />GBP7<br />log2FoldChange:  7.816237<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />GBP6<br />log2FoldChange:  3.392232<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />GBP4<br />log2FoldChange:  9.752769<br />gs_desc: defense response to Gram-positive bacterium","gs_id: GO:0050830<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: defense response to Gram-positive bacterium"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(149,144,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0050830","legendgroup":"GO:0050830","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,-2.71277184208635,2.17008785297495,3.35897149231688,2.06580501739756,5.36101085730258,2.54482374369772,2.13221835413023,3.38889368923079,4.63675766139974,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,7.68616604774906,2.48794276739928,1.75309967700762,3.84145264917065,4.96182825464539,-3.72976035601486,3.38992601682897,2.55582300669305,2.91426300051913,2.01460366102941],"y":[20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20],"text":["gs_id: GO:0050852<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />TNFRSF21<br />log2FoldChange: -2.712772<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PSME2<br />log2FoldChange:  3.358971<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PSME1<br />log2FoldChange:  2.065805<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PSMB9<br />log2FoldChange:  5.361011<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PSMB8<br />log2FoldChange:  2.544824<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PSMB10<br />log2FoldChange:  2.132218<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />PDE4B<br />log2FoldChange:  3.388894<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />ITK<br />log2FoldChange:  4.636758<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />FYB1<br />log2FoldChange:  2.487943<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />CYLD<br />log2FoldChange:  1.753100<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />CTLA4<br />log2FoldChange:  3.841453<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />CEACAM1<br />log2FoldChange:  4.961828<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />CD28<br />log2FoldChange: -3.729760<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />BTN3A3<br />log2FoldChange:  3.389926<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />BTN3A2<br />log2FoldChange:  2.555823<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />BTN3A1<br />log2FoldChange:  2.914263<br />gs_desc: T cell receptor signaling pathway","gs_id: GO:0050852<br />BTN2A2<br />log2FoldChange:  2.014604<br />gs_desc: T cell receptor signaling pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(185,131,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0050852","legendgroup":"GO:0050852","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,3.84372011500737,2.39661366930567,3.62890094919992,2.02910367999242,2.27897800399411,3.15217022771912,-1.66153540047202,4.07364907920728,3.55795223589057,2.11243275228888,2.7120385342626,2.50768783402577,3.27556759488586,5.04348744419464,2.16563558238417,5.56012373180949,4.79046436725014,4.45361893975568,3.22568381521746,2.10210271975426,2.44590700081427,3.32564098307476,1.92376793164837,3.52431460055161,3.66316736696903,2.52518457338175,3.28111513539738,4.52807122533192,3.67245506894423,7.68616604774906,3.35195380511026,1.90531137609405,2.35887039049424,10.3554913627931,8.76155418580896,3.47849682236114,2.57211029701464,3.52553506636734,4.78812527168352,7.8042530826668],"y":[26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26],"text":["gs_id: GO:0051607<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />TSPAN32<br />log2FoldChange:  3.843720<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />TRIM22<br />log2FoldChange:  2.396614<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />TLR8<br />log2FoldChange:  3.628901<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />TLR7<br />log2FoldChange:  2.029104<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />STAT2<br />log2FoldChange:  2.278978<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />SERINC5<br />log2FoldChange: -1.661535<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />RTP4<br />log2FoldChange:  4.073649<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />RSAD2<br />log2FoldChange:  3.557952<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />PML<br />log2FoldChange:  2.112433<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />PMAIP1<br />log2FoldChange:  2.712039<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />ISG20<br />log2FoldChange:  5.043487<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IL27<br />log2FoldChange:  4.790464<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IL15<br />log2FoldChange:  4.453619<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFITM3<br />log2FoldChange:  2.102103<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFITM2<br />log2FoldChange:  2.445907<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFITM1<br />log2FoldChange:  3.325641<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFIT5<br />log2FoldChange:  1.923768<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFIT3<br />log2FoldChange:  3.524315<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFIT2<br />log2FoldChange:  3.663167<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFIH1<br />log2FoldChange:  2.525185<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFI44L<br />log2FoldChange:  3.281115<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />IFI27<br />log2FoldChange:  4.528071<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />GBP3<br />log2FoldChange:  3.672455<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />FGL2<br />log2FoldChange:  3.351954<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />DTX3L<br />log2FoldChange:  1.905311<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />DDX60<br />log2FoldChange:  2.358870<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />CD40<br />log2FoldChange:  3.478497<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />APOBEC3G<br />log2FoldChange:  2.572110<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />APOBEC3D<br />log2FoldChange:  3.525535<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />APOBEC3A<br />log2FoldChange:  4.788125<br />gs_desc: defense response to virus","gs_id: GO:0051607<br />AIM2<br />log2FoldChange:  7.804253<br />gs_desc: defense response to virus"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(211,118,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0051607","legendgroup":"GO:0051607","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.67252472783529,3.04590598607216,2.39661366930567,3.15217022771912,7.6897534215834,2.11243275228888,1.99922981854563,2.50768783402577,2.45859772219047,3.27556759488586,2.72699778946657,3.04653194988371,2.16563558238417,5.56012373180949,2.88118162177267,3.1034580940225,4.78738236447549,2.30653734457889,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,2.39194269108928,3.0659319792535,2.34031157754719,6.66580377223998,7.68616604774906,6.11224883805674,2.50801757300173,1.93191419065976],"y":[29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29,29],"text":["gs_id: GO:0060333<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />TRIM31<br />log2FoldChange:  3.045906<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />TRIM22<br />log2FoldChange:  2.396614<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />SOCS1<br />log2FoldChange:  7.689753<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />PML<br />log2FoldChange:  2.112433<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />PARP14<br />log2FoldChange:  1.999230<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />NMI<br />log2FoldChange:  2.458598<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />MT2A<br />log2FoldChange:  2.726998<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />CIITA<br />log2FoldChange:  6.112249<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />CAMK2D<br />log2FoldChange:  2.508018<br />gs_desc: interferon-gamma-mediated signaling pathway","gs_id: GO:0060333<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: interferon-gamma-mediated signaling pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(231,107,243,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0060333","legendgroup":"GO:0060333","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.50182698034051,2.98487322842038,2.27897800399411,3.15217022771912,3.55795223589057,2.54482374369772,2.50768783402577,3.27556759488586,5.04348744419464,2.16563558238417,5.56012373180949,2.10210271975426,2.44590700081427,3.32564098307476,3.52431460055161,3.66316736696903,2.6300310875818,4.52807122533192,3.1034580940225,4.78738236447549,2.30653734457889,2.39194269108928,3.0659319792535,2.34031157754719,6.66580377223998],"y":[28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28,28],"text":["gs_id: GO:0060337<br />ZBP1<br />log2FoldChange:  3.501827<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />XAF1<br />log2FoldChange:  2.984873<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />STAT2<br />log2FoldChange:  2.278978<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />RSAD2<br />log2FoldChange:  3.557952<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />PSMB8<br />log2FoldChange:  2.544824<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />ISG20<br />log2FoldChange:  5.043487<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFITM3<br />log2FoldChange:  2.102103<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFITM2<br />log2FoldChange:  2.445907<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFITM1<br />log2FoldChange:  3.325641<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFIT3<br />log2FoldChange:  3.524315<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFIT2<br />log2FoldChange:  3.663167<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFI35<br />log2FoldChange:  2.630031<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />IFI27<br />log2FoldChange:  4.528071<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: type I interferon signaling pathway","gs_id: GO:0060337<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: type I interferon signaling pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(245,100,227,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0060337","legendgroup":"GO:0060337","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.75570678841084,3.48458869155741,-3.20092457252238,10.3554913627931,-2.5740744669526,8.73344469430067,8.76155418580896,-4.47628545862656,2.5641240049707,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491],"y":[13,13,13,13,13,13,13,13,13,13,13,13,13,13,13],"text":["gs_id: GO:0070098<br />PADI2<br />log2FoldChange:  2.755707<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CXCR3<br />log2FoldChange: -3.200925<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CXCL5<br />log2FoldChange: -2.574074<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CXCL11<br />log2FoldChange:  8.733445<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CX3CR1<br />log2FoldChange: -4.476285<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCRL2<br />log2FoldChange:  2.564124<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: chemokine-mediated signaling pathway","gs_id: GO:0070098<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: chemokine-mediated signaling pathway"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(253,97,209,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0070098","legendgroup":"GO:0070098","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,1.98231370454255,3.41669562174808,1.87055840762627,2.17008785297495,3.38889368923079,3.92453990454958,3.4342201708568,1.62110192793976,1.60798004311885,2.34619510075457,-4.91175754423084,2.81740349265852,3.16227124908215,2.88118162177267,10.3554913627931,-2.5740744669526,8.73344469430067,8.76155418580896,-4.47628545862656,2.4722251103732,2.77768451745292,6.58533754666227,2.29474257213188,4.66506611369837,1.84373625404081,7.87008771892115],"y":[15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15],"text":["gs_id: GO:0071222<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />TICAM2<br />log2FoldChange:  1.982314<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />STAP1<br />log2FoldChange:  3.416696<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />SBNO2<br />log2FoldChange:  1.870558<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />RIPK2<br />log2FoldChange:  2.170088<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />PDE4B<br />log2FoldChange:  3.388894<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />PDCD1LG2<br />log2FoldChange:  3.924540<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />NOD2<br />log2FoldChange:  3.434220<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />LYN<br />log2FoldChange:  1.621102<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />LITAF<br />log2FoldChange:  1.607980<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />LILRA2<br />log2FoldChange:  2.346195<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />KLRK1<br />log2FoldChange: -4.911758<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />IL1RN<br />log2FoldChange:  2.817403<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />IL12A<br />log2FoldChange:  3.162271<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CXCL9<br />log2FoldChange: 10.355491<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CXCL5<br />log2FoldChange: -2.574074<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CXCL11<br />log2FoldChange:  8.733445<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CXCL10<br />log2FoldChange:  8.761554<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CX3CR1<br />log2FoldChange: -4.476285<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CMPK2<br />log2FoldChange:  2.472225<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CD80<br />log2FoldChange:  2.777685<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CD274<br />log2FoldChange:  6.585338<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CARD17<br />log2FoldChange:  4.665066<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />CARD16<br />log2FoldChange:  1.843736<br />gs_desc: cellular response to lipopolysaccharide","gs_id: GO:0071222<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: cellular response to lipopolysaccharide"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,98,188,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0071222","legendgroup":"GO:0071222","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.67252472783529,3.04590598607216,2.39661366930567,3.15217022771912,7.6897534215834,2.19406813625637,2.11243275228888,1.99922981854563,2.50768783402577,2.45859772219047,3.27556759488586,2.72699778946657,3.04653194988371,2.16563558238417,5.56012373180949,3.22568381521746,2.88118162177267,3.1034580940225,4.78738236447549,2.30653734457889,5.80989011391077,5.36130941320105,6.64878319497715,5.01846270030917,5.21001525485822,6.80313487971313,4.29792470711347,5.593758623049,2.39194269108928,3.0659319792535,2.34031157754719,7.81623716208599,3.39223239474714,10.3764778869772,9.75276915703323,3.67245506894423,6.66580377223998,7.68616604774906,3.48458869155741,6.11224883805674,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,2.50801757300173,1.93191419065976,7.87008771892115],"y":[4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4],"text":["gs_id: GO:0071346<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />TRIM31<br />log2FoldChange:  3.045906<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />TRIM22<br />log2FoldChange:  2.396614<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />SOCS1<br />log2FoldChange:  7.689753<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />RAB20<br />log2FoldChange:  2.194068<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />PML<br />log2FoldChange:  2.112433<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />PARP14<br />log2FoldChange:  1.999230<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />OAS2<br />log2FoldChange:  2.507688<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />NMI<br />log2FoldChange:  2.458598<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />NLRC5<br />log2FoldChange:  3.275568<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />MT2A<br />log2FoldChange:  2.726998<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />IRF7<br />log2FoldChange:  2.165636<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />IRF1<br />log2FoldChange:  5.560124<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />IL12RB1<br />log2FoldChange:  3.225684<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-G<br />log2FoldChange:  3.103458<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-F<br />log2FoldChange:  4.787382<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-E<br />log2FoldChange:  2.306537<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DRB5<br />log2FoldChange:  5.809890<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DRB1<br />log2FoldChange:  5.361309<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DRA<br />log2FoldChange:  6.648783<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DQB2<br />log2FoldChange:  5.018463<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DQB1<br />log2FoldChange:  5.210015<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DQA1<br />log2FoldChange:  6.803135<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DPB1<br />log2FoldChange:  4.297925<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-DPA1<br />log2FoldChange:  5.593759<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-C<br />log2FoldChange:  2.391943<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-B<br />log2FoldChange:  3.065932<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />HLA-A<br />log2FoldChange:  2.340312<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP7<br />log2FoldChange:  7.816237<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP6<br />log2FoldChange:  3.392232<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP5<br />log2FoldChange: 10.376478<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP4<br />log2FoldChange:  9.752769<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP3<br />log2FoldChange:  3.672455<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CIITA<br />log2FoldChange:  6.112249<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />CAMK2D<br />log2FoldChange:  2.508018<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />B2M<br />log2FoldChange:  1.931914<br />gs_desc: cellular response to interferon-gamma","gs_id: GO:0071346<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: cellular response to interferon-gamma"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,103,164,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0071346","legendgroup":"GO:0071346","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.73017093601696,5.67252472783529,-3.03526122166895,4.41195174127458,3.70967219352629,-2.71277184208635,-3.92611328347182,3.15217022771912,3.35897149231688,2.06580501739756,5.36101085730258,2.54482374369772,2.13221835413023,-4.51075962578397,1.45960495399795,3.04653194988371,2.88118162177267,3.67245506894423,6.66580377223998,7.68616604774906,3.48458869155741,-1.87721474240176,1.75309967700762,-2.65432112372819,3.47849682236114,5.04548843809308,4.23590975302546,2.29474257213188,3.76451516743393,3.92190397731977,4.17978801281491,1.9360556139888,1.84373625404081,-1.98269902259417,7.8042530826668,7.87008771892115],"y":[9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9],"text":["gs_id: GO:0071356<br />ZC3H12A<br />log2FoldChange:  2.730171<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />VCAM1<br />log2FoldChange:  5.672525<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />TNFSF8<br />log2FoldChange: -3.035261<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />TNFSF18<br />log2FoldChange:  4.411952<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />TNFSF13B<br />log2FoldChange:  3.709672<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />TNFRSF21<br />log2FoldChange: -2.712772<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />TNFRSF11A<br />log2FoldChange: -3.926113<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />STAT1<br />log2FoldChange:  3.152170<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PSME2<br />log2FoldChange:  3.358971<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PSME1<br />log2FoldChange:  2.065805<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PSMB9<br />log2FoldChange:  5.361011<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PSMB8<br />log2FoldChange:  2.544824<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PSMB10<br />log2FoldChange:  2.132218<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />PID1<br />log2FoldChange: -4.510760<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />KARS1<br />log2FoldChange:  1.459605<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />JAK2<br />log2FoldChange:  3.046532<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />ICAM1<br />log2FoldChange:  2.881182<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />GBP3<br />log2FoldChange:  3.672455<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />GBP2<br />log2FoldChange:  6.665804<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />GBP1<br />log2FoldChange:  7.686166<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />EDN1<br />log2FoldChange:  3.484589<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />EDA<br />log2FoldChange: -1.877215<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CYLD<br />log2FoldChange:  1.753100<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CRHBP<br />log2FoldChange: -2.654321<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CD40<br />log2FoldChange:  3.478497<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL8<br />log2FoldChange:  5.045488<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL7<br />log2FoldChange:  4.235910<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL5<br />log2FoldChange:  2.294743<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL18<br />log2FoldChange:  3.764515<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL15<br />log2FoldChange:  3.921904<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CCL13<br />log2FoldChange:  4.179788<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CASP4<br />log2FoldChange:  1.936056<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />CARD16<br />log2FoldChange:  1.843736<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />BRCA1<br />log2FoldChange: -1.982699<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />AIM2<br />log2FoldChange:  7.804253<br />gs_desc: cellular response to tumor necrosis factor","gs_id: GO:0071356<br />ACOD1<br />log2FoldChange:  7.870088<br />gs_desc: cellular response to tumor necrosis factor"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(254,110,138,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"GO:0071356","legendgroup":"GO:0071356","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"y":[0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,0.4,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6,30.6],"text":"xintercept: 0","type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(70,130,180,0.4)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":49.7508908068731,"r":7.30593607305936,"b":46.170982131074,"l":437.625570776256},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Enrichment overview - condition IFNg vs naive","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-11.4141256756749,11.4141256756749],"tickmode":"array","ticktext":["-10","-5","0","5","10"],"tickvals":[-10,-5,0,5,10],"categoryorder":"array","categoryarray":["-10","-5","0","5","10"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"log2FoldChange","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,30.6],"tickmode":"array","ticktext":["positive regulation of inflammatory response | GO:0050729","regulation of transcription involved in G1/S transition of m | GO:0000083","defense response to Gram-positive bacterium | GO:0050830","cellular response to interferon-gamma | GO:0071346","G protein-coupled receptor signaling pathway | GO:0007186","positive regulation of interferon-gamma production | GO:0032729","negative regulation of viral genome replication | GO:0045071","positive regulation of T cell proliferation | GO:0042102","cellular response to tumor necrosis factor | GO:0071356","neutrophil chemotaxis | GO:0030593","antigen processing and presentation of exogenous peptide ant | GO:0019886","antigen processing and presentation of exogenous peptide ant | GO:0002479","chemokine-mediated signaling pathway | GO:0070098","regulation of complement activation | GO:0030449","cellular response to lipopolysaccharide | GO:0071222","defense response to protozoan | GO:0042832","double-strand break repair via break-induced replication | GO:0000727","complement activation, classical pathway | GO:0006958","innate immune response | GO:0045087","T cell receptor signaling pathway | GO:0050852","inflammatory response | GO:0006954","response to interferon-gamma | GO:0034341","DNA replication initiation | GO:0006270","antigen processing and presentation via MHC class Ib | GO:0002475","positive regulation of T cell mediated cytotoxicity | GO:0001916","defense response to virus | GO:0051607","antigen processing and presentation of endogenous peptide an | GO:0019885","type I interferon signaling pathway | GO:0060337","interferon-gamma-mediated signaling pathway | GO:0060333","adaptive immune response | GO:0002250"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30],"categoryorder":"array","categoryarray":["positive regulation of inflammatory response | GO:0050729","regulation of transcription involved in G1/S transition of m | GO:0000083","defense response to Gram-positive bacterium | GO:0050830","cellular response to interferon-gamma | GO:0071346","G protein-coupled receptor signaling pathway | GO:0007186","positive regulation of interferon-gamma production | GO:0032729","negative regulation of viral genome replication | GO:0045071","positive regulation of T cell proliferation | GO:0042102","cellular response to tumor necrosis factor | GO:0071356","neutrophil chemotaxis | GO:0030593","antigen processing and presentation of exogenous peptide ant | GO:0019886","antigen processing and presentation of exogenous peptide ant | GO:0002479","chemokine-mediated signaling pathway | GO:0070098","regulation of complement activation | GO:0030449","cellular response to lipopolysaccharide | GO:0071222","defense response to protozoan | GO:0042832","double-strand break repair via break-induced replication | GO:0000727","complement activation, classical pathway | GO:0006958","innate immune response | GO:0045087","T cell receptor signaling pathway | GO:0050852","inflammatory response | GO:0006954","response to interferon-gamma | GO:0034341","DNA replication initiation | GO:0006270","antigen processing and presentation via MHC class Ib | GO:0002475","positive regulation of T cell mediated cytotoxicity | GO:0001916","defense response to virus | GO:0051607","antigen processing and presentation of endogenous peptide an | GO:0019885","type I interferon signaling pathway | GO:0060337","interferon-gamma-mediated signaling pathway | GO:0060333","adaptive immune response | GO:0002250"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"40695387ba3a":{"fill":{},"text":{},"x":{},"y":{},"type":"scatter"},"40692ee31fa0":{"xintercept":{}}},"cur_data":"40695387ba3a","visdat":{"40695387ba3a":["function (y) ","x"],"40692ee31fa0":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>Display the relationship between genesets and genes that belong to them (useful e.g. to see overlap between terms):</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" data-line-number="1"><span class="kw"><a href="../reference/gs_alluvial.html">gs_alluvial</a></span>(<span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb19-2" data-line-number="2">            <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb19-3" data-line-number="3">            <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb19-4" data-line-number="4">            <span class="dt">n_gs =</span> <span class="dv">4</span>)</a></code></pre></div>
<div id="htmlwidget-1b81e92c5b60ff21551d" style="width:960px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1b81e92c5b60ff21551d">{"x":{"visdat":{"4069d9cc4f9":["function () ","plotlyVisDat"]},"cur_data":"4069d9cc4f9","attrs":{"4069d9cc4f9":{"domain":{"x":[0,1],"y":[0,1]},"orientation":"h","valueformat":".0f","node":{"label":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","B2M ","BCL6 ","BTN3A1 ","BTN3A2 ","BTN3A3 ","C1QB ","C1R ","C1RL ","C1S ","C2 ","C3 ","C4A ","C4B ","CD1A ","CD1C ","CD274 ","CD28 ","CD40 ","CD7 ","CD74 ","CD80 ","CEACAM1 ","CLEC10A ","CLEC6A ","CTLA4 ","CTSS ","ERAP2 ","EXO1 ","FGL1 ","FGL2 ","GCNT3 ","GPR183 ","HLA-A ","HLA-B ","HLA-C ","HLA-DMA ","HLA-DMB ","HLA-DOA ","HLA-DOB ","HLA-DPA1 ","HLA-DPB1 ","HLA-DQA1 ","HLA-DQB1 ","HLA-DQB2 ","HLA-DRA ","HLA-DRB1 ","HLA-DRB5 ","HLA-E ","HLA-F ","HLA-G ","ICAM1 ","IL12A ","IL12RB1 ","IL18BP ","IL27 ","IRF1 ","IRF7 ","ITK ","JAK2 ","JAK3 ","KLRK1 ","LAG3 ","LAMP3 ","LILRA1 ","LILRB3 ","LYN ","MCOLN2 ","P2RX7 ","PDCD1 ","PDCD1LG2 ","RELB ","RIPK2 ","RNF19B ","RSAD2 ","SERPING1 ","SIT1 ","SLAMF1 ","SLAMF6 ","SLAMF7 ","SLC11A1 ","TAP1 ","TAP2 ","TBX21 ","TLR8 ","TNFRSF11A ","TNFRSF21 ","TNFSF13B ","TNFSF18 ","ZC3H12A ","CAMK2D ","CIITA ","GBP1 ","GBP2 ","MT2A ","NLRC5 ","NMI ","OAS2 ","PARP14 ","PML ","SOCS1 ","STAT1 ","TRIM22 ","TRIM31 ","VCAM1 ","IFI27 ","IFI35 ","IFIT2 ","IFIT3 ","IFITM1 ","IFITM2 ","IFITM3 ","ISG20 ","PSMB8 ","STAT2 ","XAF1 ","ZBP1 ","TAPBP "],"color":["#440154FF","#31688EFF","#35B779FF","#FDE725FF","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue"],"pad":15,"thickness":20,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,3],"target":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,4,93,94,95,96,36,37,38,43,44,45,46,47,48,49,50,51,52,53,54,59,60,62,97,98,99,100,101,102,103,104,105,106,107,96,36,37,38,51,52,53,108,109,110,111,112,113,114,59,60,115,98,100,116,77,104,117,118,119,4,30,36,37,38,51,52,53,84,85,120],"value":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"sankey"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"Geneset-Gene Sankey Diagram","font":{"size":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"showSendToCloud":false},"data":[{"domain":{"x":[0,1],"y":[0,1]},"orientation":"h","valueformat":".0f","node":{"label":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","B2M ","BCL6 ","BTN3A1 ","BTN3A2 ","BTN3A3 ","C1QB ","C1R ","C1RL ","C1S ","C2 ","C3 ","C4A ","C4B ","CD1A ","CD1C ","CD274 ","CD28 ","CD40 ","CD7 ","CD74 ","CD80 ","CEACAM1 ","CLEC10A ","CLEC6A ","CTLA4 ","CTSS ","ERAP2 ","EXO1 ","FGL1 ","FGL2 ","GCNT3 ","GPR183 ","HLA-A ","HLA-B ","HLA-C ","HLA-DMA ","HLA-DMB ","HLA-DOA ","HLA-DOB ","HLA-DPA1 ","HLA-DPB1 ","HLA-DQA1 ","HLA-DQB1 ","HLA-DQB2 ","HLA-DRA ","HLA-DRB1 ","HLA-DRB5 ","HLA-E ","HLA-F ","HLA-G ","ICAM1 ","IL12A ","IL12RB1 ","IL18BP ","IL27 ","IRF1 ","IRF7 ","ITK ","JAK2 ","JAK3 ","KLRK1 ","LAG3 ","LAMP3 ","LILRA1 ","LILRB3 ","LYN ","MCOLN2 ","P2RX7 ","PDCD1 ","PDCD1LG2 ","RELB ","RIPK2 ","RNF19B ","RSAD2 ","SERPING1 ","SIT1 ","SLAMF1 ","SLAMF6 ","SLAMF7 ","SLC11A1 ","TAP1 ","TAP2 ","TBX21 ","TLR8 ","TNFRSF11A ","TNFRSF21 ","TNFSF13B ","TNFSF18 ","ZC3H12A ","CAMK2D ","CIITA ","GBP1 ","GBP2 ","MT2A ","NLRC5 ","NMI ","OAS2 ","PARP14 ","PML ","SOCS1 ","STAT1 ","TRIM22 ","TRIM31 ","VCAM1 ","IFI27 ","IFI35 ","IFIT2 ","IFIT3 ","IFITM1 ","IFITM2 ","IFITM3 ","ISG20 ","PSMB8 ","STAT2 ","XAF1 ","ZBP1 ","TAPBP "],"color":["#440154FF","#31688EFF","#35B779FF","#FDE725FF","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue","steelblue"],"pad":15,"thickness":20,"line":{"color":"black","width":0.5}},"link":{"source":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,3],"target":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,4,93,94,95,96,36,37,38,43,44,45,46,47,48,49,50,51,52,53,54,59,60,62,97,98,99,100,101,102,103,104,105,106,107,96,36,37,38,51,52,53,108,109,110,111,112,113,114,59,60,115,98,100,116,77,104,117,118,119,4,30,36,37,38,51,52,53,84,85,120],"value":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]},"type":"sankey","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>Display a bipartite graph where genesets and genes are included, and make it interactive:</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb20-1" data-line-number="1">ggs &lt;-<span class="st"> </span><span class="kw"><a href="../reference/ggs_graph.html">ggs_graph</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb20-2" data-line-number="2">                 <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb20-3" data-line-number="3">                 anno_df,</a>
<a class="sourceLine" id="cb20-4" data-line-number="4">                 <span class="dt">n_gs =</span> <span class="dv">20</span>)</a>
<a class="sourceLine" id="cb20-5" data-line-number="5">ggs</a>
<a class="sourceLine" id="cb20-6" data-line-number="6"><span class="co">#&gt; IGRAPH a73f3a0 UN-- 266 656 -- </span></a>
<a class="sourceLine" id="cb20-7" data-line-number="7"><span class="co">#&gt; + attr: name (v/c), nodetype (v/c), shape (v/c), color (v/c), title (v/c)</span></a>
<a class="sourceLine" id="cb20-8" data-line-number="8"><span class="co">#&gt; + edges from a73f3a0 (vertex names):</span></a>
<a class="sourceLine" id="cb20-9" data-line-number="9"><span class="co">#&gt;  [1] adaptive immune response--B2M     adaptive immune response--BCL6   </span></a>
<a class="sourceLine" id="cb20-10" data-line-number="10"><span class="co">#&gt;  [3] adaptive immune response--BTN3A1  adaptive immune response--BTN3A2 </span></a>
<a class="sourceLine" id="cb20-11" data-line-number="11"><span class="co">#&gt;  [5] adaptive immune response--BTN3A3  adaptive immune response--C1QB   </span></a>
<a class="sourceLine" id="cb20-12" data-line-number="12"><span class="co">#&gt;  [7] adaptive immune response--C1R     adaptive immune response--C1RL   </span></a>
<a class="sourceLine" id="cb20-13" data-line-number="13"><span class="co">#&gt;  [9] adaptive immune response--C1S     adaptive immune response--C2     </span></a>
<a class="sourceLine" id="cb20-14" data-line-number="14"><span class="co">#&gt; [11] adaptive immune response--C3      adaptive immune response--C4A    </span></a>
<a class="sourceLine" id="cb20-15" data-line-number="15"><span class="co">#&gt; [13] adaptive immune response--C4B     adaptive immune response--CD1A   </span></a>
<a class="sourceLine" id="cb20-16" data-line-number="16"><span class="co">#&gt; [15] adaptive immune response--CD1C    adaptive immune response--CD274  </span></a>
<a class="sourceLine" id="cb20-17" data-line-number="17"><span class="co">#&gt; + ... omitted several edges</span></a>
<a class="sourceLine" id="cb20-18" data-line-number="18"></a>
<a class="sourceLine" id="cb20-19" data-line-number="19"><span class="co"># could be viewed interactively with</span></a>
<a class="sourceLine" id="cb20-20" data-line-number="20"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(visNetwork)</a>
<a class="sourceLine" id="cb20-21" data-line-number="21"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(magrittr)</a>
<a class="sourceLine" id="cb20-22" data-line-number="22">ggs <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb20-23" data-line-number="23"><span class="st">  </span><span class="kw"><a href="https://rdrr.io/pkg/visNetwork/man/visNetwork-igraph.html">visIgraph</a></span>() <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb20-24" data-line-number="24"><span class="st">  </span><span class="kw"><a href="https://rdrr.io/pkg/visNetwork/man/visOptions.html">visOptions</a></span>(<span class="dt">highlightNearest =</span> <span class="kw"><a href="https://rdrr.io/r/base/list.html">list</a></span>(<span class="dt">enabled =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb20-25" data-line-number="25">                                     <span class="dt">degree =</span> <span class="dv">1</span>,</a>
<a class="sourceLine" id="cb20-26" data-line-number="26">                                     <span class="dt">hover =</span> <span class="ot">TRUE</span>),</a>
<a class="sourceLine" id="cb20-27" data-line-number="27">             <span class="dt">nodesIdSelection =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<div id="htmlwidget-e203227d8707242b8bb0" style="width:960px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-e203227d8707242b8bb0">{"x":{"nodes":{"id":["adaptive immune response","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation via MHC class Ib","cellular response to lipopolysaccharide","chemokine-mediated signaling pathway","complement activation, classical pathway","defense response to protozoan","defense response to virus","DNA replication initiation","double-strand break repair via break-induced replication","inflammatory response","innate immune response","interferon-gamma-mediated signaling pathway","positive regulation of T cell mediated cytotoxicity","regulation of complement activation","response to interferon-gamma","T cell receptor signaling pathway","type I interferon signaling pathway","ACOD1","ADAM8","ADGRE5","ADORA3","AGT","AIM2","AOC3","APOBEC3A","APOBEC3D","APOBEC3G","APOL1","APOL2","APOL3","APPL2","B2M","BATF2","BCL6","BTN2A2","BTN3A1","BTN3A2","BTN3A3","C1QB","C1R","C1RL","C1S","C2","C3","C4A","C4B","CALCOCO2","CAMK2D","CARD16","CARD17","CASP4","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CCNE2","CCRL2","CD1A","CD1C","CD274","CD28","CD300LF","CD40","CD7","CD74","CD80","CDC45","CDC6","CDH5","CDT1","CEACAM1","CFB","CFH","CIITA","CLEC10A","CLEC6A","CMPK2","COLEC12","CR1L","CRHBP","CTLA4","CTSS","CX3CR1","CXCL10","CXCL11","CXCL5","CXCL9","CXCR3","CYLD","CYSLTR1","DDX60","DTX3L","EDN1","ERAP2","EXO1","FCN1","FGL1","FGL2","FYB1","GBP1","GBP2","GBP3","GBP4","GBP5","GBP6","GBP7","GCH1","GCNT3","GGT5","GHRL","GINS2","GPR183","GRAMD4","GSDMD","H2BC21","HGF","HLA-A","HLA-B","HLA-C","HLA-DMA","HLA-DMB","HLA-DOA","HLA-DOB","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","ICAM2","IDO1","IFI27","IFI35","IFI44L","IFIH1","IFIT2","IFIT3","IFIT5","IFITM1","IFITM2","IFITM3","IL12A","IL12RB1","IL15","IL18BP","IL1RN","IL22RA2","IL27","IRF1","IRF7","ISG20","ITK","JAK2","JAK3","KARS1","KIF11","KIF15","KIF26A","KLF4","KLRK1","LAG3","LAMP3","LILRA1","LILRA2","LILRB3","LITAF","LRRK2","LY75","LYN","MCM10","MCM2","MCM4","MCM5","MCM6","MCM7","MCOLN2","MEFV","MMP25","MMP9","MSRB1","MT2A","MUC1","NCF1","NLRC5","NLRP12","NMI","NOD2","NUB1","OAS2","OPTN","ORC1","OSM","P2RX7","PADI2","PARP14","PDCD1","PDCD1LG2","PDE4B","PMAIP1","PML","POLB","POLE2","PSMB10","PSMB8","PSMB9","PSME1","PSME2","PTGER4","PYHIN1","RAB20","RELB","RIPK2","RNF19B","RSAD2","RTP4","SBNO2","SERINC5","SERPING1","SIT1","SLAMF1","SLAMF6","SLAMF7","SLAMF8","SLC11A1","SOCS1","STAP1","STAT1","STAT2","SUCNR1","TAP1","TAP2","TAPBP","TBX21","TGM2","TICAM2","TICRR","TIFA","TLR10","TLR5","TLR7","TLR8","TNFAIP6","TNFRSF11A","TNFRSF21","TNFSF13B","TNFSF18","TRAFD1","TRIM22","TRIM31","TSPAN32","UBD","VCAM1","XAF1","ZBP1","ZC3H12A"],"nodetype":["GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","GeneSet","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature","Feature"],"shape":["box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","box","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse","ellipse"],"color":["gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","gold","#A5002666","#6298C666","#D7312766","#598DC066","#AF092666","#A5002666","#598DC066","#A5002666","#B9132666","#E9563866","#A5002666","#AF092666","#A5002666","#A3D2E566","#F88A5066","#A5002666","#F88A5066","#F88A5066","#DD3D2D66","#EF633E66","#C31D2666","#C31D2666","#E34A3366","#F67C4A66","#EF633E66","#F67C4A66","#F9975666","#A5002666","#A5002666","#E34A3366","#EF633E66","#F9975666","#A5002666","#F88A5066","#A5002666","#A5002666","#AF092666","#F46F4466","#A5002666","#A5002666","#31369566","#E9563866","#81B7D666","#97C9E066","#A5002666","#35429B66","#E9563866","#C31D2666","#A5002666","#A5002666","#E34A3366","#4576B466","#598DC066","#598DC066","#598DC066","#A5002666","#A5002666","#A5002666","#A5002666","#F88A5066","#A5002666","#EF633E66","#4576B466","#A5002666","#6298C666","#A5002666","#F9975666","#31369566","#A5002666","#A5002666","#6298C666","#A5002666","#4169AE66","#FBA45C66","#81B7D666","#F46F4466","#F9975666","#C31D2666","#B9132666","#6298C666","#6CA3CC66","#76AED166","#CD272666","#EF633E66","#A5002666","#A5002666","#B9132666","#A5002666","#A5002666","#C31D2666","#A5002666","#A5002666","#DD3D2D66","#E34A3366","#394FA166","#4F81BA66","#394FA166","#81B7D666","#F88A5066","#F46F4466","#A5002666","#F46F4466","#D7312766","#F46F4466","#A5002666","#AF092666","#A5002666","#B9132666","#A5002666","#A5002666","#A5002666","#A5002666","#A5002666","#A5002666","#A5002666","#A5002666","#F46F4466","#A5002666","#D7312766","#DD3D2D66","#F88A5066","#A5002666","#A5002666","#E9563866","#CD272666","#EF633E66","#B9132666","#B9132666","#F88A5066","#CD272666","#EF633E66","#F67C4A66","#D7312766","#CD272666","#A5002666","#A5002666","#E34A3366","#4F81BA66","#A5002666","#A5002666","#F67C4A66","#A5002666","#A5002666","#D7312766","#E34A3366","#FDB16366","#81B7D666","#6298C666","#76AED166","#E34A3366","#31369566","#A5002666","#A5002666","#DD3D2D66","#F46F4466","#E9563866","#FBA45C66","#A5002666","#F88A5066","#FBA45C66","#4169AE66","#81B7D666","#81B7D666","#8CC0DB66","#76AED166","#76AED166","#A5002666","#D7312766","#F67C4A66","#76AED166","#E9563866","#E34A3366","#A5002666","#F67C4A66","#CD272666","#6CA3CC66","#EF633E66","#C31D2666","#EF633E66","#EF633E66","#AF092666","#81B7D666","#8CC0DB66","#A5002666","#E34A3366","#F88A5066","#B9132666","#A5002666","#C31D2666","#E9563866","#F67C4A66","#F9975666","#76AED166","#F67C4A66","#EF633E66","#A5002666","#F88A5066","#CD272666","#F67C4A66","#F46F4466","#F67C4A66","#F88A5066","#F67C4A66","#E9563866","#B9132666","#A5002666","#F9975666","#A3D2E566","#A5002666","#4F81BA66","#A5002666","#F46F4466","#A5002666","#F9975666","#76AED166","#A5002666","#C31D2666","#D7312766","#F46F4466","#C31D2666","#A5002666","#C31D2666","#F9975666","#A5002666","#E9563866","#F88A5066","#598DC066","#A5002666","#E34A3366","#81B7D666","#F88A5066","#B9132666","#A5002666","#31369566","#6298C666","#AF092666","#A5002666","#E34A3366","#F46F4466","#D7312766","#A5002666","#A5002666","#A5002666","#DD3D2D66","#C31D2666","#E34A3366"],"title":["<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0002250\" target=\"_blank\">GO:0002250<\/a><\/h4><br>adaptive immune response<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0019885\" target=\"_blank\">GO:0019885<\/a><\/h4><br>antigen processing and presentation of endogenous peptide antigen via MHC class I<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0002479\" target=\"_blank\">GO:0002479<\/a><\/h4><br>antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0019886\" target=\"_blank\">GO:0019886<\/a><\/h4><br>antigen processing and presentation of exogenous peptide antigen via MHC class II<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0002475\" target=\"_blank\">GO:0002475<\/a><\/h4><br>antigen processing and presentation via MHC class Ib<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0071222\" target=\"_blank\">GO:0071222<\/a><\/h4><br>cellular response to lipopolysaccharide<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0070098\" target=\"_blank\">GO:0070098<\/a><\/h4><br>chemokine-mediated signaling pathway<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0006958\" target=\"_blank\">GO:0006958<\/a><\/h4><br>complement activation, classical pathway<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0042832\" target=\"_blank\">GO:0042832<\/a><\/h4><br>defense response to protozoan<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0051607\" target=\"_blank\">GO:0051607<\/a><\/h4><br>defense response to virus<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0006270\" target=\"_blank\">GO:0006270<\/a><\/h4><br>DNA replication initiation<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0000727\" target=\"_blank\">GO:0000727<\/a><\/h4><br>double-strand break repair via break-induced replication<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0006954\" target=\"_blank\">GO:0006954<\/a><\/h4><br>inflammatory response<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0045087\" target=\"_blank\">GO:0045087<\/a><\/h4><br>innate immune response<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0060333\" target=\"_blank\">GO:0060333<\/a><\/h4><br>interferon-gamma-mediated signaling pathway<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0001916\" target=\"_blank\">GO:0001916<\/a><\/h4><br>positive regulation of T cell mediated cytotoxicity<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0030449\" target=\"_blank\">GO:0030449<\/a><\/h4><br>regulation of complement activation<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0034341\" target=\"_blank\">GO:0034341<\/a><\/h4><br>response to interferon-gamma<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0050852\" target=\"_blank\">GO:0050852<\/a><\/h4><br>T cell receptor signaling pathway<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4><a href=\"http://amigo.geneontology.org/amigo/term/GO:0060337\" target=\"_blank\">GO:0060337<\/a><\/h4><br>type I interferon signaling pathway<br><br>An immune response mediated by cells expressing<br>specific receptors for antigen produced through a<br>somatic diversification process, and allowing for<br>an enhanced secondary response to subsequent<br>exposures to the same antigen (immunological<br>memory).<br>A series of molecular signals initiated by the<br>binding of interferon-gamma to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription. Interferon gamma is the only<br>member of the type II interferon found so far.<br>A series of molecular signals initiated by the<br>binding of a type I interferon to a receptor on<br>the surface of a cell, and ending with regulation<br>of a downstream cellular process, e.g.<br>transcription. Type I interferons include the<br>interferon-alpha, beta, delta, episilon, zeta,<br>kappa, tau, and omega gene families.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of endogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein. Class I here refers to classical class I<br>molecules.<br>Reactions triggered in response to the presence<br>of a virus that act to protect the cell or<br>organism.<br>Any process that activates or increases the<br>frequency, rate or extent of T cell mediated<br>cytotoxicity.<br>The process in which an antigen-presenting cell<br>expresses antigen (peptide or lipid) on its cell<br>surface in association with an MHC class Ib<br>protein complex. Class Ib here refers to<br>non-classical class I molecules, such as those of<br>the CD1 or HLA-E gene families.<br>The process in which DNA-dependent DNA<br>replication is started; this begins with the ATP<br>dependent loading of an initiator complex onto<br>the DNA, this is followed by DNA melting and<br>helicase activity. In bacteria, the gene products<br>that enable the helicase activity are loaded<br>after the initial melting and in archaea and<br>eukaryotes, the gene products that enable the<br>helicase activity are inactive when they are<br>loaded and subsequently activate.<br>Any process that results in a change in state or<br>activity of a cell or an organism (in terms of<br>movement, secretion, enzyme production, gene<br>expression, etc.) as a result of an<br>interferon-gamma stimulus. Interferon-gamma is<br>also known as type II interferon.<br>The immediate defensive reaction (by vertebrate<br>tissue) to infection or injury caused by chemical<br>or physical agents. The process is characterized<br>by local vasodilation, extravasation of plasma<br>into intercellular spaces and accumulation of<br>white blood cells and macrophages.<br>A series of molecular signals initiated by the<br>cross-linking of an antigen receptor on a T cell.<br>Innate immune responses are defense responses<br>mediated by germline encoded components that<br>directly recognize components of potential<br>pathogens.<br>Any process involved in the activation of any of<br>the steps of the classical pathway of the<br>complement cascade which allows for the direct<br>killing of microbes, the disposal of immune<br>complexes, and the regulation of other immune<br>processes.<br>The error-free repair of a double-strand break in<br>DNA in which the centromere-proximal end of a<br>broken chromosome searches for a homologous<br>region in an intact chromosome. DNA synthesis<br>initiates from the 3' end of the invading DNA<br>strand, using the intact chromosome as the<br>template, and progresses to the end of the<br>chromosome.<br>Reactions triggered in response to the presence<br>of a protozoan that act to protect the cell or<br>organism.<br>Any process that results in a change in state or<br>activity of a cell (in terms of movement,<br>secretion, enzyme production, gene expression,<br>etc.) as a result of a lipopolysaccharide<br>stimulus; lipopolysaccharide is a major component<br>of the cell wall of gram-negative bacteria.<br>Any process that modulates the frequency, rate or<br>extent of complement activation.<br>A series of molecular signals initiated by the<br>binding of a chemokine to a receptor on the<br>surface of a cell, and ending with regulation of<br>a downstream cellular process, e.g.<br>transcription.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class I protein complex following intracellular<br>transport via a TAP (transporter associated with<br>antigen processing) pathway. The peptide is<br>typically a fragment of a larger exogenous<br>protein which has been degraded within the cell<br>and is dependent on TAP transport from the<br>cytosol to ER for association with the MHC class<br>I molecule. Class I here refers to classical<br>class I molecules.<br>The process in which an antigen-presenting cell<br>expresses a peptide antigen of exogenous origin<br>on its cell surface in association with an MHC<br>class II protein complex. The peptide antigen is<br>typically, but not always, processed from a whole<br>protein.","<h4>ACOD1<\/h4><br>logFC =  7.87","<h4>ADAM8<\/h4><br>logFC = -2.67","<h4>ADGRE5<\/h4><br>logFC =  3.05","<h4>ADORA3<\/h4><br>logFC = -2.72","<h4>AGT<\/h4><br>logFC =  3.77","<h4>AIM2<\/h4><br>logFC =  7.80","<h4>AOC3<\/h4><br>logFC = -2.83","<h4>APOBEC3A<\/h4><br>logFC =  4.79","<h4>APOBEC3D<\/h4><br>logFC =  3.53","<h4>APOBEC3G<\/h4><br>logFC =  2.57","<h4>APOL1<\/h4><br>logFC =  7.14","<h4>APOL2<\/h4><br>logFC =  3.81","<h4>APOL3<\/h4><br>logFC =  7.98","<h4>APPL2<\/h4><br>logFC = -1.62","<h4>B2M<\/h4><br>logFC =  1.93","<h4>BATF2<\/h4><br>logFC =  4.69","<h4>BCL6<\/h4><br>logFC =  1.92","<h4>BTN2A2<\/h4><br>logFC =  2.01","<h4>BTN3A1<\/h4><br>logFC =  2.91","<h4>BTN3A2<\/h4><br>logFC =  2.56","<h4>BTN3A3<\/h4><br>logFC =  3.39","<h4>C1QB<\/h4><br>logFC =  3.52","<h4>C1R<\/h4><br>logFC =  2.80","<h4>C1RL<\/h4><br>logFC =  2.08","<h4>C1S<\/h4><br>logFC =  2.54","<h4>C2<\/h4><br>logFC =  2.16","<h4>C3<\/h4><br>logFC =  1.89","<h4>C4A<\/h4><br>logFC =  5.03","<h4>C4B<\/h4><br>logFC =  6.31","<h4>CALCOCO2<\/h4><br>logFC =  2.83","<h4>CAMK2D<\/h4><br>logFC =  2.51","<h4>CARD16<\/h4><br>logFC =  1.84","<h4>CARD17<\/h4><br>logFC =  4.67","<h4>CASP4<\/h4><br>logFC =  1.94","<h4>CCL13<\/h4><br>logFC =  4.18","<h4>CCL15<\/h4><br>logFC =  3.92","<h4>CCL18<\/h4><br>logFC =  3.76","<h4>CCL5<\/h4><br>logFC =  2.29","<h4>CCL7<\/h4><br>logFC =  4.24","<h4>CCL8<\/h4><br>logFC =  5.05","<h4>CCNE2<\/h4><br>logFC = -3.86","<h4>CCRL2<\/h4><br>logFC =  2.56","<h4>CD1A<\/h4><br>logFC = -2.13","<h4>CD1C<\/h4><br>logFC = -1.84","<h4>CD274<\/h4><br>logFC =  6.59","<h4>CD28<\/h4><br>logFC = -3.73","<h4>CD300LF<\/h4><br>logFC =  2.60","<h4>CD40<\/h4><br>logFC =  3.48","<h4>CD7<\/h4><br>logFC =  4.54","<h4>CD74<\/h4><br>logFC =  4.15","<h4>CD80<\/h4><br>logFC =  2.78","<h4>CDC45<\/h4><br>logFC = -3.11","<h4>CDC6<\/h4><br>logFC = -2.85","<h4>CDH5<\/h4><br>logFC = -2.82","<h4>CDT1<\/h4><br>logFC = -2.72","<h4>CEACAM1<\/h4><br>logFC =  4.96","<h4>CFB<\/h4><br>logFC =  4.33","<h4>CFH<\/h4><br>logFC =  4.99","<h4>CIITA<\/h4><br>logFC =  6.11","<h4>CLEC10A<\/h4><br>logFC =  1.93","<h4>CLEC6A<\/h4><br>logFC =  5.78","<h4>CMPK2<\/h4><br>logFC =  2.47","<h4>COLEC12<\/h4><br>logFC = -3.07","<h4>CR1L<\/h4><br>logFC =  4.33","<h4>CRHBP<\/h4><br>logFC = -2.65","<h4>CTLA4<\/h4><br>logFC =  3.84","<h4>CTSS<\/h4><br>logFC =  1.80","<h4>CX3CR1<\/h4><br>logFC = -4.48","<h4>CXCL10<\/h4><br>logFC =  8.76","<h4>CXCL11<\/h4><br>logFC =  8.73","<h4>CXCL5<\/h4><br>logFC = -2.57","<h4>CXCL9<\/h4><br>logFC = 10.36","<h4>CXCR3<\/h4><br>logFC = -3.20","<h4>CYLD<\/h4><br>logFC =  1.75","<h4>CYSLTR1<\/h4><br>logFC = -2.12","<h4>DDX60<\/h4><br>logFC =  2.36","<h4>DTX3L<\/h4><br>logFC =  1.91","<h4>EDN1<\/h4><br>logFC =  3.48","<h4>ERAP2<\/h4><br>logFC =  3.65","<h4>EXO1<\/h4><br>logFC = -2.59","<h4>FCN1<\/h4><br>logFC = -2.55","<h4>FGL1<\/h4><br>logFC = -2.32","<h4>FGL2<\/h4><br>logFC =  3.35","<h4>FYB1<\/h4><br>logFC =  2.49","<h4>GBP1<\/h4><br>logFC =  7.69","<h4>GBP2<\/h4><br>logFC =  6.67","<h4>GBP3<\/h4><br>logFC =  3.67","<h4>GBP4<\/h4><br>logFC =  9.75","<h4>GBP5<\/h4><br>logFC = 10.38","<h4>GBP6<\/h4><br>logFC =  3.39","<h4>GBP7<\/h4><br>logFC =  7.82","<h4>GCH1<\/h4><br>logFC =  4.74","<h4>GCNT3<\/h4><br>logFC =  3.02","<h4>GGT5<\/h4><br>logFC =  2.87","<h4>GHRL<\/h4><br>logFC = -3.53","<h4>GINS2<\/h4><br>logFC = -2.91","<h4>GPR183<\/h4><br>logFC = -3.66","<h4>GRAMD4<\/h4><br>logFC = -2.15","<h4>GSDMD<\/h4><br>logFC =  2.00","<h4>H2BC21<\/h4><br>logFC =  2.36","<h4>HGF<\/h4><br>logFC =  4.06","<h4>HLA-A<\/h4><br>logFC =  2.34","<h4>HLA-B<\/h4><br>logFC =  3.07","<h4>HLA-C<\/h4><br>logFC =  2.39","<h4>HLA-DMA<\/h4><br>logFC =  4.07","<h4>HLA-DMB<\/h4><br>logFC =  3.82","<h4>HLA-DOA<\/h4><br>logFC =  5.78","<h4>HLA-DOB<\/h4><br>logFC =  3.56","<h4>HLA-DPA1<\/h4><br>logFC =  5.59","<h4>HLA-DPB1<\/h4><br>logFC =  4.30","<h4>HLA-DQA1<\/h4><br>logFC =  6.80","<h4>HLA-DQB1<\/h4><br>logFC =  5.21","<h4>HLA-DQB2<\/h4><br>logFC =  5.02","<h4>HLA-DRA<\/h4><br>logFC =  6.65","<h4>HLA-DRB1<\/h4><br>logFC =  5.36","<h4>HLA-DRB5<\/h4><br>logFC =  5.81","<h4>HLA-E<\/h4><br>logFC =  2.31","<h4>HLA-F<\/h4><br>logFC =  4.79","<h4>HLA-G<\/h4><br>logFC =  3.10","<h4>ICAM1<\/h4><br>logFC =  2.88","<h4>ICAM2<\/h4><br>logFC =  1.95","<h4>IDO1<\/h4><br>logFC =  9.55","<h4>IFI27<\/h4><br>logFC =  4.53","<h4>IFI35<\/h4><br>logFC =  2.63","<h4>IFI44L<\/h4><br>logFC =  3.28","<h4>IFIH1<\/h4><br>logFC =  2.53","<h4>IFIT2<\/h4><br>logFC =  3.66","<h4>IFIT3<\/h4><br>logFC =  3.52","<h4>IFIT5<\/h4><br>logFC =  1.92","<h4>IFITM1<\/h4><br>logFC =  3.33","<h4>IFITM2<\/h4><br>logFC =  2.45","<h4>IFITM3<\/h4><br>logFC =  2.10","<h4>IL12A<\/h4><br>logFC =  3.16","<h4>IL12RB1<\/h4><br>logFC =  3.23","<h4>IL15<\/h4><br>logFC =  4.45","<h4>IL18BP<\/h4><br>logFC =  4.06","<h4>IL1RN<\/h4><br>logFC =  2.82","<h4>IL22RA2<\/h4><br>logFC = -2.93","<h4>IL27<\/h4><br>logFC =  4.79","<h4>IRF1<\/h4><br>logFC =  5.56","<h4>IRF7<\/h4><br>logFC =  2.17","<h4>ISG20<\/h4><br>logFC =  5.04","<h4>ITK<\/h4><br>logFC =  4.64","<h4>JAK2<\/h4><br>logFC =  3.05","<h4>JAK3<\/h4><br>logFC =  2.81","<h4>KARS1<\/h4><br>logFC =  1.46","<h4>KIF11<\/h4><br>logFC = -2.18","<h4>KIF15<\/h4><br>logFC = -2.67","<h4>KIF26A<\/h4><br>logFC = -2.29","<h4>KLF4<\/h4><br>logFC =  2.72","<h4>KLRK1<\/h4><br>logFC = -4.91","<h4>LAG3<\/h4><br>logFC =  4.61","<h4>LAMP3<\/h4><br>logFC =  5.22","<h4>LILRA1<\/h4><br>logFC =  2.97","<h4>LILRA2<\/h4><br>logFC =  2.35","<h4>LILRB3<\/h4><br>logFC =  2.64","<h4>LITAF<\/h4><br>logFC =  1.61","<h4>LRRK2<\/h4><br>logFC =  4.61","<h4>LY75<\/h4><br>logFC =  1.95","<h4>LYN<\/h4><br>logFC =  1.62","<h4>MCM10<\/h4><br>logFC = -3.33","<h4>MCM2<\/h4><br>logFC = -2.21","<h4>MCM4<\/h4><br>logFC = -2.23","<h4>MCM5<\/h4><br>logFC = -1.95","<h4>MCM6<\/h4><br>logFC = -2.38","<h4>MCM7<\/h4><br>logFC = -2.33","<h4>MCOLN2<\/h4><br>logFC =  3.96","<h4>MEFV<\/h4><br>logFC =  3.08","<h4>MMP25<\/h4><br>logFC =  2.09","<h4>MMP9<\/h4><br>logFC = -2.36","<h4>MSRB1<\/h4><br>logFC =  2.67","<h4>MT2A<\/h4><br>logFC =  2.73","<h4>MUC1<\/h4><br>logFC =  3.92","<h4>NCF1<\/h4><br>logFC =  2.09","<h4>NLRC5<\/h4><br>logFC =  3.28","<h4>NLRP12<\/h4><br>logFC = -2.40","<h4>NMI<\/h4><br>logFC =  2.46","<h4>NOD2<\/h4><br>logFC =  3.43","<h4>NUB1<\/h4><br>logFC =  2.42","<h4>OAS2<\/h4><br>logFC =  2.51","<h4>OPTN<\/h4><br>logFC =  3.82","<h4>ORC1<\/h4><br>logFC = -2.16","<h4>OSM<\/h4><br>logFC = -2.04","<h4>P2RX7<\/h4><br>logFC =  4.66","<h4>PADI2<\/h4><br>logFC =  2.76","<h4>PARP14<\/h4><br>logFC =  2.00","<h4>PDCD1<\/h4><br>logFC =  3.61","<h4>PDCD1LG2<\/h4><br>logFC =  3.92","<h4>PDE4B<\/h4><br>logFC =  3.39","<h4>PMAIP1<\/h4><br>logFC =  2.71","<h4>PML<\/h4><br>logFC =  2.11","<h4>POLB<\/h4><br>logFC =  1.83","<h4>POLE2<\/h4><br>logFC = -2.34","<h4>PSMB10<\/h4><br>logFC =  2.13","<h4>PSMB8<\/h4><br>logFC =  2.54","<h4>PSMB9<\/h4><br>logFC =  5.36","<h4>PSME1<\/h4><br>logFC =  2.07","<h4>PSME2<\/h4><br>logFC =  3.36","<h4>PTGER4<\/h4><br>logFC =  2.23","<h4>PYHIN1<\/h4><br>logFC =  2.32","<h4>RAB20<\/h4><br>logFC =  2.19","<h4>RELB<\/h4><br>logFC =  2.02","<h4>RIPK2<\/h4><br>logFC =  2.17","<h4>RNF19B<\/h4><br>logFC =  2.58","<h4>RSAD2<\/h4><br>logFC =  3.56","<h4>RTP4<\/h4><br>logFC =  4.07","<h4>SBNO2<\/h4><br>logFC =  1.87","<h4>SERINC5<\/h4><br>logFC = -1.66","<h4>SERPING1<\/h4><br>logFC =  5.15","<h4>SIT1<\/h4><br>logFC = -3.02","<h4>SLAMF1<\/h4><br>logFC =  4.85","<h4>SLAMF6<\/h4><br>logFC =  2.38","<h4>SLAMF7<\/h4><br>logFC =  6.06","<h4>SLAMF8<\/h4><br>logFC =  1.89","<h4>SLC11A1<\/h4><br>logFC = -2.26","<h4>SOCS1<\/h4><br>logFC =  7.69","<h4>STAP1<\/h4><br>logFC =  3.42","<h4>STAT1<\/h4><br>logFC =  3.15","<h4>STAT2<\/h4><br>logFC =  2.28","<h4>SUCNR1<\/h4><br>logFC =  3.48","<h4>TAP1<\/h4><br>logFC =  5.36","<h4>TAP2<\/h4><br>logFC =  3.46","<h4>TAPBP<\/h4><br>logFC =  1.87","<h4>TBX21<\/h4><br>logFC =  8.66","<h4>TGM2<\/h4><br>logFC =  2.71","<h4>TICAM2<\/h4><br>logFC =  1.98","<h4>TICRR<\/h4><br>logFC = -2.83","<h4>TIFA<\/h4><br>logFC =  5.19","<h4>TLR10<\/h4><br>logFC =  2.76","<h4>TLR5<\/h4><br>logFC = -2.18","<h4>TLR7<\/h4><br>logFC =  2.03","<h4>TLR8<\/h4><br>logFC =  3.63","<h4>TNFAIP6<\/h4><br>logFC =  3.88","<h4>TNFRSF11A<\/h4><br>logFC = -3.93","<h4>TNFRSF21<\/h4><br>logFC = -2.71","<h4>TNFSF13B<\/h4><br>logFC =  3.71","<h4>TNFSF18<\/h4><br>logFC =  4.41","<h4>TRAFD1<\/h4><br>logFC =  2.72","<h4>TRIM22<\/h4><br>logFC =  2.40","<h4>TRIM31<\/h4><br>logFC =  3.05","<h4>TSPAN32<\/h4><br>logFC =  3.84","<h4>UBD<\/h4><br>logFC = 10.27","<h4>VCAM1<\/h4><br>logFC =  5.67","<h4>XAF1<\/h4><br>logFC =  2.98","<h4>ZBP1<\/h4><br>logFC =  3.50","<h4>ZC3H12A<\/h4><br>logFC =  2.73"],"label":["adaptive immune response","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation via MHC class Ib","cellular response to lipopolysaccharide","chemokine-mediated signaling pathway","complement activation, classical pathway","defense response to protozoan","defense response to virus","DNA replication initiation","double-strand break repair via break-induced replication","inflammatory response","innate immune response","interferon-gamma-mediated signaling pathway","positive regulation of T cell mediated cytotoxicity","regulation of complement activation","response to interferon-gamma","T cell receptor signaling pathway","type I interferon signaling pathway","ACOD1","ADAM8","ADGRE5","ADORA3","AGT","AIM2","AOC3","APOBEC3A","APOBEC3D","APOBEC3G","APOL1","APOL2","APOL3","APPL2","B2M","BATF2","BCL6","BTN2A2","BTN3A1","BTN3A2","BTN3A3","C1QB","C1R","C1RL","C1S","C2","C3","C4A","C4B","CALCOCO2","CAMK2D","CARD16","CARD17","CASP4","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CCNE2","CCRL2","CD1A","CD1C","CD274","CD28","CD300LF","CD40","CD7","CD74","CD80","CDC45","CDC6","CDH5","CDT1","CEACAM1","CFB","CFH","CIITA","CLEC10A","CLEC6A","CMPK2","COLEC12","CR1L","CRHBP","CTLA4","CTSS","CX3CR1","CXCL10","CXCL11","CXCL5","CXCL9","CXCR3","CYLD","CYSLTR1","DDX60","DTX3L","EDN1","ERAP2","EXO1","FCN1","FGL1","FGL2","FYB1","GBP1","GBP2","GBP3","GBP4","GBP5","GBP6","GBP7","GCH1","GCNT3","GGT5","GHRL","GINS2","GPR183","GRAMD4","GSDMD","H2BC21","HGF","HLA-A","HLA-B","HLA-C","HLA-DMA","HLA-DMB","HLA-DOA","HLA-DOB","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","ICAM2","IDO1","IFI27","IFI35","IFI44L","IFIH1","IFIT2","IFIT3","IFIT5","IFITM1","IFITM2","IFITM3","IL12A","IL12RB1","IL15","IL18BP","IL1RN","IL22RA2","IL27","IRF1","IRF7","ISG20","ITK","JAK2","JAK3","KARS1","KIF11","KIF15","KIF26A","KLF4","KLRK1","LAG3","LAMP3","LILRA1","LILRA2","LILRB3","LITAF","LRRK2","LY75","LYN","MCM10","MCM2","MCM4","MCM5","MCM6","MCM7","MCOLN2","MEFV","MMP25","MMP9","MSRB1","MT2A","MUC1","NCF1","NLRC5","NLRP12","NMI","NOD2","NUB1","OAS2","OPTN","ORC1","OSM","P2RX7","PADI2","PARP14","PDCD1","PDCD1LG2","PDE4B","PMAIP1","PML","POLB","POLE2","PSMB10","PSMB8","PSMB9","PSME1","PSME2","PTGER4","PYHIN1","RAB20","RELB","RIPK2","RNF19B","RSAD2","RTP4","SBNO2","SERINC5","SERPING1","SIT1","SLAMF1","SLAMF6","SLAMF7","SLAMF8","SLC11A1","SOCS1","STAP1","STAT1","STAT2","SUCNR1","TAP1","TAP2","TAPBP","TBX21","TGM2","TICAM2","TICRR","TIFA","TLR10","TLR5","TLR7","TLR8","TNFAIP6","TNFRSF11A","TNFRSF21","TNFSF13B","TNFSF18","TRAFD1","TRIM22","TRIM31","TSPAN32","UBD","VCAM1","XAF1","ZBP1","ZC3H12A"],"x":[0.071763743807999,0.438638725561432,0.438854179342348,0.434709987423746,0.372347254461456,-0.423892693433965,-0.571575861886171,-0.289466434390884,-0.0579281966788042,-0.168366047559381,0.646942822364831,0.865559955135187,-0.546901718278396,-0.0417703953612542,0.122695968648399,0.185602684585632,-0.39504956539069,0.0349594814413863,0.22070378521089,0.147002332807404,-0.294828040718609,-0.447534838688722,-0.923408184638937,-0.877465988795487,-0.867242078215663,-0.379156887424615,-0.93775894040115,-0.320140962616953,-0.21354817343623,-0.16196626142429,0.256127988284591,-0.907716413492616,-0.854163035716966,-0.438594388548174,0.272365630032086,-0.0379447807026646,-0.363500187562171,0.574970359411608,0.175723290114479,0.23402297552597,0.286266000821143,-0.281942736378574,-0.254268826318837,-0.0714500172478603,-0.232308594436454,-0.218572077125669,-0.243645146089909,-0.263989378801532,-0.285543896400611,0.316560195324542,0.300044625868853,-0.541624619491826,-0.67703656613746,-0.497500102571597,-0.353255094707121,-0.323376643082783,-0.326899206886855,-0.341818827676685,-0.350050177327776,-0.322730363359221,0.688336327457403,-0.776113334686534,0.345600451402968,0.382637747155456,-0.222301905054458,-0.160912668676102,-0.0784824815746071,-0.193668564770854,0.130982986354036,0.456874560512465,-0.263641178770802,0.776479764642041,0.596223517966104,-0.796657130645379,0.543841475268846,0.135303164935053,-0.38830218789218,-0.433077832721174,-0.17538755670095,0.00159589573115326,0.0802702658425278,-0.588662420259249,0.0685790777548667,-0.617641023481758,-0.889204517739466,0.236667947062244,0.291401004265579,-0.450046499127437,-0.505720482701872,-0.683613288179213,-0.67263989092688,-0.538805530858598,-0.78487366905665,-0.128699834565654,-0.955095723246437,-0.209333387220128,-0.259363344252099,-0.272449581675617,0.493871412812589,-0.0511029823368917,0.103219467104522,0.196199965658075,-0.0344491284298137,0.616387682395703,0.12806007223487,0.14332357028658,-0.113091985433129,0.145816831807567,-0.271101187533926,0.132921020605066,0.0917264553375623,0.216283916233083,0.101994983251354,-0.934660293629107,-1,1,0.202212943595656,-0.145837791060985,-0.477013844134266,-0.247418713993774,-0.962121887927662,0.234087739983494,0.211459834848932,0.25904916017672,0.414666417384269,0.433867215608104,0.476503137868001,0.400264564221714,0.159253146452873,0.220891648055711,0.16628236284319,0.194447776924313,0.24752206972775,0.250189996024408,0.179473812342518,0.217272275407211,0.0855393772716759,0.236210603288591,0.266777686941607,-0.158524445451651,0.196185195213624,-0.948269970321574,0.0088796865912828,0.208549388188691,-0.376301145797657,-0.106432478755437,0.0530136856279448,-0.0306480274250321,-0.15545024227363,-0.00962797412129346,0.015452403185495,0.0504728629017432,-0.0680470565725637,0.0316743782661852,-0.574139524173762,0.149987179751516,-0.652974530416843,-0.983962923302876,-0.21535251576062,0.0722888274163038,0.0425337489193935,-0.0446322715353298,0.312139571833417,-0.115532461788122,0.0651811460078791,-0.857287816130466,0.779545878102654,0.815847808897659,0.806387685345884,-0.998989397927394,-0.133507354428053,0.248037066350009,0.0287020434286092,0.247034252465922,-0.435197858439058,0.35695539627637,-0.642881566859211,-0.846096581054231,-0.994346478752706,-0.264266197614809,0.457567135074466,0.83276561481357,0.70948559358581,0.840025675171575,0.775097271046489,0.754677843138638,0.0885715087939745,-0.266911663995132,-0.963336982188312,-0.819463543497382,0.320260384743747,0.262361861083723,0.144953048766693,0.44200924751813,0.0921728472108516,-0.870745550359924,-0.193463680671676,-0.424748881396683,0.309544051507112,0.0451945165891259,0.0243682123056803,0.439912950010408,-0.86912827609816,-0.174652255271279,-0.869991103152276,0.225220633915885,0.312694946616199,-0.208584990182228,-0.103022698279774,-0.308436952593901,-0.0287181622680911,-0.920276792240276,0.50893140293981,0.365254967837994,0.326944568038476,0.388088197585969,0.369034684339705,0.402160044638869,-0.879476621308494,-0.360994307434113,0.235790580844444,-0.239333824530678,-0.150866471128342,0.0785180689919056,0.00488041281772444,-0.438864694713202,-0.677703098609567,-0.271717107934493,-0.306022651684175,0.2650664795001,0.025250934583485,0.0364332414578041,-0.0125635770134342,-0.511354753576673,-0.136121944531221,0.234295654420426,-0.692219062192601,0.0280229530400804,0.0474879351165274,-0.826938472614776,0.513622518303704,0.475647722803618,0.734172744905275,-0.0631696497622246,-0.894258934509791,-0.390260772488188,0.530647253372588,-0.0251603550734474,-0.46115675165736,-0.47289563357253,-0.388555828634941,-0.207188375310213,-0.914840793169564,-0.332677879688324,0.33377308999912,0.0394538768734187,0.00173380657561428,-0.307207963138968,0.0748057100787827,0.286027681475173,-0.191135534000796,0.272062686167249,-0.180523155347749,0.329673576367203,0.2692651754221,-0.172939055098082],"y":[-0.586481510769463,-0.374128537059434,-0.309303880701367,-0.546227136803441,-0.413798836399264,-0.54461797231324,-0.140941328604764,-0.6053285409035,0.0442773199685138,0.00399098319252045,0.870538689901769,0.729270401383759,-0.332637260844169,-0.139604713910763,-0.217671945731498,-0.45231109860376,-0.550274210364689,-0.157102155790843,-0.530031099447322,-0.0473512977151143,-0.292890051610753,-0.062587891076168,-0.210739438293035,-0.170316965015588,-0.496375025462874,-0.0468548630365879,-0.12255730261493,0.126759886591781,0.142258454335447,0.167077493511217,0.222170042571166,-0.584546286771567,-0.574013518491238,-0.129402940189155,-0.330083476164063,0.453004166660397,-0.647916691656656,-0.775162136834387,-0.787079590445223,-0.802175999352145,-0.783700616967887,-0.441793122167857,-0.47380502340126,-0.54410719815419,-0.50681234753622,-0.472974376188042,-0.444538489216684,-0.510969648963337,-0.487620448633832,-0.0239809549761728,-0.0814426063522956,-0.892704917042119,-0.818828649480092,-0.042217753814959,-0.116055196828326,-0.094589534448708,-0.184400770493135,-0.254128289590427,-0.159681651601928,-0.137035698794597,0.961702264845639,-0.0851258511271284,-0.622586885513413,-0.599945318417776,-0.816153824537568,-0.62417315244048,0.297324888299106,-0.243205847148783,-0.914031280004939,-0.75698068343646,-0.791242118487458,0.720174366950292,0.995481515470893,-0.602550394069074,0.929481340027396,-0.507789134322067,-0.375675423748042,-0.387893111457411,-0.171768450355346,-0.446213076954393,-0.457178119858398,-0.858972899271898,0.26770459437441,-0.643358574825809,-0.334014281841965,-0.756064086288866,-0.455611130558029,-0.262654937424075,-0.227678504415512,-0.367718787781141,-0.408190356638926,-0.228129254961027,-0.14466635912678,-0.416215979745402,-0.26338691497248,0.200255785108312,0.174983867572128,-0.0154142623801162,-0.604787050573505,-0.975701362403205,0.299129343994815,-0.966902183480535,-0.289651596627712,-0.697667803437996,-0.180534813757641,-0.0635185077829526,0.0660360036355434,0.0574306502842181,-0.17079421082082,0.0979527850709634,0.0979864175023955,0.0727788069527375,-1,-0.488442225536043,-0.368987398655557,0.618776215430494,-0.912508436688034,0.284606396260642,-0.151513078391768,0.265533359006624,-0.402578015501679,-0.276094456256221,-0.289560448693934,-0.26678788670515,-0.791116711983834,-0.713169029352775,-0.696451810441751,-0.750154504363318,-0.409582452557172,-0.363544274757464,-0.363763502845459,-0.375230992696076,-0.388282575858246,-0.363083421203386,-0.398577346803435,-0.403665670125901,-0.319909232690795,-0.301436019273334,-0.288990711892074,-0.35794356110259,0.254615676014216,-0.336896527632013,0.14789046510973,0.161264652196946,0.373130104726408,0.20912487728213,0.135979705143853,0.159981556770889,0.212958465637127,0.0645319255003181,0.0377212945115599,0.0614319906617007,-0.350412755650863,-0.284322096252744,0.0134474083689031,-0.981061711243817,-0.607781187825049,-0.461572643206981,-0.311806548376095,-0.20935461905517,-0.223389413665723,0.122861950751492,-0.727810969522862,-0.320710430688421,-0.497689762627955,-0.234243113419203,-0.711625489429454,-0.582874666827254,-0.646842612017935,-0.304541898255335,-0.525589043374244,-0.485851874716795,-0.908736228279707,-0.980473772609451,-0.341851857131317,-0.898349181373304,-0.864135691437937,-0.439055014539394,-0.232933094269891,-0.410557744362393,0.898533606328789,0.78897302212568,0.745733480729531,0.849514588155623,0.849254018319245,0.789710389100924,-0.411641673036173,-0.113340231100752,-0.173899870371223,-0.527845489732215,0.177566598178018,-0.0853856483812752,0.256371052306383,-0.0974087778862315,-0.00338772154615352,-0.381350267901884,-0.147762455386029,-0.304089019502238,0.0136363478558228,-0.0102011017933319,0.298530135210314,0.966732160429006,-0.066056045680971,-0.584158551991189,0.112075016121416,-0.095751040828022,-0.919482599500897,-0.779493464718156,-0.740810866438142,0.402600534525315,-0.0353222050475502,-0.534794969049541,1,-0.256937130266598,-0.208838503336022,-0.286928429698404,-0.323496502406819,-0.243086441011732,-0.117217442738717,0.186940046136089,0.0239250174790893,-0.378569021922658,-0.473918928718867,-0.945373981450174,-0.215255921992554,0.335770759164353,-0.557789476534178,0.12630941579967,-0.464134167584742,-0.92993765889503,-0.529391892469053,-0.411753111621753,-0.491697876191448,-0.0864362097172181,-0.261283016604109,-0.0434741544467739,-0.615690524794106,-0.0464432844076841,0.169750296174763,-0.639844903948257,-0.517426003810501,-0.496408256133976,-0.325583232047307,-0.925412400823237,-0.282016397253256,-0.330503665135798,0.839311496253849,0.299867102351385,-0.0166817196983596,-0.0994371402786134,-0.0109239718535739,-0.288161610104166,-0.430601746225517,-0.668175009016508,-0.776318401141466,-0.995154434821001,-0.956167343477378,0.230536330952406,-0.0461096004281432,-0.123800427679763,0.330728367884155,0.0437237974870781,-0.201877853270549,0.0920936514645705,0.131492985832688,-0.406547652385414]},"edges":{"from":["adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","defense response to virus","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","antigen processing and presentation via MHC class Ib","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","DNA replication initiation","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","inflammatory response","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","innate immune response","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","complement activation, classical pathway","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","double-strand break repair via break-induced replication","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","defense response to protozoan","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","regulation of complement activation","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation of exogenous peptide antigen via MHC class II"],"to":["B2M","BCL6","BTN3A1","BTN3A2","BTN3A3","C1QB","C1R","C1RL","C1S","C2","C3","C4A","C4B","CD1A","CD1C","CD274","CD28","CD40","CD7","CD74","CD80","CEACAM1","CLEC10A","CLEC6A","CTLA4","CTSS","ERAP2","EXO1","FGL1","FGL2","GCNT3","GPR183","HLA-A","HLA-B","HLA-C","HLA-DMA","HLA-DMB","HLA-DOA","HLA-DOB","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","IL12A","IL12RB1","IL18BP","IL27","IRF1","IRF7","ITK","JAK2","JAK3","KLRK1","LAG3","LAMP3","LILRA1","LILRB3","LYN","MCOLN2","P2RX7","PDCD1","PDCD1LG2","RELB","RIPK2","RNF19B","RSAD2","SERPING1","SIT1","SLAMF1","SLAMF6","SLAMF7","SLC11A1","TAP1","TAP2","TBX21","TLR8","TNFRSF11A","TNFRSF21","TNFSF13B","TNFSF18","ZC3H12A","B2M","CAMK2D","CIITA","GBP1","GBP2","HLA-A","HLA-B","HLA-C","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","IRF1","IRF7","JAK2","MT2A","NLRC5","NMI","OAS2","PARP14","PML","SOCS1","STAT1","TRIM22","TRIM31","VCAM1","GBP2","HLA-A","HLA-B","HLA-C","HLA-E","HLA-F","HLA-G","IFI27","IFI35","IFIT2","IFIT3","IFITM1","IFITM2","IFITM3","IRF1","IRF7","ISG20","NLRC5","OAS2","PSMB8","RSAD2","STAT1","STAT2","XAF1","ZBP1","B2M","ERAP2","HLA-A","HLA-B","HLA-C","HLA-E","HLA-F","HLA-G","TAP1","TAP2","TAPBP","AIM2","APOBEC3A","APOBEC3D","APOBEC3G","CD40","CXCL10","CXCL9","DDX60","DTX3L","FGL2","GBP1","GBP3","IFI27","IFI44L","IFIH1","IFIT2","IFIT3","IFIT5","IFITM1","IFITM2","IFITM3","IL12RB1","IL15","IL27","IRF1","IRF7","ISG20","NLRC5","OAS2","PMAIP1","PML","RSAD2","RTP4","SERINC5","STAT1","STAT2","TLR7","TLR8","TRIM22","TSPAN32","ZC3H12A","B2M","CD1A","CD1C","HLA-A","HLA-B","HLA-C","HLA-E","HLA-F","HLA-G","IL12A","IL12RB1","P2RX7","B2M","CD1A","CD1C","HLA-A","HLA-B","HLA-C","HLA-E","HLA-F","HLA-G","TAP2","CCNE2","CDC45","CDC6","CDT1","MCM10","MCM2","MCM4","MCM5","MCM6","MCM7","ORC1","POLE2","TICRR","ACOD1","B2M","CALCOCO2","CAMK2D","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CIITA","EDN1","GBP1","GBP2","GBP3","GBP4","GBP5","GBP6","GBP7","GCH1","HLA-A","HLA-B","HLA-C","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","IFITM1","IFITM2","IFITM3","IL12RB1","IRF1","IRF7","JAK2","MEFV","MT2A","NLRC5","NMI","NUB1","OAS2","PARP14","PML","RAB20","SLC11A1","SOCS1","STAT1","TRIM22","TRIM31","UBD","VCAM1","ACOD1","ADAM8","ADGRE5","ADORA3","AGT","AIM2","AOC3","APOL2","APOL3","APPL2","BCL6","C1QB","C1R","C1S","C2","C3","C4A","C4B","CASP4","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CCRL2","CD28","CD40","CDH5","CFB","CFH","CIITA","CR1L","CRHBP","CXCL10","CXCL11","CXCL5","CXCL9","CXCR3","CYLD","CYSLTR1","GBP5","GGT5","GHRL","GSDMD","HGF","HLA-E","ICAM1","IDO1","IL15","IL1RN","IL22RA2","IL27","JAK2","KARS1","KLF4","LRRK2","LY75","LYN","MEFV","MMP25","MMP9","NLRP12","NMI","NOD2","OSM","P2RX7","POLB","PTGER4","RELB","RIPK2","SBNO2","SERPING1","SLAMF8","SLC11A1","STAP1","SUCNR1","TGM2","TICAM2","TLR10","TLR5","TLR7","TLR8","TNFAIP6","TNFRSF11A","VCAM1","ZC3H12A","BTN2A2","BTN3A1","BTN3A2","BTN3A3","CD28","CEACAM1","CTLA4","CYLD","FYB1","GBP1","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","ITK","PDE4B","PSMB10","PSMB8","PSMB9","PSME1","PSME2","RIPK2","TNFRSF21","ZC3H12A","ACOD1","ADAM8","AIM2","APOBEC3A","APOBEC3D","APOBEC3G","APOL1","APPL2","B2M","C1QB","C1R","C1RL","C1S","C2","C3","C4A","C4B","CALCOCO2","CAMK2D","CASP4","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CD300LF","CEACAM1","CFB","CFH","CIITA","CLEC10A","CLEC6A","COLEC12","CTSS","CX3CR1","CYLD","DDX60","DTX3L","EDN1","FCN1","GBP1","GBP2","GBP3","GBP4","GBP5","GBP6","GBP7","GCH1","GRAMD4","GSDMD","H2BC21","HLA-A","HLA-B","HLA-C","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","HLA-E","HLA-F","HLA-G","ICAM1","ICAM2","IFI27","IFI35","IFIH1","IFIT2","IFIT3","IFIT5","IFITM1","IFITM2","IFITM3","IL12A","IL12RB1","IL27","IRF1","IRF7","ISG20","JAK2","JAK3","KLRK1","LAG3","LILRA2","LYN","MCOLN2","MEFV","MSRB1","MT2A","MUC1","NCF1","NLRC5","NMI","NOD2","NUB1","OAS2","OPTN","PARP14","PML","PSMB10","PSMB8","PSMB9","PSME1","PSME2","PYHIN1","RAB20","RELB","RIPK2","RSAD2","SERINC5","SERPING1","SLAMF1","SLAMF6","SLAMF7","SLAMF8","SLC11A1","SOCS1","STAT1","STAT2","TICAM2","TIFA","TLR10","TLR5","TLR7","TLR8","TRAFD1","TRIM22","TRIM31","UBD","VCAM1","XAF1","ZBP1","C1QB","C1R","C1RL","C1S","C2","C3","C4A","C4B","SERPING1","CDC45","GINS2","MCM2","MCM4","MCM5","MCM6","MCM7","BATF2","CD40","GBP2","GBP4","GBP6","GBP7","IL12A","SLC11A1","TSPAN32","ACOD1","CARD16","CARD17","CCL5","CD274","CD80","CMPK2","CX3CR1","CXCL10","CXCL11","CXCL5","CXCL9","ICAM1","IL12A","IL1RN","KLRK1","LILRA2","LITAF","LYN","NOD2","PDCD1LG2","PDE4B","RIPK2","SBNO2","STAP1","TICAM2","ZC3H12A","C1QB","C1R","C1S","C2","C3","C4A","C4B","CFB","CFH","CR1L","SERPING1","CCL13","CCL15","CCL18","CCL5","CCL7","CCL8","CCRL2","CX3CR1","CXCL10","CXCL11","CXCL5","CXCL9","CXCR3","EDN1","PADI2","B2M","HLA-A","HLA-B","HLA-C","HLA-E","HLA-F","HLA-G","NCF1","PSMB10","PSMB8","PSMB9","PSME1","PSME2","TAP1","TAP2","TAPBP","CD74","CTSS","HLA-DMA","HLA-DMB","HLA-DOA","HLA-DOB","HLA-DPA1","HLA-DPB1","HLA-DQA1","HLA-DQB1","HLA-DQB2","HLA-DRA","HLA-DRB1","HLA-DRB5","KIF11","KIF15","KIF26A","LAG3"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false},"manipulation":{"enabled":false},"edges":{"smooth":false},"physics":{"stabilization":false},"interaction":{"hover":true}},"groups":null,"width":null,"height":null,"idselection":{"enabled":true,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)","highlight":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","igraphlayout":{"type":"square"},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","highlight":{"enabled":true,"hoverNearest":true,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="summary-representations" class="section level2">
<h2 class="hasAnchor">
<a href="#summary-representations" class="anchor"></a>Summary representations</h2>
<p>Plot an enrichment map of the enrichment results, where you can choose with <code>n_gs</code> the number of top genesets to be displayed, and specify in <code>gs_ids</code> the gene set identifiers - these will be merged together for each function’s context:</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb21-1" data-line-number="1">em &lt;-<span class="st"> </span><span class="kw"><a href="../reference/enrichment_map.html">enrichment_map</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb21-2" data-line-number="2">                     res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb21-3" data-line-number="3">                     <span class="dt">n_gs =</span> <span class="dv">30</span>,</a>
<a class="sourceLine" id="cb21-4" data-line-number="4">                     anno_df)</a>
<a class="sourceLine" id="cb21-5" data-line-number="5"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"visNetwork"</span>)</a>
<a class="sourceLine" id="cb21-6" data-line-number="6"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="st">"magrittr"</span>)</a>
<a class="sourceLine" id="cb21-7" data-line-number="7">em <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb21-8" data-line-number="8"><span class="st">  </span><span class="kw"><a href="https://rdrr.io/pkg/visNetwork/man/visNetwork-igraph.html">visIgraph</a></span>() <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb21-9" data-line-number="9"><span class="st">  </span><span class="kw"><a href="https://rdrr.io/pkg/visNetwork/man/visOptions.html">visOptions</a></span>(<span class="dt">highlightNearest =</span> <span class="kw"><a href="https://rdrr.io/r/base/list.html">list</a></span>(<span class="dt">enabled =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb21-10" data-line-number="10">                                     <span class="dt">degree =</span> <span class="dv">1</span>,</a>
<a class="sourceLine" id="cb21-11" data-line-number="11">                                     <span class="dt">hover =</span> <span class="ot">TRUE</span>),</a>
<a class="sourceLine" id="cb21-12" data-line-number="12">             <span class="dt">nodesIdSelection =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<div id="htmlwidget-d205e4baef90aa8461dd" style="width:960px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-d205e4baef90aa8461dd">{"x":{"nodes":{"id":["adaptive immune response","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation via MHC class Ib","cellular response to interferon-gamma","cellular response to lipopolysaccharide","cellular response to tumor necrosis factor","chemokine-mediated signaling pathway","complement activation, classical pathway","defense response to Gram-positive bacterium","defense response to protozoan","defense response to virus","DNA replication initiation","double-strand break repair via break-induced replication","G protein-coupled receptor signaling pathway","inflammatory response","innate immune response","interferon-gamma-mediated signaling pathway","negative regulation of viral genome replication","neutrophil chemotaxis","positive regulation of inflammatory response","positive regulation of interferon-gamma production","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell proliferation","regulation of complement activation","regulation of transcription involved in G1/S transition of mitotic cell cycle","response to interferon-gamma","T cell receptor signaling pathway","type I interferon signaling pathway"],"size":[47.169905660283,16.583123951777,20,21.2132034355964,15.8113883008419,35,25.9807621135332,30,19.3649167310371,15,17.3205080756888,15,32.0156211871642,18.0277563773199,13.228756555323,38.405728739343,46.6368952654441,58.309518948453,29.1547594742265,18.0277563773199,20.6155281280883,23.4520787991171,18.0277563773199,17.3205080756888,20,16.583123951777,15,38.0788655293195,26.4575131106459,25],"original_size":[89,11,16,18,10,49,27,36,15,9,12,9,41,13,7,59,87,136,34,13,17,22,13,12,16,11,9,58,28,25],"color.background":["#800026CC","#FDAE4ACC","#FFF6B6CC","#FFF6B6CC","#FED470CC","#FFFCC4CC","#FFF3AFCC","#FFFCC4CC","#FFF6B6CC","#FFF0A8CC","#FFFFCCCC","#FFF3AFCC","#FECD6ACC","#FEE693CC","#FFF3AFCC","#FFFCC4CC","#FEE693CC","#FFEDA0CC","#BB0026CC","#FFFCC4CC","#FFF9BDCC","#FFFFCCCC","#FFFCC4CC","#FED470CC","#FFFCC4CC","#FFF6B6CC","#FFFFCCCC","#FEE693CC","#FFEDA0CC","#FDA245CC"],"color.highlight":["#800026","#FDAE4A","#FFF6B6","#FFF6B6","#FED470","#FFFCC4","#FFF3AF","#FFFCC4","#FFF6B6","#FFF0A8","#FFFFCC","#FFF3AF","#FECD6A","#FEE693","#FFF3AF","#FFFCC4","#FEE693","#FFEDA0","#BB0026","#FFFCC4","#FFF9BD","#FFFFCC","#FFFCC4","#FED470","#FFFCC4","#FFF6B6","#FFFFCC","#FEE693","#FFEDA0","#FDA245"],"color.hover":["#80002680","#FDAE4A80","#FFF6B680","#FFF6B680","#FED47080","#FFFCC480","#FFF3AF80","#FFFCC480","#FFF6B680","#FFF0A880","#FFFFCC80","#FFF3AF80","#FECD6A80","#FEE69380","#FFF3AF80","#FFFCC480","#FEE69380","#FFEDA080","#BB002680","#FFFCC480","#FFF9BD80","#FFFFCC80","#FFFCC480","#FED47080","#FFFCC480","#FFF6B680","#FFFFCC80","#FEE69380","#FFEDA080","#FDA24580"],"color.border":["black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black","black"],"label":["adaptive immune response","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","antigen processing and presentation via MHC class Ib","cellular response to interferon-gamma","cellular response to lipopolysaccharide","cellular response to tumor necrosis factor","chemokine-mediated signaling pathway","complement activation, classical pathway","defense response to Gram-positive bacterium","defense response to protozoan","defense response to virus","DNA replication initiation","double-strand break repair via break-induced replication","G protein-coupled receptor signaling pathway","inflammatory response","innate immune response","interferon-gamma-mediated signaling pathway","negative regulation of viral genome replication","neutrophil chemotaxis","positive regulation of inflammatory response","positive regulation of interferon-gamma production","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell proliferation","regulation of complement activation","regulation of transcription involved in G1/S transition of mitotic cell cycle","response to interferon-gamma","T cell receptor signaling pathway","type I interferon signaling pathway"],"x":[-0.189491171511255,0.160425880490846,0.0201569031749482,-0.233497393362639,0.253445072819605,-0.107655490099941,-0.770502777835811,-0.254125044668417,-0.470613328774544,0.152108343289947,-0.628665247525209,-1,0.414536804292826,0.744049758183335,1,-0.68279122218881,-0.306482782853279,0.062692698102782,0.125155798683982,0.836739103512035,-0.544048283395392,-0.550446738723093,-0.748101274388583,-0.0792865100069917,-0.525805904169603,0.0112895577300964,0.460224676460574,-0.0311274141028453,-0.333890223764784,0.351597522567575],"y":[-0.132728034995609,0.125556678146211,0.000656054210757206,0.112475533970019,0.0303599983981664,-0.18936960685052,-0.269493050281218,-0.371930044224833,-0.516149286863001,-0.712734562169028,0.417508075187121,0.47530380259688,-0.328049683813594,0.852580923016106,0.671705914022695,-0.770854846750123,-0.58148847943968,-0.374459147346618,-0.0954552147794596,-0.288485018484599,-0.423951096660088,-1,0.127937109325515,0.158719359751516,0.0840166383529592,-0.985015638027524,1,-0.235597171028871,-0.0764205691570621,-0.124613575937831]},"edges":{"from":["adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","adaptive immune response","interferon-gamma-mediated signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","interferon-gamma-mediated signaling pathway","antigen processing and presentation via MHC class Ib","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","innate immune response","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","cellular response to interferon-gamma","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","response to interferon-gamma","innate immune response","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","negative regulation of viral genome replication","cellular response to interferon-gamma","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","defense response to virus","defense response to virus","cellular response to interferon-gamma","antigen processing and presentation via MHC class Ib","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","positive regulation of T cell mediated cytotoxicity","cellular response to interferon-gamma","defense response to Gram-positive bacterium","antigen processing and presentation via MHC class Ib","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation via MHC class Ib","DNA replication initiation","DNA replication initiation","inflammatory response","response to interferon-gamma","innate immune response","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","neutrophil chemotaxis","cellular response to tumor necrosis factor","cellular response to interferon-gamma","inflammatory response","cellular response to lipopolysaccharide","inflammatory response","chemokine-mediated signaling pathway","inflammatory response","cellular response to tumor necrosis factor","G protein-coupled receptor signaling pathway","cellular response to interferon-gamma","inflammatory response","innate immune response","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","cellular response to tumor necrosis factor","positive regulation of T cell proliferation","positive regulation of interferon-gamma production","cellular response to interferon-gamma","cellular response to tumor necrosis factor","cellular response to interferon-gamma","complement activation, classical pathway","defense response to protozoan","defense response to Gram-positive bacterium","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","cellular response to lipopolysaccharide","chemokine-mediated signaling pathway","cellular response to tumor necrosis factor","chemokine-mediated signaling pathway","cellular response to interferon-gamma","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","cellular response to tumor necrosis factor","G protein-coupled receptor signaling pathway","cellular response to interferon-gamma","cellular response to interferon-gamma","positive regulation of interferon-gamma production","cellular response to interferon-gamma","defense response to Gram-positive bacterium","defense response to Gram-positive bacterium","G protein-coupled receptor signaling pathway"],"to":["interferon-gamma-mediated signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","antigen processing and presentation of exogenous peptide antigen via MHC class II","positive regulation of T cell proliferation","positive regulation of interferon-gamma production","cellular response to interferon-gamma","type I interferon signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","positive regulation of T cell mediated cytotoxicity","interferon-gamma-mediated signaling pathway","response to interferon-gamma","T cell receptor signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","type I interferon signaling pathway","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","response to interferon-gamma","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","cellular response to interferon-gamma","response to interferon-gamma","innate immune response","negative regulation of viral genome replication","defense response to virus","positive regulation of T cell mediated cytotoxicity","response to interferon-gamma","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell proliferation","positive regulation of T cell mediated cytotoxicity","positive regulation of T cell mediated cytotoxicity","response to interferon-gamma","antigen processing and presentation via MHC class Ib","cellular response to interferon-gamma","double-strand break repair via break-induced replication","regulation of transcription involved in G1/S transition of mitotic cell cycle","response to interferon-gamma","T cell receptor signaling pathway","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","response to interferon-gamma","innate immune response","inflammatory response","regulation of complement activation","inflammatory response","neutrophil chemotaxis","inflammatory response","inflammatory response","inflammatory response","positive regulation of inflammatory response","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","T cell receptor signaling pathway","innate immune response","innate immune response","regulation of complement activation","positive regulation of interferon-gamma production","defense response to protozoan","chemokine-mediated signaling pathway","neutrophil chemotaxis","positive regulation of T cell proliferation","positive regulation of interferon-gamma production","neutrophil chemotaxis","chemokine-mediated signaling pathway","G protein-coupled receptor signaling pathway","chemokine-mediated signaling pathway","cellular response to tumor necrosis factor","cellular response to interferon-gamma","cellular response to interferon-gamma","neutrophil chemotaxis","neutrophil chemotaxis","neutrophil chemotaxis","cellular response to tumor necrosis factor","positive regulation of T cell proliferation","positive regulation of T cell proliferation","positive regulation of T cell proliferation","positive regulation of interferon-gamma production","positive regulation of inflammatory response"],"width":[6.04470524826989,4.71404520791032,5.19290786889499,4.74045463139977,5.77350269189626,5.48318880553316,6.03022689155527,7.36459694318659,4.49719013397517,5.71040240720161,5.31609533071195,4.66252404120157,5.82222509739582,7.14589601010496,6.06976978666884,4.88677777425221,5.99144689515278,6.15124685229245,10.8278058400742,5.82771517414359,7.07106781186548,5.70597402157456,6.03022689155527,11.780301787479,6.32455532033676,7.33799385705343,6.22171016838255,6.43267520902677,6.64211164155071,6.06339062590832,6.41688947919748,5.5048188256318,6.22171016838255,9.35414346693485,11.0940039245046,4.75190963311491,10.8465228909328,5.13956168750047,5.49841414769158,6.51657528343359,6.74199862463242,5,11.766968108291,5.08000508000762,8.16496580927726,4.89897948556636,5.49442255794756,5.34522483824849,4.79070137580408,9.42809041582063,5.18874521662771,9.25820099772552,6.66666666666667,4.98058245091752,4.83493778415228,9.23548145182799,4.60566186471838,4.57116203913837,4.85071250072666,4.53742606486515,6.16236022513321,12.9986736723936,6.91214711777591,5.5048188256318,5.02865352970744,5.16397779494322,4.8637345711392,5.8554004376912,6.11542099807853,4.79070137580408,7.1115900221876,4.8092880658867,5.06369683541833,6.4888568452305,5.72077553547355,4.47213595499958,4.64990554975277,5.14495755427527,5.55630121766958,8.48874687627166,11.5470053837925,4.47213595499958,7.90569415042095,5.77350269189626,6.15124685229245,5.69494797451499,4.71404520791032,10.2353263143832,5.64076074817766,6.83130051063973,4.95594627783352,4.61265604014442,4.91303684440518,5.20755643923295,5.5167728436737,6.12372435695795,4.87122300447455,6.54653670707977,6.45497224367903,4.50987616801697,4.89897948556636,5.22232967867093,4.68164588784522],"color":["lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey","lightgrey"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false},"manipulation":{"enabled":false},"edges":{"smooth":false},"physics":{"stabilization":false},"interaction":{"hover":true}},"groups":null,"width":null,"height":null,"idselection":{"enabled":true,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)","highlight":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","igraphlayout":{"type":"square"},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","highlight":{"enabled":true,"hoverNearest":true,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"}},"evals":[],"jsHooks":[]}</script><p>Display a volcano plot for genesets, plotting significance (y-axis) versus Z score (x-axis), with color and size encoding some geneset features:</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" data-line-number="1"><span class="kw"><a href="../reference/gs_volcano.html">gs_volcano</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb22-2" data-line-number="2">           <span class="dt">p_threshold =</span> <span class="fl">0.05</span>,</a>
<a class="sourceLine" id="cb22-3" data-line-number="3">           <span class="dt">color_by =</span> <span class="st">"aggr_score"</span>,</a>
<a class="sourceLine" id="cb22-4" data-line-number="4">           <span class="dt">volcano_labels =</span> <span class="dv">10</span>,</a>
<a class="sourceLine" id="cb22-5" data-line-number="5">           <span class="dt">gs_ids =</span> <span class="ot">NULL</span>,</a>
<a class="sourceLine" id="cb22-6" data-line-number="6">           <span class="dt">plot_title =</span> <span class="st">"my volcano"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/volcano-1.png" width="960"></p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb23-1" data-line-number="1">res_enrich_simplified &lt;-<span class="st"> </span><span class="kw"><a href="../reference/gs_simplify.html">gs_simplify</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb23-2" data-line-number="2">                                     <span class="dt">gs_overlap =</span> <span class="fl">0.7</span>)</a>
<a class="sourceLine" id="cb23-3" data-line-number="3"><span class="kw"><a href="https://rdrr.io/r/base/dim.html">dim</a></span>(res_enrich_macrophage)</a>
<a class="sourceLine" id="cb23-4" data-line-number="4"><span class="co">#&gt; [1] 500  10</span></a>
<a class="sourceLine" id="cb23-5" data-line-number="5"><span class="kw"><a href="https://rdrr.io/r/base/dim.html">dim</a></span>(res_enrich_simplified)</a>
<a class="sourceLine" id="cb23-6" data-line-number="6"><span class="co">#&gt; [1] 118  10</span></a>
<a class="sourceLine" id="cb23-7" data-line-number="7"><span class="kw"><a href="../reference/gs_volcano.html">gs_volcano</a></span>(res_enrich_simplified,</a>
<a class="sourceLine" id="cb23-8" data-line-number="8">           <span class="dt">color_by =</span> <span class="st">"aggr_score"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/volcano-2.png" width="960"></p>
<p>Plot a dendrogram of the enrichment results (or a subset thereof), using node color, node size, and branch color to encode relevant geneset features (and their groups):</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb24-1" data-line-number="1"><span class="kw"><a href="../reference/gs_dendro.html">gs_dendro</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb24-2" data-line-number="2">          <span class="dt">n_gs =</span> <span class="dv">50</span>,</a>
<a class="sourceLine" id="cb24-3" data-line-number="3">          <span class="dt">gs_dist_type =</span> <span class="st">"kappa"</span>, </a>
<a class="sourceLine" id="cb24-4" data-line-number="4">          <span class="dt">clust_method =</span> <span class="st">"ward.D2"</span>,</a>
<a class="sourceLine" id="cb24-5" data-line-number="5">          <span class="dt">color_leaves_by =</span> <span class="st">"z_score"</span>,</a>
<a class="sourceLine" id="cb24-6" data-line-number="6">          <span class="dt">size_leaves_by =</span> <span class="st">"gs_pvalue"</span>,</a>
<a class="sourceLine" id="cb24-7" data-line-number="7">          <span class="dt">color_branches_by =</span> <span class="st">"clusters"</span>,</a>
<a class="sourceLine" id="cb24-8" data-line-number="8">          <span class="dt">create_plot =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/dendro-1.png" width="960"></p>
<p>Produce a MultiDimensional Scaling plot of the genesets in the enrichment results:</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb25-1" data-line-number="1"><span class="kw"><a href="../reference/gs_mds.html">gs_mds</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb25-2" data-line-number="2">       res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb25-3" data-line-number="3">       anno_df,</a>
<a class="sourceLine" id="cb25-4" data-line-number="4">       <span class="dt">n_gs =</span> <span class="dv">200</span>,</a>
<a class="sourceLine" id="cb25-5" data-line-number="5">       <span class="dt">gs_ids =</span> <span class="ot">NULL</span>,</a>
<a class="sourceLine" id="cb25-6" data-line-number="6">       <span class="dt">similarity_measure =</span> <span class="st">"kappa_matrix"</span>,</a>
<a class="sourceLine" id="cb25-7" data-line-number="7">       <span class="dt">mds_k =</span> <span class="dv">2</span>,</a>
<a class="sourceLine" id="cb25-8" data-line-number="8">       <span class="dt">mds_labels =</span> <span class="dv">5</span>,</a>
<a class="sourceLine" id="cb25-9" data-line-number="9">       <span class="dt">mds_colorby =</span> <span class="st">"z_score"</span>,</a>
<a class="sourceLine" id="cb25-10" data-line-number="10">       <span class="dt">gs_labels =</span> <span class="ot">NULL</span>,</a>
<a class="sourceLine" id="cb25-11" data-line-number="11">       <span class="dt">plot_title =</span> <span class="ot">NULL</span>) </a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/mds-1.png" width="960"></p>
<p>Obtain a simple overview of enrichment results, showing e.g. significance and direction of change (<code>z_score</code>)</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb26-1" data-line-number="1"><span class="kw"><a href="../reference/gs_summary_overview.html">gs_summary_overview</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb26-2" data-line-number="2">                    <span class="dt">n_gs =</span> <span class="dv">30</span>,</a>
<a class="sourceLine" id="cb26-3" data-line-number="3">                    <span class="dt">p_value_column =</span> <span class="st">"gs_pvalue"</span>,</a>
<a class="sourceLine" id="cb26-4" data-line-number="4">                    <span class="dt">color_by =</span> <span class="st">"z_score"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/overview-1.png" width="960"></p>
<p>Plot a summary heatmap of genes vs genesets, encoding the logFC of genes and representing the overlap among genesets:</p>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb27-1" data-line-number="1"><span class="kw"><a href="../reference/gs_summary_heat.html">gs_summary_heat</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb27-2" data-line-number="2">                res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb27-3" data-line-number="3">                anno_df,</a>
<a class="sourceLine" id="cb27-4" data-line-number="4">                <span class="dt">n_gs =</span> <span class="dv">20</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/sumheat-1.png" width="960"></p>
<p>Plot a heatmap on the geneset scores, computed sample-wise via <code>gs_scores</code> so that the geneset members compose a Z score that gets summarized:</p>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb28-1" data-line-number="1">vst_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/vst.html">vst</a></span>(dds_macrophage)</a>
<a class="sourceLine" id="cb28-2" data-line-number="2">scores_mat &lt;-<span class="st"> </span><span class="kw"><a href="../reference/gs_scores.html">gs_scores</a></span>(</a>
<a class="sourceLine" id="cb28-3" data-line-number="3">  <span class="dt">se =</span> vst_macrophage,</a>
<a class="sourceLine" id="cb28-4" data-line-number="4">  <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb28-5" data-line-number="5">  <span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb28-6" data-line-number="6">  <span class="dt">annotation_obj =</span> anno_df</a>
<a class="sourceLine" id="cb28-7" data-line-number="7">)</a>
<a class="sourceLine" id="cb28-8" data-line-number="8"><span class="kw"><a href="../reference/gs_scoresheat.html">gs_scoresheat</a></span>(scores_mat,</a>
<a class="sourceLine" id="cb28-9" data-line-number="9">              <span class="dt">n_gs =</span> <span class="dv">30</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/scoresheat-1.png" width="960"></p>
</div>
<div id="reporting" class="section level2">
<h2 class="hasAnchor">
<a href="#reporting" class="anchor"></a>Reporting</h2>
<p>Enjoy a fully fledged <code>happy_hour</code> by running offline the analysis contained in the report, included in the <em><a href="https://bioconductor.org/packages/3.11/GeneTonic">GeneTonic</a></em> package.</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb29-1" data-line-number="1"><span class="kw"><a href="../reference/happy_hour.html">happy_hour</a></span>(<span class="dt">dds =</span> dds_macrophage,</a>
<a class="sourceLine" id="cb29-2" data-line-number="2">           <span class="dt">res_de =</span> res_de,</a>
<a class="sourceLine" id="cb29-3" data-line-number="3">           <span class="dt">res_enrich =</span> res_enrich,</a>
<a class="sourceLine" id="cb29-4" data-line-number="4">           <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb29-5" data-line-number="5">           <span class="dt">project_id =</span> <span class="st">"examplerun"</span>,</a>
<a class="sourceLine" id="cb29-6" data-line-number="6">           <span class="dt">mygenesets =</span> res_enrich<span class="op">$</span>gs_id[<span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="dv">1</span><span class="op">:</span><span class="dv">5</span>,<span class="dv">11</span>,<span class="dv">31</span>)],</a>
<a class="sourceLine" id="cb29-7" data-line-number="7">           <span class="dt">mygenes =</span> <span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"ENSG00000125347"</span>,</a>
<a class="sourceLine" id="cb29-8" data-line-number="8">                       <span class="st">"ENSG00000172399"</span>,</a>
<a class="sourceLine" id="cb29-9" data-line-number="9">                       <span class="st">"ENSG00000137496"</span>)</a>
<a class="sourceLine" id="cb29-10" data-line-number="10">)</a></code></pre></div>
<p>The functionality is also the same activated in the Bookmarks section of the interactive app. Here you can choose a number of genesets (<code>mygenesets</code>) and genes (<code>mygenes</code>), specifying their identifiers, and obtain in batch a number of summary representations focused on these elements.</p>
<p>Advanced users can also provide a custom RMarkdown file, and to guarantee it works in the context of <code>happy_hour</code>, we recommend to inspect the template available here:</p>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb30-1" data-line-number="1">template_rmd &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">"extdata"</span>,</a>
<a class="sourceLine" id="cb30-2" data-line-number="2">                            <span class="st">"cocktail_recipe.Rmd"</span>,</a>
<a class="sourceLine" id="cb30-3" data-line-number="3">                            <span class="dt">package =</span> <span class="st">"GeneTonic"</span>)</a>
<a class="sourceLine" id="cb30-4" data-line-number="4">template_rmd</a>
<a class="sourceLine" id="cb30-5" data-line-number="5"><span class="co">#&gt; [1] "/home/travis/R/Library/GeneTonic/extdata/cocktail_recipe.Rmd"</span></a></code></pre></div>
</div>
<div id="comparison-between-sets" class="section level2">
<h2 class="hasAnchor">
<a href="#comparison-between-sets" class="anchor"></a>Comparison between sets</h2>
<p>While <code>GeneTonic</code> works currently with a single <code>res_enrich</code> object, there are some functions that allow a visual comparison between two different enrichment results (the functions are displaying a shuffled instance of the single <code>res_enrich</code>):</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb31-1" data-line-number="1"><span class="co"># generating some shuffled gene sets</span></a>
<a class="sourceLine" id="cb31-2" data-line-number="2">res_enrich2 &lt;-<span class="st"> </span>res_enrich_macrophage[<span class="dv">1</span><span class="op">:</span><span class="dv">50</span>, ]</a>
<a class="sourceLine" id="cb31-3" data-line-number="3"><span class="kw"><a href="https://rdrr.io/r/base/Random.html">set.seed</a></span>(<span class="dv">42</span>)</a>
<a class="sourceLine" id="cb31-4" data-line-number="4">shuffled_ones &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-envirAPI.html">sample</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span>(<span class="dv">50</span>)) <span class="co"># to generate permuted p-values</span></a>
<a class="sourceLine" id="cb31-5" data-line-number="5">res_enrich2<span class="op">$</span>gs_pvalue &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>gs_pvalue[shuffled_ones]</a>
<a class="sourceLine" id="cb31-6" data-line-number="6">res_enrich2<span class="op">$</span>z_score &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>z_score[shuffled_ones]</a>
<a class="sourceLine" id="cb31-7" data-line-number="7">res_enrich2<span class="op">$</span>aggr_score &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>aggr_score[shuffled_ones]</a>
<a class="sourceLine" id="cb31-8" data-line-number="8"></a>
<a class="sourceLine" id="cb31-9" data-line-number="9"><span class="kw"><a href="../reference/gs_summary_overview_pair.html">gs_summary_overview_pair</a></span>(<span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb31-10" data-line-number="10">                         <span class="dt">res_enrich2 =</span> res_enrich2,</a>
<a class="sourceLine" id="cb31-11" data-line-number="11">                         <span class="dt">n_gs =</span> <span class="dv">25</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/comparepair-1.png" width="960"></p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb32-1" data-line-number="1">res_enrich2 &lt;-<span class="st"> </span>res_enrich_macrophage[<span class="dv">1</span><span class="op">:</span><span class="dv">42</span>, ]</a>
<a class="sourceLine" id="cb32-2" data-line-number="2">res_enrich3 &lt;-<span class="st"> </span>res_enrich_macrophage[<span class="dv">1</span><span class="op">:</span><span class="dv">42</span>, ]</a>
<a class="sourceLine" id="cb32-3" data-line-number="3">res_enrich4 &lt;-<span class="st"> </span>res_enrich_macrophage[<span class="dv">1</span><span class="op">:</span><span class="dv">42</span>, ]</a>
<a class="sourceLine" id="cb32-4" data-line-number="4"></a>
<a class="sourceLine" id="cb32-5" data-line-number="5"><span class="kw"><a href="https://rdrr.io/r/base/Random.html">set.seed</a></span>(<span class="dv">2</span><span class="op">*</span><span class="dv">42</span>)</a>
<a class="sourceLine" id="cb32-6" data-line-number="6">shuffled_ones_<span class="dv">2</span> &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-envirAPI.html">sample</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span>(<span class="dv">42</span>)) <span class="co"># to generate permuted p-values</span></a>
<a class="sourceLine" id="cb32-7" data-line-number="7">res_enrich2<span class="op">$</span>gs_pvalue &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>gs_pvalue[shuffled_ones_<span class="dv">2</span>]</a>
<a class="sourceLine" id="cb32-8" data-line-number="8">res_enrich2<span class="op">$</span>z_score &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>z_score[shuffled_ones_<span class="dv">2</span>]</a>
<a class="sourceLine" id="cb32-9" data-line-number="9">res_enrich2<span class="op">$</span>aggr_score &lt;-<span class="st"> </span>res_enrich2<span class="op">$</span>aggr_score[shuffled_ones_<span class="dv">2</span>]</a>
<a class="sourceLine" id="cb32-10" data-line-number="10"></a>
<a class="sourceLine" id="cb32-11" data-line-number="11"><span class="kw"><a href="https://rdrr.io/r/base/Random.html">set.seed</a></span>(<span class="dv">3</span><span class="op">*</span><span class="dv">42</span>)</a>
<a class="sourceLine" id="cb32-12" data-line-number="12">shuffled_ones_<span class="dv">3</span> &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-envirAPI.html">sample</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span>(<span class="dv">42</span>)) <span class="co"># to generate permuted p-values</span></a>
<a class="sourceLine" id="cb32-13" data-line-number="13">res_enrich3<span class="op">$</span>gs_pvalue &lt;-<span class="st"> </span>res_enrich3<span class="op">$</span>gs_pvalue[shuffled_ones_<span class="dv">3</span>]</a>
<a class="sourceLine" id="cb32-14" data-line-number="14">res_enrich3<span class="op">$</span>z_score &lt;-<span class="st"> </span>res_enrich3<span class="op">$</span>z_score[shuffled_ones_<span class="dv">3</span>]</a>
<a class="sourceLine" id="cb32-15" data-line-number="15">res_enrich3<span class="op">$</span>aggr_score &lt;-<span class="st"> </span>res_enrich3<span class="op">$</span>aggr_score[shuffled_ones_<span class="dv">3</span>]</a>
<a class="sourceLine" id="cb32-16" data-line-number="16"></a>
<a class="sourceLine" id="cb32-17" data-line-number="17"><span class="kw"><a href="https://rdrr.io/r/base/Random.html">set.seed</a></span>(<span class="dv">4</span><span class="op">*</span><span class="dv">42</span>)</a>
<a class="sourceLine" id="cb32-18" data-line-number="18">shuffled_ones_<span class="dv">4</span> &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-envirAPI.html">sample</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/seq.html">seq_len</a></span>(<span class="dv">42</span>)) <span class="co"># to generate permuted p-values</span></a>
<a class="sourceLine" id="cb32-19" data-line-number="19">res_enrich4<span class="op">$</span>gs_pvalue &lt;-<span class="st"> </span>res_enrich4<span class="op">$</span>gs_pvalue[shuffled_ones_<span class="dv">4</span>]</a>
<a class="sourceLine" id="cb32-20" data-line-number="20">res_enrich4<span class="op">$</span>z_score &lt;-<span class="st"> </span>res_enrich4<span class="op">$</span>z_score[shuffled_ones_<span class="dv">4</span>]</a>
<a class="sourceLine" id="cb32-21" data-line-number="21">res_enrich4<span class="op">$</span>aggr_score &lt;-<span class="st"> </span>res_enrich4<span class="op">$</span>aggr_score[shuffled_ones_<span class="dv">4</span>]</a>
<a class="sourceLine" id="cb32-22" data-line-number="22"></a>
<a class="sourceLine" id="cb32-23" data-line-number="23">compa_list &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/list.html">list</a></span>(</a>
<a class="sourceLine" id="cb32-24" data-line-number="24">  <span class="dt">scenario2 =</span> res_enrich2,</a>
<a class="sourceLine" id="cb32-25" data-line-number="25">  <span class="dt">scenario3 =</span> res_enrich3,</a>
<a class="sourceLine" id="cb32-26" data-line-number="26">  <span class="dt">scenario4 =</span> res_enrich4</a>
<a class="sourceLine" id="cb32-27" data-line-number="27">)</a>
<a class="sourceLine" id="cb32-28" data-line-number="28"></a>
<a class="sourceLine" id="cb32-29" data-line-number="29"><span class="kw"><a href="../reference/gs_horizon.html">gs_horizon</a></span>(res_enrich_macrophage,</a>
<a class="sourceLine" id="cb32-30" data-line-number="30">           <span class="dt">compared_res_enrich_list =</span> compa_list,</a>
<a class="sourceLine" id="cb32-31" data-line-number="31">           <span class="dt">n_gs =</span> <span class="dv">20</span>,</a>
<a class="sourceLine" id="cb32-32" data-line-number="32">           <span class="dt">sort_by =</span> <span class="st">"clustered"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/compare4-1.png" width="960"></p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb33-1" data-line-number="1"><span class="co"># with only one set</span></a>
<a class="sourceLine" id="cb33-2" data-line-number="2"><span class="kw"><a href="../reference/gs_radar.html">gs_radar</a></span>(<span class="dt">res_enrich =</span> res_enrich_macrophage)</a></code></pre></div>
<div id="htmlwidget-0f94608c1d372ff00d79" style="width:960px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-0f94608c1d372ff00d79">{"x":{"visdat":{"406979b9ddef":["function () ","plotlyVisDat"]},"cur_data":"406979b9ddef","attrs":{"406979b9ddef":{"mode":"markers","fill":"toself","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatterpolar"},"406979b9ddef.1":{"mode":"markers","fill":"toself","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatterpolar","r":[22.0362121726544,19.9208187539524,12.3187587626244,11.3767507096021,9.69897000433602,9.2518119729938,9.23657200643706,7.72124639904717,7.65757731917779,7.53760200210104,7.0268721464003,6.95860731484177,6.60205999132796,6.38721614328026,6.35654732351381,6.14874165128092,6.07058107428571,5.92081875395237,5.85387196432176,5.79588001734407,22.0362121726544],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 1","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"polar":{"radialaxis":{"visible":true,"range":[0,22.0362121726544]}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","mode":"markers","fill":"toself","type":"scatterpolar","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"fillcolor":"rgba(255,127,14,0.5)","mode":"markers","fill":"toself","type":"scatterpolar","r":[22.0362121726544,19.9208187539524,12.3187587626244,11.3767507096021,9.69897000433602,9.2518119729938,9.23657200643706,7.72124639904717,7.65757731917779,7.53760200210104,7.0268721464003,6.95860731484177,6.60205999132796,6.38721614328026,6.35654732351381,6.14874165128092,6.07058107428571,5.92081875395237,5.85387196432176,5.79588001734407,22.0362121726544],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 1","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"line":{"color":"rgba(255,127,14,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb34-1" data-line-number="1"><span class="co"># with a dataset to compare against</span></a>
<a class="sourceLine" id="cb34-2" data-line-number="2"><span class="kw"><a href="../reference/gs_radar.html">gs_radar</a></span>(<span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb34-3" data-line-number="3">         <span class="dt">res_enrich2 =</span> res_enrich2)</a></code></pre></div>
<div id="htmlwidget-b2bf2b2e3a2ab432b57e" style="width:960px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-b2bf2b2e3a2ab432b57e">{"x":{"visdat":{"40691491df4a":["function () ","plotlyVisDat"]},"cur_data":"40691491df4a","attrs":{"40691491df4a":{"mode":"markers","fill":"toself","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatterpolar"},"40691491df4a.1":{"mode":"markers","fill":"toself","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatterpolar","r":[22.0362121726544,19.9208187539524,12.3187587626244,11.3767507096021,9.69897000433602,9.2518119729938,9.23657200643706,7.72124639904717,7.65757731917779,7.53760200210104,7.0268721464003,6.95860731484177,6.60205999132796,6.38721614328026,6.35654732351381,6.14874165128092,6.07058107428571,5.92081875395237,5.85387196432176,5.79588001734407,22.0362121726544],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 1","inherit":true},"40691491df4a.2":{"mode":"markers","fill":"toself","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatterpolar","r":[4.72124639904717,5.85387196432176,5.38721614328026,5.17392519729917,6.95860731484177,5.72124639904717,9.23657200643706,3.61978875828839,3.82390874094432,3.69897000433602,4.52287874528034,6.35654732351381,7.0268721464003,4.92081875395237,4.04575749056067,6.07058107428571,5.17392519729917,3.82390874094432,5,4.04575749056067,4.72124639904717],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 2","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"polar":{"radialaxis":{"visible":true,"range":[0,22.0362121726544]}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","mode":"markers","fill":"toself","type":"scatterpolar","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"fillcolor":"rgba(255,127,14,0.5)","mode":"markers","fill":"toself","type":"scatterpolar","r":[22.0362121726544,19.9208187539524,12.3187587626244,11.3767507096021,9.69897000433602,9.2518119729938,9.23657200643706,7.72124639904717,7.65757731917779,7.53760200210104,7.0268721464003,6.95860731484177,6.60205999132796,6.38721614328026,6.35654732351381,6.14874165128092,6.07058107428571,5.92081875395237,5.85387196432176,5.79588001734407,22.0362121726544],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 1","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"fillcolor":"rgba(44,160,44,0.5)","mode":"markers","fill":"toself","type":"scatterpolar","r":[4.72124639904717,5.85387196432176,5.38721614328026,5.17392519729917,6.95860731484177,5.72124639904717,9.23657200643706,3.61978875828839,3.82390874094432,3.69897000433602,4.52287874528034,6.35654732351381,7.0268721464003,4.92081875395237,4.04575749056067,6.07058107428571,5.17392519729917,3.82390874094432,5,4.04575749056067,4.72124639904717],"theta":["adaptive immune response","interferon-gamma-mediated signaling pathway","type I interferon signaling pathway","antigen processing and presentation of endogenous peptide antigen via MHC class I","defense response to virus","positive regulation of T cell mediated cytotoxicity","antigen processing and presentation via MHC class Ib","DNA replication initiation","response to interferon-gamma","inflammatory response","T cell receptor signaling pathway","innate immune response","complement activation, classical pathway","double-strand break repair via break-induced replication","defense response to protozoan","cellular response to lipopolysaccharide","regulation of complement activation","chemokine-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","antigen processing and presentation of exogenous peptide antigen via MHC class II","adaptive immune response"],"name":"scenario 2","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>In particular, the <code>gs_horizon</code> function tries to group the different genesets based on their relative enrichment across conditions, similar to a representation of <span class="citation">(Dutertre et al. 2019)</span> (e.g. Fig. 5G). Thanks to Charlotte Soneson for providing a compact working implementation of this!</p>
</div>
<div id="miscellaneous-functions" class="section level2">
<h2 class="hasAnchor">
<a href="#miscellaneous-functions" class="anchor"></a>Miscellaneous functions</h2>
<p>Extract results and sort them (filtering by FDR) with <code>deseqresult2df</code>, select one top-ranking gene, extract its expression values and plot them, and display some buttons to link to external databases:</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb35-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="kw"><a href="../reference/deseqresult2df.html">deseqresult2df</a></span>(res_macrophage_IFNg_vs_naive))</a>
<a class="sourceLine" id="cb35-2" data-line-number="2"><span class="co">#&gt;                              id   baseMean log2FoldChange     lfcSE     stat       pvalue</span></a>
<a class="sourceLine" id="cb35-3" data-line-number="3"><span class="co">#&gt; ENSG00000125347 ENSG00000125347 30585.0749       5.560124 0.2182663 20.89248 6.268470e-97</span></a>
<a class="sourceLine" id="cb35-4" data-line-number="4"><span class="co">#&gt; ENSG00000162645 ENSG00000162645 36790.5921       6.665804 0.2865732 19.77088 5.304222e-87</span></a>
<a class="sourceLine" id="cb35-5" data-line-number="5"><span class="co">#&gt; ENSG00000111181 ENSG00000111181   688.5779       4.710592 0.1961023 18.92172 7.555181e-80</span></a>
<a class="sourceLine" id="cb35-6" data-line-number="6"><span class="co">#&gt; ENSG00000174944 ENSG00000174944  1324.3975       9.863904 0.4715127 18.79886 7.715124e-79</span></a>
<a class="sourceLine" id="cb35-7" data-line-number="7"><span class="co">#&gt; ENSG00000100336 ENSG00000100336  2551.4789       8.708551 0.4214096 18.29230 9.530829e-75</span></a>
<a class="sourceLine" id="cb35-8" data-line-number="8"><span class="co">#&gt; ENSG00000145365 ENSG00000145365  3637.2221       5.193448 0.2368542 17.70476 3.853076e-70</span></a>
<a class="sourceLine" id="cb35-9" data-line-number="9"><span class="co">#&gt;                         padj  SYMBOL</span></a>
<a class="sourceLine" id="cb35-10" data-line-number="10"><span class="co">#&gt; ENSG00000125347 1.116164e-92    IRF1</span></a>
<a class="sourceLine" id="cb35-11" data-line-number="11"><span class="co">#&gt; ENSG00000162645 4.722349e-83    GBP2</span></a>
<a class="sourceLine" id="cb35-12" data-line-number="12"><span class="co">#&gt; ENSG00000111181 4.484252e-76 SLC6A12</span></a>
<a class="sourceLine" id="cb35-13" data-line-number="13"><span class="co">#&gt; ENSG00000174944 3.434388e-75  P2RY14</span></a>
<a class="sourceLine" id="cb35-14" data-line-number="14"><span class="co">#&gt; ENSG00000100336 3.394119e-71   APOL4</span></a>
<a class="sourceLine" id="cb35-15" data-line-number="15"><span class="co">#&gt; ENSG00000145365 1.143465e-66    TIFA</span></a>
<a class="sourceLine" id="cb35-16" data-line-number="16"><span class="co"># to make sure normalized values are available...</span></a>
<a class="sourceLine" id="cb35-17" data-line-number="17">dds_macrophage &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/pkg/DESeq2/man/estimateSizeFactors.html">estimateSizeFactors</a></span>(dds_macrophage)   </a>
<a class="sourceLine" id="cb35-18" data-line-number="18"><span class="kw"><a href="../reference/gene_plot.html">gene_plot</a></span>(dds_macrophage,</a>
<a class="sourceLine" id="cb35-19" data-line-number="19">          <span class="dt">gene =</span> <span class="st">"ENSG00000125347"</span>,</a>
<a class="sourceLine" id="cb35-20" data-line-number="20">          <span class="dt">intgroup =</span> <span class="st">"condition"</span>,</a>
<a class="sourceLine" id="cb35-21" data-line-number="21">          <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb35-22" data-line-number="22">          <span class="dt">plot_type =</span> <span class="st">"auto"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/misc-1.png" width="960"></p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb36-1" data-line-number="1"><span class="kw"><a href="../reference/gene_plot.html">gene_plot</a></span>(dds_macrophage,</a>
<a class="sourceLine" id="cb36-2" data-line-number="2">          <span class="dt">gene =</span> <span class="st">"ENSG00000174944"</span>,</a>
<a class="sourceLine" id="cb36-3" data-line-number="3">          <span class="dt">intgroup =</span> <span class="st">"condition"</span>,</a>
<a class="sourceLine" id="cb36-4" data-line-number="4">          <span class="dt">assay =</span> <span class="st">"abundance"</span>,</a>
<a class="sourceLine" id="cb36-5" data-line-number="5">          <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb36-6" data-line-number="6">          <span class="dt">plot_type =</span> <span class="st">"auto"</span>)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/misc-2.png" width="960"></p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb37-1" data-line-number="1"></a>
<a class="sourceLine" id="cb37-2" data-line-number="2"><span class="kw"><a href="../reference/geneinfo_2_html.html">geneinfo_2_html</a></span>(<span class="st">"IRF1"</span>)</a></code></pre></div>
<b>IRF1</b><br>Link to the NCBI Gene database: <a href="http://www.ncbi.nlm.nih.gov/gene/?term=IRF1%5Bsym%5D" target="_blank" class="btn btn-primary" style="color: #ffffff; background-color: #0092AC">IRF1</a><br>Link to the GeneCards database: <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=IRF1" target="_blank" class="btn btn-primary" style="color: #ffffff; background-color: #0092AC">IRF1</a><br><p>Plot a signature heatmap for a gene set, where a matrix of genes times samples for the set members is extracted, plus generate some HTML code to summarize the information of a Gene Ontology term:</p>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb38-1" data-line-number="1"><span class="kw"><a href="../reference/gs_heatmap.html">gs_heatmap</a></span>(<span class="dt">se =</span> vst_macrophage,</a>
<a class="sourceLine" id="cb38-2" data-line-number="2">           <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb38-3" data-line-number="3">           <span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb38-4" data-line-number="4">           <span class="dt">annotation_obj =</span> anno_df,</a>
<a class="sourceLine" id="cb38-5" data-line-number="5">           <span class="dt">geneset_id =</span> <span class="st">"GO:0060337"</span>  ,</a>
<a class="sourceLine" id="cb38-6" data-line-number="6">           <span class="dt">cluster_columns =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb38-7" data-line-number="7">           <span class="dt">anno_col_info =</span> <span class="st">"condition"</span></a>
<a class="sourceLine" id="cb38-8" data-line-number="8">)</a></code></pre></div>
<p><img src="GeneTonic_manual_files/figure-html/gsheatmap-1.png" width="960"></p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb39-1" data-line-number="1"></a>
<a class="sourceLine" id="cb39-2" data-line-number="2"><span class="kw"><a href="../reference/go_2_html.html">go_2_html</a></span>(<span class="st">"GO:0060337"</span>,</a>
<a class="sourceLine" id="cb39-3" data-line-number="3">          <span class="dt">res_enrich =</span> res_enrich_macrophage)</a></code></pre></div>
<b>GO ID: </b><a href="http://amigo.geneontology.org/amigo/term/GO:0060337" target="_blank" class="btn btn-primary" style="color: #ffffff; background-color: #0092AC">GO:0060337</a><br><b>Term: </b>type I interferon signaling pathway<br><b>p-value: </b>4.8e-13<br><b>Z-score: </b>5.00<br><b>Aggregated score: </b>3.34<br><b>Ontology: </b>BP<br><br><b>Definition: </b>A series of molecular signals initiated by the binding of a type I interferon to a receptor on the surface of a cell, and ending with regulation of a downstream cellular process, e.g. transcription. Type I interferons include the interferon-alpha, beta, delta, episilon, zeta, kappa, tau, and omega gene families.<br><b>Synonym: </b>type I interferon-activated signaling pathway<br><b>Synonym: </b>type I interferon-mediated signaling pathway<br><b>Synonym: </b>type I interferon-mediated signalling pathway<br><p>Some functions are just to ensure that the input objects are conform with the format expected by <code>GeneTonic</code>:</p>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb40-1" data-line-number="1">res_enrich &lt;-<span class="st"> </span><span class="kw"><a href="../reference/shake_enrichResult.html">shake_enrichResult</a></span>(enrichment_results_from_clusterProfiler)</a>
<a class="sourceLine" id="cb40-2" data-line-number="2">res_enrich &lt;-<span class="st"> </span><span class="kw"><a href="../reference/shake_topGOtableResult.html">shake_topGOtableResult</a></span>(enrichment_results_from_topGOtable)</a></code></pre></div>
<p>As an internal check, one can see if all required arguments are fine for running <code>GeneTonic</code> by running <code>checkup_GeneTonic</code>:</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb41-1" data-line-number="1"><span class="kw"><a href="../reference/checkup_GeneTonic.html">checkup_GeneTonic</a></span>(<span class="dt">dds =</span> dds_macrophage,</a>
<a class="sourceLine" id="cb41-2" data-line-number="2">                  <span class="dt">res_de =</span> res_macrophage_IFNg_vs_naive,</a>
<a class="sourceLine" id="cb41-3" data-line-number="3">                  <span class="dt">res_enrich =</span> res_enrich_macrophage,</a>
<a class="sourceLine" id="cb41-4" data-line-number="4">                  <span class="dt">annotation_obj =</span> anno_df)</a>
<a class="sourceLine" id="cb41-5" data-line-number="5"><span class="co"># if all is fine, it should return an invisible NULL and a simple message</span></a></code></pre></div>
</div>
</div>
<div id="additionalinfo" class="section level1">
<h1 class="hasAnchor">
<a href="#additionalinfo" class="anchor"></a>Additional Information</h1>
<p>Bug reports can be posted on the Bioconductor support site (<a href="https://support.bioconductor.org/" class="uri">https://support.bioconductor.org/</a>) or raised as issues in the <code>GeneTonic</code> GitHub repository (<a href="https://github.com/federicomarini/GeneTonic/issues" class="uri">https://github.com/federicomarini/GeneTonic/issues</a>). The GitHub repository also contains the development version of the package, where new functionality is added over time - careful, you might be running bleeding edge versions!</p>
<p>The authors appreciate well-considered suggestions for improvements or new features, or even better, pull requests.</p>
<p>If you use <code>GeneTonic</code> for your analysis, please cite it as shown below:</p>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb42-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/citation.html">citation</a></span>(<span class="st">"GeneTonic"</span>)</a>
<a class="sourceLine" id="cb42-2" data-line-number="2"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb42-3" data-line-number="3"><span class="co">#&gt; To cite package 'GeneTonic' in publications use:</span></a>
<a class="sourceLine" id="cb42-4" data-line-number="4"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb42-5" data-line-number="5"><span class="co">#&gt;   Federico Marini (2020). GeneTonic: Enjoy Analyzing And Integrating The Results From</span></a>
<a class="sourceLine" id="cb42-6" data-line-number="6"><span class="co">#&gt;   Differential Expression Analysis And Functional Enrichment Analysis. R package version</span></a>
<a class="sourceLine" id="cb42-7" data-line-number="7"><span class="co">#&gt;   0.99.6. https://github.com/federicomarini/GeneTonic</span></a>
<a class="sourceLine" id="cb42-8" data-line-number="8"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb42-9" data-line-number="9"><span class="co">#&gt; A BibTeX entry for LaTeX users is</span></a>
<a class="sourceLine" id="cb42-10" data-line-number="10"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb42-11" data-line-number="11"><span class="co">#&gt;   @Manual{,</span></a>
<a class="sourceLine" id="cb42-12" data-line-number="12"><span class="co">#&gt;     title = {GeneTonic: Enjoy Analyzing And Integrating The Results From Differential</span></a>
<a class="sourceLine" id="cb42-13" data-line-number="13"><span class="co">#&gt; Expression Analysis And Functional Enrichment Analysis},</span></a>
<a class="sourceLine" id="cb42-14" data-line-number="14"><span class="co">#&gt;     author = {Federico Marini},</span></a>
<a class="sourceLine" id="cb42-15" data-line-number="15"><span class="co">#&gt;     year = {2020},</span></a>
<a class="sourceLine" id="cb42-16" data-line-number="16"><span class="co">#&gt;     note = {R package version 0.99.6},</span></a>
<a class="sourceLine" id="cb42-17" data-line-number="17"><span class="co">#&gt;     url = {https://github.com/federicomarini/GeneTonic},</span></a>
<a class="sourceLine" id="cb42-18" data-line-number="18"><span class="co">#&gt;   }</span></a></code></pre></div>
</div>
<div id="faqs" class="section level1">
<h1 class="hasAnchor">
<a href="#faqs" class="anchor"></a>FAQs</h1>
<p><strong>Q: Why is this package called <code>GeneTonic</code>?</strong></p>
<p>A: For obvious reasons :) Like in a cocktail, ingredients (expression matrix, results from DE, and results from functional enrichment analysis) do taste better together. Plus, I like puns with words. And cocktails. Sometimes simultaneously.</p>
<p><strong>Q: My configuration on two machines is somewhat different, so I am having difficulty in finding out what packages are different. Is there something tio help on this?</strong></p>
<p>A: Yes, you can check out <em><a href="https://github.com/federicomarini/sessionDiffo">sessionDiffo</a></em>, a small utility to compare the outputs of two different <code>sessionInfo</code> outputs. This can help you pinpoint what packages might be causing the issue.</p>
<p><strong>Q: I am using a different service/software for generating the results of functional enrichment analysis. How do I plug this into <code>GeneTonic</code>?</strong></p>
<p>A: You can set up a small conversion function, on the line of <code>shake_topGOtableResult</code> and <code>shake_enrichResult</code> (<em>use the source, Luke</em> to see how you can build your own), or request such a function - make sure to specify the formatting specifications of your tool of choice (e.g. David, g:Profiler, or commercial solutions such as Ingenuity Pathway Analysis).</p>
<p><strong>Q: I’d like to try <code>GeneTonic</code> but I could not install it/I just want a quick peek into it. What can I do for this?</strong></p>
<p>A: We set up an instance of <code>GeneTonic</code> running on the <code>macrophage</code> dataset at this address: <a href="http://shiny.imbei.uni-mainz.de:3838/GeneTonic" class="uri">http://shiny.imbei.uni-mainz.de:3838/GeneTonic</a>.</p>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h1>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb43-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span>()</a>
<a class="sourceLine" id="cb43-2" data-line-number="2"><span class="co">#&gt; R Under development (unstable) (2020-02-12 r77798)</span></a>
<a class="sourceLine" id="cb43-3" data-line-number="3"><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></a>
<a class="sourceLine" id="cb43-4" data-line-number="4"><span class="co">#&gt; Running under: Ubuntu 16.04.6 LTS</span></a>
<a class="sourceLine" id="cb43-5" data-line-number="5"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb43-6" data-line-number="6"><span class="co">#&gt; Matrix products: default</span></a>
<a class="sourceLine" id="cb43-7" data-line-number="7"><span class="co">#&gt; BLAS:   /home/travis/R-bin/lib/R/lib/libRblas.so</span></a>
<a class="sourceLine" id="cb43-8" data-line-number="8"><span class="co">#&gt; LAPACK: /home/travis/R-bin/lib/R/lib/libRlapack.so</span></a>
<a class="sourceLine" id="cb43-9" data-line-number="9"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb43-10" data-line-number="10"><span class="co">#&gt; locale:</span></a>
<a class="sourceLine" id="cb43-11" data-line-number="11"><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       </span></a>
<a class="sourceLine" id="cb43-12" data-line-number="12"><span class="co">#&gt;  [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></a>
<a class="sourceLine" id="cb43-13" data-line-number="13"><span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              </span></a>
<a class="sourceLine" id="cb43-14" data-line-number="14"><span class="co">#&gt; [10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></a>
<a class="sourceLine" id="cb43-15" data-line-number="15"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb43-16" data-line-number="16"><span class="co">#&gt; attached base packages:</span></a>
<a class="sourceLine" id="cb43-17" data-line-number="17"><span class="co">#&gt; [1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     </span></a>
<a class="sourceLine" id="cb43-18" data-line-number="18"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb43-19" data-line-number="19"><span class="co">#&gt; other attached packages:</span></a>
<a class="sourceLine" id="cb43-20" data-line-number="20"><span class="co">#&gt;  [1] magrittr_1.5                visNetwork_2.0.9            plotly_4.9.2               </span></a>
<a class="sourceLine" id="cb43-21" data-line-number="21"><span class="co">#&gt;  [4] ggplot2_3.2.1               org.Hs.eg.db_3.10.0         AnnotationDbi_1.49.1       </span></a>
<a class="sourceLine" id="cb43-22" data-line-number="22"><span class="co">#&gt;  [7] DESeq2_1.27.24              SummarizedExperiment_1.17.1 DelayedArray_0.13.3        </span></a>
<a class="sourceLine" id="cb43-23" data-line-number="23"><span class="co">#&gt; [10] BiocParallel_1.21.2         matrixStats_0.55.0          Biobase_2.47.2             </span></a>
<a class="sourceLine" id="cb43-24" data-line-number="24"><span class="co">#&gt; [13] GenomicRanges_1.39.2        GenomeInfoDb_1.23.12        IRanges_2.21.3             </span></a>
<a class="sourceLine" id="cb43-25" data-line-number="25"><span class="co">#&gt; [16] S4Vectors_0.25.12           BiocGenerics_0.33.0         macrophage_1.3.1           </span></a>
<a class="sourceLine" id="cb43-26" data-line-number="26"><span class="co">#&gt; [19] GeneTonic_0.99.6            BiocStyle_2.15.6           </span></a>
<a class="sourceLine" id="cb43-27" data-line-number="27"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb43-28" data-line-number="28"><span class="co">#&gt; loaded via a namespace (and not attached):</span></a>
<a class="sourceLine" id="cb43-29" data-line-number="29"><span class="co">#&gt;   [1] colorspace_1.4-1       rjson_0.2.20           ellipsis_0.3.0         dynamicTreeCut_1.63-1 </span></a>
<a class="sourceLine" id="cb43-30" data-line-number="30"><span class="co">#&gt;   [5] rprojroot_1.3-2        circlize_0.4.8         XVector_0.27.0         GlobalOptions_0.1.1   </span></a>
<a class="sourceLine" id="cb43-31" data-line-number="31"><span class="co">#&gt;   [9] fs_1.3.1               clue_0.3-57            farver_2.0.3           ggrepel_0.8.1         </span></a>
<a class="sourceLine" id="cb43-32" data-line-number="32"><span class="co">#&gt;  [13] DT_0.12                bit64_0.9-7            splines_4.0.0          geneplotter_1.65.0    </span></a>
<a class="sourceLine" id="cb43-33" data-line-number="33"><span class="co">#&gt;  [17] knitr_1.28             polyclip_1.10-0        jsonlite_1.6.1         annotate_1.65.1       </span></a>
<a class="sourceLine" id="cb43-34" data-line-number="34"><span class="co">#&gt;  [21] cluster_2.1.0          GO.db_3.10.0           png_0.1-7              ggforce_0.3.1         </span></a>
<a class="sourceLine" id="cb43-35" data-line-number="35"><span class="co">#&gt;  [25] shiny_1.4.0            BiocManager_1.30.10    compiler_4.0.0         httr_1.4.1            </span></a>
<a class="sourceLine" id="cb43-36" data-line-number="36"><span class="co">#&gt;  [29] backports_1.1.5        assertthat_0.2.1       Matrix_1.2-18          fastmap_1.0.1         </span></a>
<a class="sourceLine" id="cb43-37" data-line-number="37"><span class="co">#&gt;  [33] lazyeval_0.2.2         later_1.0.0            tweenr_1.0.1           htmltools_0.4.0       </span></a>
<a class="sourceLine" id="cb43-38" data-line-number="38"><span class="co">#&gt;  [37] tools_4.0.0            igraph_1.2.4.2         gtable_0.3.0           glue_1.3.1            </span></a>
<a class="sourceLine" id="cb43-39" data-line-number="39"><span class="co">#&gt;  [41] GenomeInfoDbData_1.2.2 dplyr_0.8.4            Rcpp_1.0.3             pkgdown_1.4.1         </span></a>
<a class="sourceLine" id="cb43-40" data-line-number="40"><span class="co">#&gt;  [45] vctrs_0.2.2            crosstalk_1.0.0        rintrojs_0.2.2         xfun_0.12             </span></a>
<a class="sourceLine" id="cb43-41" data-line-number="41"><span class="co">#&gt;  [49] stringr_1.4.0          mime_0.9               lifecycle_0.1.0        shinycssloaders_0.3   </span></a>
<a class="sourceLine" id="cb43-42" data-line-number="42"><span class="co">#&gt;  [53] XML_3.99-0.3           dendextend_1.13.3      zlibbioc_1.33.1        MASS_7.3-51.5         </span></a>
<a class="sourceLine" id="cb43-43" data-line-number="43"><span class="co">#&gt;  [57] scales_1.1.0           promises_1.1.0         RColorBrewer_1.1-2     ComplexHeatmap_2.3.0  </span></a>
<a class="sourceLine" id="cb43-44" data-line-number="44"><span class="co">#&gt;  [61] yaml_2.2.1             memoise_1.1.0          gridExtra_2.3          stringi_1.4.5         </span></a>
<a class="sourceLine" id="cb43-45" data-line-number="45"><span class="co">#&gt;  [65] RSQLite_2.2.0          genefilter_1.69.0      desc_1.2.0             shape_1.4.4           </span></a>
<a class="sourceLine" id="cb43-46" data-line-number="46"><span class="co">#&gt;  [69] rlang_0.4.4            pkgconfig_2.0.3        bitops_1.0-6           evaluate_0.14         </span></a>
<a class="sourceLine" id="cb43-47" data-line-number="47"><span class="co">#&gt;  [73] lattice_0.20-38        purrr_0.3.3            labeling_0.3           htmlwidgets_1.5.1     </span></a>
<a class="sourceLine" id="cb43-48" data-line-number="48"><span class="co">#&gt;  [77] bit_1.1-15.2           tidyselect_1.0.0       bookdown_0.17          R6_2.4.1              </span></a>
<a class="sourceLine" id="cb43-49" data-line-number="49"><span class="co">#&gt;  [81] DBI_1.1.0              withr_2.1.2            pillar_1.4.3           survival_3.1-8        </span></a>
<a class="sourceLine" id="cb43-50" data-line-number="50"><span class="co">#&gt;  [85] RCurl_1.98-1.1         tibble_2.1.3           crayon_1.3.4           shinyWidgets_0.5.0    </span></a>
<a class="sourceLine" id="cb43-51" data-line-number="51"><span class="co">#&gt;  [89] bs4Dash_0.5.0          rmarkdown_2.1          viridis_0.5.1          GetoptLong_0.1.8      </span></a>
<a class="sourceLine" id="cb43-52" data-line-number="52"><span class="co">#&gt;  [93] locfit_1.5-9.1         grid_4.0.0             data.table_1.12.8      blob_1.2.1            </span></a>
<a class="sourceLine" id="cb43-53" data-line-number="53"><span class="co">#&gt;  [97] digest_0.6.24          xtable_1.8-4           tidyr_1.0.2            httpuv_1.5.2          </span></a>
<a class="sourceLine" id="cb43-54" data-line-number="54"><span class="co">#&gt; [101] munsell_0.5.0          viridisLite_0.3.0</span></a></code></pre></div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-Alasoo2018">
<p>Alasoo, Kaur, Julia Rodrigues, Subhankar Mukhopadhyay, Andrew J. Knights, Alice L. Mann, Kousik Kundu, Christine Hale, Gordon Dougan, and Daniel J. Gaffney. 2018. “Shared genetic effects on chromatin and gene expression indicate a role for enhancer priming in immune response.” <em>Nature Genetics</em> 50 (3). Springer US: 424–31. <a href="https://doi.org/10.1038/s41588-018-0046-7" class="uri">https://doi.org/10.1038/s41588-018-0046-7</a>.</p>
</div>
<div id="ref-Dutertre2019">
<p>Dutertre, Charles Antoine, Etienne Becht, Sergio Erdal Irac, Ahad Khalilnezhad, Vipin Narang, Shabnam Khalilnezhad, Pei Y. Ng, et al. 2019. “Single-Cell Analysis of Human Mononuclear Phagocytes Reveals Subset-Defining Markers and Identifies Circulating Inflammatory Dendritic Cells.” <em>Immunity</em> 51 (3). Elsevier Inc.: 573–589.e8. <a href="https://doi.org/10.1016/j.immuni.2019.08.008" class="uri">https://doi.org/10.1016/j.immuni.2019.08.008</a>.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">

        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li>
<a href="#gettingstarted">Getting started</a><ul class="nav nav-pills nav-stacked">
<li><a href="#all-set">All set!</a></li>
      </ul>
</li>
      <li>
<a href="#userinterface">Description of the <code>GeneTonic</code> user interface</a><ul class="nav nav-pills nav-stacked">
<li><a href="#header-navbar">Header (navbar)</a></li>
      <li><a href="#sidebar">Sidebar</a></li>
      <li><a href="#controlbar">Controlbar</a></li>
      <li><a href="#body">Body</a></li>
      </ul>
</li>
      <li>
<a href="#functionality">The <code>GeneTonic</code> functionality</a><ul class="nav nav-pills nav-stacked">
<li><a href="#the-welcome-panel">The Welcome panel</a></li>
      <li><a href="#the-gene-geneset-panel">The Gene-Geneset panel</a></li>
      <li><a href="#the-enrichment-map-panel">The Enrichment Map panel</a></li>
      <li><a href="#the-overview-panel">The Overview panel</a></li>
      <li><a href="#the-gsviz-panel">The GSViz panel</a></li>
      <li><a href="#the-bookmarks-panel">The Bookmarks panel</a></li>
      </ul>
</li>
      <li>
<a href="#functions">Using the <code>GeneTonic</code> functions</a><ul class="nav nav-pills nav-stacked">
<li><a href="#overview-functions-genes-and-gene-sets">Overview functions: genes and gene sets</a></li>
      <li><a href="#summary-representations">Summary representations</a></li>
      <li><a href="#reporting">Reporting</a></li>
      <li><a href="#comparison-between-sets">Comparison between sets</a></li>
      <li><a href="#miscellaneous-functions">Miscellaneous functions</a></li>
      </ul>
</li>
      <li><a href="#additionalinfo">Additional Information</a></li>
      <li><a href="#faqs">FAQs</a></li>
      <li><a href="#session-info">Session Info</a></li>
      <li><a href="#references">References</a></li>
      </ul>
</div>
      </div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://federicomarini.github.io">Federico Marini</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
